##`r sigSetName` {.tabset .tabset-fade .tabset-pills}  

Correlations greater than 0.2 or less than -0.2 are in red.   
  
```{r}

## get genes for this gene-set
sigSet <- btm_ind[[sigSetName]]

## combine rnaseq reads with meta data
gene_dt <- melt(mal067_voom$E[sigSet,])
dplyr::inner_join(prune_meta$targets, gene_dt, by=c("col_id" = "Var2")) %>%
  dplyr::rename(gene=Var1) %>%
  mutate(gene=as.character(gene)) ->
  gene_dt

## join case factor to flow data
dplyr::inner_join(gene_dt, data, by=c("sid", "visit")) %>%
  as.data.table() ->
  all_DT

```

### Both  
  
#### Sample size  

```{r}
countSampleSizes(all_DT[!duplicated(sid)], "visit", 
                 c("case", "stimulation", "age")) %>%
  headerKable(labels=c("case", "stimulation", "age"), levels=c(3,4,2))
```

```{r}

  corAllClass <- all_DT[visit=="M3",.(pearson=signif(cor(value, 
                                                         difference_trunc), 3),
                           spearman=signif(cor(value, difference_trunc,
                                               method="spearman"), 3))]
  
  corGeneClass <- all_DT[visit=="M3",.(pearson=signif(cor(value, 
                                                          difference_trunc), 3),
                            spearman=signif(cor(value, difference_trunc,
                                                method="spearman"), 3)),
                by=.(gene)]
```
  
#### Correlation: all genes together  
  
```{r}
corAllClass %>%
mutate(pearson=cell_spec(pearson, color=ifelse(pearson >= 0.2 | pearson <= -0.2, 
                                        "red", "black"))) %>%
mutate(spearman=cell_spec(spearman, color=ifelse(spearman >= 0.2 | pearson <= -0.2, 
                                        "red", "black"))) %>%
headerKable()
```

#### Correlation plots  
  
```{r}

new_cor <- melt(corGeneClass, measure=c("pearson", "spearman"))

 ggplot(melt(corGeneClass, measure=c("pearson", "spearman")), 
         aes(x=variable, y=value)) + geom_boxplot() + ylim(-1,1) + xlab("type") +
    ylab("correlation") + geom_hline(yintercept=0, linetype="dashed", color="red")
```

#### Correlation tables  
 
```{r}

corGeneClass %>% 
  mutate(pearson=cell_spec(pearson, color=ifelse(pearson >= 0.2 | pearson <= -0.2, 
                                        "red", "black"))) %>%
  mutate(spearman=cell_spec(spearman, color=ifelse(spearman >= 0.2 | pearson<= -0.2, 
                                        "red", "black"))) %>%
headerKable()
```



### Young  
  
#### Sample size  

```{r}

countSampleSizes(all_DT[!duplicated(sid) & age=="young"], "visit", 
                 c("case", "stimulation", "age")) %>%
  headerKable(labels=c("case", "stimulation", "age"), levels=c(3,4,2))
```

```{r}

  corAllClass <- all_DT[age=="young",.(pearson=signif(cor(value, 
                                                         difference_trunc), 3),
                           spearman=signif(cor(value, difference_trunc,
                                               method="spearman"), 3))]
  
  corGeneClass <- all_DT[age=="young",.(pearson=signif(cor(value, 
                                                          difference_trunc), 3),
                            spearman=signif(cor(value, difference_trunc,
                                                method="spearman"), 3)),
                by=.(gene)]
```
  
#### Correlation: all genes together  
  
```{r}
corAllClass %>%
mutate(pearson=cell_spec(pearson, color=ifelse(pearson >= 0.2 | pearson <= -0.2, 
                                        "red", "black"))) %>%
mutate(spearman=cell_spec(spearman, color=ifelse(spearman >= 0.2 | pearson <= -0.2, 
                                        "red", "black"))) %>%
headerKable()
```

#### Correlation plots  
  
```{r}

new_cor <- melt(corGeneClass, measure=c("pearson", "spearman"))

 ggplot(melt(corGeneClass, measure=c("pearson", "spearman")), 
         aes(x=variable, y=value)) + geom_boxplot() + ylim(-1,1) + xlab("type") +
    ylab("correlation") + geom_hline(yintercept=0, linetype="dashed", color="red")
```

#### Correlation tables  
 
```{r}

corGeneClass %>% 
  mutate(pearson=cell_spec(pearson, color=ifelse(pearson >= 0.2 | pearson <= -0.2, 
                                        "red", "black"))) %>%
  mutate(spearman=cell_spec(spearman, color=ifelse(spearman >= 0.2 | pearson<= -0.2, 
                                        "red", "black"))) %>%
headerKable()
```


### Old  
  
#### Sample size  

```{r}

countSampleSizes(all_DT[!duplicated(sid) & age=="old"], "visit", 
                 c("case", "stimulation", "age")) %>%
  headerKable(labels=c("case", "stimulation", "age"), levels=c(3,4,2))
```

```{r}

  corAllClass <- all_DT[age=="old",.(pearson=signif(cor(value, 
                                                         difference_trunc), 3),
                           spearman=signif(cor(value, difference_trunc,
                                               method="spearman"), 3)), by=visit]
  
  corGeneClass <- all_DT[age=="old",.(pearson=signif(cor(value, 
                                                          difference_trunc), 3),
                            spearman=signif(cor(value, difference_trunc,
                                                method="spearman"), 3)),
                by=.(gene, visit)]
```
  
#### Correlation: all genes together  
  
```{r}
corAllClass %>%
mutate(pearson=cell_spec(pearson, color=ifelse(pearson >= 0.2 | pearson <= -0.2, 
                                        "red", "black"))) %>%
mutate(spearman=cell_spec(spearman, color=ifelse(spearman >= 0.2 | pearson <= -0.2, 
                                        "red", "black"))) %>%
headerKable()
```

#### Correlation plots  
  
```{r}

new_cor <- melt(corGeneClass, measure=c("pearson", "spearman"))

 ggplot(melt(corGeneClass, measure=c("pearson", "spearman")), 
         aes(x=variable, y=value)) + geom_boxplot() + ylim(-1,1) + xlab("type") +
    ylab("correlation") + facet_wrap(~visit) + 
   geom_hline(yintercept=0, linetype="dashed", color="red")
```

#### Correlation tables  
 
```{r}

cor_tab <- dcast(corGeneClass, gene~visit, 
                        value.var=c("pearson", "spearman"))

cor_tab %>% 
  mutate(pearson_M0=cell_spec(pearson_M0, color=ifelse(pearson_M0 >= 0.2 | 
                                        pearson_M0 <= -0.2, "red", "black"))) %>%
  mutate(pearson_M3=cell_spec(pearson_M3, color=ifelse(pearson_M3 >= 0.2 | 
                                        pearson_M3 <= -0.2, "red", "black"))) %>%
  mutate(spearman_M0=cell_spec(spearman_M0, color=ifelse(spearman_M0 >= 0.2 | 
                                        spearman_M0 <= -0.2, "red", "black"))) %>%
  mutate(spearman_M3=cell_spec(spearman_M3, color=ifelse(spearman_M3 >= 0.2 | 
                                          spearman_M3 <= -0.2, "red", "black"))) %>%
  
headerKable()
```
