<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />


<meta name="author" content="Carl Murie and Raphael Gottardo" />

<meta name="date" content="2019-03-20" />

<title>MAL067: RTS,S malaria vaccine study</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/lumen.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>


</head>

<body>

<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #ffffff;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>

<div class="container-fluid main-container">

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  background: white;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open')
  });
});
</script>

<!-- code folding -->





<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">mal067project</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/jdblischak/workflowr">
    <span class="fa fa-github"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">MAL067: RTS,S malaria vaccine study</h1>
<h4 class="author"><em>Carl Murie and Raphael Gottardo</em></h4>
<h4 class="date"><em>March 20, 2019</em></h4>

</div>


<p><strong>Last updated:</strong> 2019-03-20</p>
<strong>workflowr checks:</strong> <small>(Click a bullet for more information)</small>
<ul>
<li>
<p><details> <summary> <strong style="color:blue;">✔</strong> <strong>R Markdown file:</strong> up-to-date </summary></p>
<p>Great! Since the R Markdown file has been committed to the Git repository, you know the exact version of the code that produced these results.</p>
</details>
</li>
<li>
<p><details> <summary> <strong style="color:blue;">✔</strong> <strong>Environment:</strong> empty </summary></p>
<p>Great job! The global environment was empty. Objects defined in the global environment can affect the analysis in your R Markdown file in unknown ways. For reproduciblity it’s best to always run the code in an empty environment.</p>
</details>
</li>
<li>
<p><details> <summary> <strong style="color:blue;">✔</strong> <strong>Seed:</strong> <code>set.seed(20181108)</code> </summary></p>
<p>The command <code>set.seed(20181108)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</details>
</li>
<li>
<p><details> <summary> <strong style="color:blue;">✔</strong> <strong>Session information:</strong> recorded </summary></p>
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</details>
</li>
<li>
<p><details> <summary> <strong style="color:blue;">✔</strong> <strong>Repository version:</strong> 78199ec </summary></p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility. The version displayed above was the version of the Git repository at the time these results were generated. <br><br> Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
<pre><code>
Ignored files:
    Ignored:    .Rhistory

Untracked files:
    Untracked:  analysis/children/
    Untracked:  analysis/experimental_design.Rmd
    Untracked:  analysis/exploratory_Enriched_Monocytes.Rmd
    Untracked:  analysis/exploratory_ICS_CD4_CSP.Rmd
    Untracked:  analysis/exploratory_ICS_CD4_HBS.Rmd
    Untracked:  analysis/exploratory_ICS_CD8_CSP.Rmd
    Untracked:  analysis/exploratory_ICS_CD8_HBS.Rmd
    Untracked:  analysis/exploratory_ICS_Mag.Rmd
    Untracked:  analysis/exploratory_ML_ratio.Rmd
    Untracked:  analysis/exploratory_antibody.Rmd
    Untracked:  analysis/exploratory_cevac_rna.Rmd
    Untracked:  analysis/exploratory_cytokine_rna.Rmd
    Untracked:  analysis/exploratory_iavi_rna.Rmd
    Untracked:  analysis/exploratory_monocyte_derosa.Rmd
    Untracked:  analysis/primary_interaction_M3_CSP.Rmd
    Untracked:  analysis/primary_interaction_M3_HBS.Rmd
    Untracked:  analysis/primary_main_M0_CSP.Rmd
    Untracked:  analysis/primary_main_M0_DMSO.Rmd
    Untracked:  analysis/primary_main_M0_HBS.Rmd
    Untracked:  analysis/primary_main_M3_CSP.Rmd
    Untracked:  analysis/primary_main_M3_DMSO.Rmd
    Untracked:  analysis/primary_main_M3_HBS.Rmd
    Untracked:  analysis/secondary_DMSO_time_disease.Rmd
    Untracked:  analysis/secondary_faust.Rmd
    Untracked:  analysis/secondary_vaccine_disease_interaction.Rmd
    Untracked:  code/mal_utilities.R
    Untracked:  cor_table.RDS
    Untracked:  data/170830-RTSS case control phenotyping.txt
    Untracked:  data/BTM_for_GSEA_20131008.gmt
    Untracked:  data/CD4_CSP_PFS.rds
    Untracked:  data/CD4_HBS_PFS.rds
    Untracked:  data/CD8_CSP_PFS.rds
    Untracked:  data/CD8_HBS_PFS.rds
    Untracked:  data/PRIMARY_CD4_MAGNITUDE.rds
    Untracked:  data/faustOutputAllRtss_v02.csv
    Untracked:  data/ics_extra_vars_20180515.csv
    Untracked:  data/m067_seattle_data.RData
    Untracked:  data/phenoLineagePctLive.txt
    Untracked:  docs/figure/
    Untracked:  output/dmso_M0_both_disease.csv
    Untracked:  output/dmso_M0_old_disease.csv
    Untracked:  output/dmso_M3_both_disease.csv
    Untracked:  output/dmso_M3_old_disease.csv
    Untracked:  output/dmso_M3_young_disease.csv

</code></pre>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes. </details>
</li>
</ul>
<details> <summary> <small><strong>Expand here to see past versions:</strong></small> </summary>
<ul>
<table style="border-collapse:separate; border-spacing:5px;">
<thead>
<tr>
<th style="text-align:left;">
File
</th>
<th style="text-align:left;">
Version
</th>
<th style="text-align:left;">
Author
</th>
<th style="text-align:left;">
Date
</th>
<th style="text-align:left;">
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
Rmd
</td>
<td style="text-align:left;">
78199ec
</td>
<td style="text-align:left;">
carlmurie
</td>
<td style="text-align:left;">
2019-03-20
</td>
<td style="text-align:left;">
wflow_publish(all = TRUE)
</td>
</tr>
<tr>
<td style="text-align:left;">
Rmd
</td>
<td style="text-align:left;">
00f25de
</td>
<td style="text-align:left;">
carlmurie
</td>
<td style="text-align:left;">
2018-11-08
</td>
<td style="text-align:left;">
Start workflowr project.
</td>
</tr>
</tbody>
</table>
</ul>
<p></details></p>
<hr />
<div id="overview" class="section level1">
<h1><span class="header-section-number">1</span> Overview</h1>
<p>This project aims to understand characteristics of the RTS,S vaccine and identify collerates of vaccine efficacy (or more generally speaking clinical disease outcome) based on immunological assay data generated from MAL067.</p>
<p><strong>Primary analysis</strong></p>
<ol style="list-style-type: decimal">
<li>Main effects: by stimulation and timepoint
<ul>
<li>differences in vaccine (rts,s - comparator)</li>
<li>differences in disease (case - control)</li>
<li>look at disease results for all significant gene-sets from vaccine comparison</li>
</ul></li>
<li>Interaction effects: by timepoint
<ul>
<li>differences in vaccine (rts,s - comparator) vs stimulation (stim - dmso)</li>
<li>differences in disease (case - control) vs stimulation (stim - dmso)</li>
<li>look at disease results for all significant gene-sets from vaccine comparison</li>
</ul></li>
</ol>
</div>
<div id="experimental-design" class="section level1">
<h1><span class="header-section-number">2</span> Experimental Design</h1>
<div id="design" class="section level3">
<h3><span class="header-section-number">2.0.1</span> <a href="experimental_design.html">Design</a></h3>
</div>
</div>
<div id="primary-analysis" class="section level1">
<h1><span class="header-section-number">3</span> Primary Analysis</h1>
<div id="gsea" class="section level2">
<h2><span class="header-section-number">3.1</span> GSEA</h2>
<div id="main-effects-vaccine-and-disease-for-each-stimulation" class="section level3">
<h3><span class="header-section-number">3.1.1</span> Main effects: vaccine and disease for each stimulation</h3>
<div id="dmso-m0" class="section level4">
<h4><span class="header-section-number">3.1.1.1</span> <a href="primary_main_M0_DMSO.html">DMSO: M0</a></h4>
</div>
<div id="dmso-m3" class="section level4">
<h4><span class="header-section-number">3.1.1.2</span> <a href="primary_main_M3_DMSO.html">DMSO: M3</a></h4>
</div>
<div id="csp-m0" class="section level4">
<h4><span class="header-section-number">3.1.1.3</span> <a href="primary_main_M0_CSP.html">CSP: M0</a></h4>
</div>
<div id="csp-m3" class="section level4">
<h4><span class="header-section-number">3.1.1.4</span> <a href="primary_main_M3_CSP.html">CSP: M3</a></h4>
</div>
<div id="hbs-m0" class="section level4">
<h4><span class="header-section-number">3.1.1.5</span> <a href="primary_main_M0_HBS.html">HBS: M0</a></h4>
</div>
<div id="hbs-m3" class="section level4">
<h4><span class="header-section-number">3.1.1.6</span> <a href="primary_main_M3_HBS.html">HBS: M3</a></h4>
</div>
</div>
<div id="interaction-effects-stimulation-stim---dmso-vs-diseasevaccine" class="section level3">
<h3><span class="header-section-number">3.1.2</span> Interaction effects: stimulation (stim - dmso) vs disease/vaccine</h3>
<div id="csp-m3-1" class="section level4">
<h4><span class="header-section-number">3.1.2.1</span> <a href="primary_interaction_M3_CSP.html">CSP: M3</a></h4>
</div>
<div id="hbs-m3-1" class="section level4">
<h4><span class="header-section-number">3.1.2.2</span> <a href="primary_interaction_M3_HBS.html">HBS: M3</a></h4>
</div>
</div>
<div id="secondary" class="section level3">
<h3><span class="header-section-number">3.1.3</span> Secondary</h3>
<div id="gsea-timedisease-interaction-group" class="section level4">
<h4><span class="header-section-number">3.1.3.1</span> <a href="secondary_DMSO_time_disease.html">GSEA: time*disease interaction group</a></h4>
</div>
<div id="gsea-vaccinedisease-interaction-for-time-point-m3" class="section level4">
<h4><span class="header-section-number">3.1.3.2</span> <a href="secondary_vaccine_disease_interaction.html">GSEA: vaccine*disease interaction for time-point M3</a></h4>
</div>
</div>
<div id="exploratory" class="section level3">
<h3><span class="header-section-number">3.1.4</span> Exploratory</h3>
<div id="faust-moncyte-analysis" class="section level4">
<h4><span class="header-section-number">3.1.4.1</span> <a href="secondary_faust.html">Faust moncyte analysis</a></h4>
</div>
<div id="enriched-moncytes-ii" class="section level4">
<h4><span class="header-section-number">3.1.4.2</span> <a href="exploratory_Enriched_Monocytes.html">enriched moncytes II</a></h4>
</div>
<div id="monocytelymphocyte-ratio-disease-prediction" class="section level4">
<h4><span class="header-section-number">3.1.4.3</span> <a href="exploratory_ML_ratio.html">monocyte/lymphocyte ratio disease prediction</a></h4>
</div>
<div id="cytokine-stimdmso-ratios-disease-prediction" class="section level4">
<h4><span class="header-section-number">3.1.4.4</span> <a href="exploratory_cytokine_rna.html">cytokine stim/DMSO ratios disease prediction</a></h4>
</div>
<div id="iavi-isotype-disease-prediction" class="section level4">
<h4><span class="header-section-number">3.1.4.5</span> <a href="exploratory_iavi_rna.html">IAVI isotype disease prediction</a></h4>
</div>
<div id="cevac-disease-prediction" class="section level4">
<h4><span class="header-section-number">3.1.4.6</span> <a href="exploratory_cevac_rna.html">CEVAC disease prediction</a></h4>
</div>
<div id="antibody-disease-prediction" class="section level4">
<h4><span class="header-section-number">3.1.4.7</span> <a href="exploratory_antibody.html">antibody disease prediction</a></h4>
</div>
<div id="derosa-cell-subset-frequencies-disease-and-vaccine-effects" class="section level4">
<h4><span class="header-section-number">3.1.4.8</span> <a href="exploratory_monocyte_derosa.html">DeRosa cell subset frequencies disease and vaccine effects</a></h4>
</div>
<div id="ics-cd4-csp-correlation-with-rnaseq-gene-expression" class="section level4">
<h4><span class="header-section-number">3.1.4.9</span> <a href="exploratory_ICS_CD4_CSP.html">ICS CD4 CSP: correlation with RNASeq gene expression</a></h4>
</div>
<div id="ics-cd4-hbs-correlation-with-rnaseq-gene-expression" class="section level4">
<h4><span class="header-section-number">3.1.4.10</span> <a href="exploratory_ICS_CD4_HBS.html">ICS CD4 HBS: correlation with RNASeq gene expression</a></h4>
</div>
<div id="ics-cd8-csp-correlation-with-rnaseq-gene-expression" class="section level4">
<h4><span class="header-section-number">3.1.4.11</span> <a href="exploratory_ICS_CD8_CSP.html">ICS CD8 CSP: correlation with RNASeq gene expression</a></h4>
</div>
<div id="ics-cd8-hbs-correlation-with-rnaseq-gene-expression" class="section level4">
<h4><span class="header-section-number">3.1.4.12</span> <a href="exploratory_ICS_CD8_HBS.html">ICS CD8 HBS: correlation with RNASeq gene expression</a></h4>
</div>
<div id="ics-cd4-magnitude-difference-truncated-correlation-with-rnaseq-gene-expression" class="section level4">
<h4><span class="header-section-number">3.1.4.13</span> <a href="exploratory_ICS_Mag.html">ICS CD4 magnitude difference (truncated): correlation with RNASeq gene expression</a></h4>
</div>
</div>
</div>
<div id="session-information" class="section level2">
<h2><span class="header-section-number">3.2</span> Session information</h2>
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 3.5.0 (2018-04-23)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Ubuntu 14.04.5 LTS

Matrix products: default
BLAS/LAPACK: /app/easybuild/software/OpenBLAS/0.2.18-GCC-5.4.0-2.26-LAPACK-3.6.1/lib/libopenblas_prescottp-r0.2.18.so

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              
 [3] LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8    
 [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8   
 [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                 
 [9] LC_ADDRESS=C               LC_TELEPHONE=C            
[11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

loaded via a namespace (and not attached):
 [1] workflowr_1.1.1   Rcpp_1.0.0        digest_0.6.18    
 [4] rprojroot_1.3-2   R.methodsS3_1.7.1 backports_1.1.2  
 [7] magrittr_1.5      git2r_0.24.0      evaluate_0.13    
[10] stringi_1.3.1     whisker_0.3-2     R.oo_1.22.0      
[13] R.utils_2.7.0     rmarkdown_1.11    tools_3.5.0      
[16] stringr_1.4.0     xfun_0.5          yaml_2.2.0       
[19] compiler_3.5.0    htmltools_0.3.6   knitr_1.21       </code></pre>
</div>
</div>

<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>

<hr>
<p>
  This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a>
  analysis was created with
  <a href="https://github.com/jdblischak/workflowr">workflowr</a> 1.1.1
</p>
<hr>



</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
