<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Carl Murie" />

<meta name="date" content="2020-02-07" />

<title>Manuscript: M3 DMSO</title>

<script src="site_libs/jquery-1.12.4/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/lumen.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<script src="site_libs/kePrint-0.0.1/kePrint.js"></script>
<script src="site_libs/htmlwidgets-1.5.1/htmlwidgets.js"></script>
<link href="site_libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet" />
<script src="site_libs/datatables-binding-0.11/datatables.js"></script>
<link href="site_libs/dt-core-1.10.19/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="site_libs/dt-core-1.10.19/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="site_libs/dt-core-1.10.19/js/jquery.dataTables.min.js"></script>
<link href="site_libs/crosstalk-1.0.0/css/crosstalk.css" rel="stylesheet" />
<script src="site_libs/crosstalk-1.0.0/js/crosstalk.min.js"></script>
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>




<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #ffffff;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  background: white;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row-fluid">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">mal067project</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/jdblischak/workflowr">
    <span class="fa fa-github"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Manuscript: M3 DMSO</h1>
<h4 class="author">Carl Murie</h4>
<h4 class="date">February 07, 2020</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span> workflowr <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2020-02-07
</p>
<p>
<strong>Checks:</strong> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7 <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>mal067project/</code> <span class="glyphicon glyphicon-question-sign" aria-hidden="true" title="This is the local directory in which the code in this file was executed."> </span>
</p>
<p>
This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a> analysis was created with <a
  href="https://github.com/jdblischak/workflowr">workflowr</a> (version 1.6.0). The <em>Checks</em> tab describes the reproducibility checks that were applied when the results were created. The <em>Past versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date </a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate" class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git repository, you know the exact version of the code that produced these results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the global environment can affect the analysis in your R Markdown file in unknown ways. For reproduciblity it’s best to always run the code in an empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20181108code"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Seed:</strong> <code>set.seed(20181108)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20181108code" class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20181108)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Session information:</strong> recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be confident that you successfully produced the results during this run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongdd65764"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Repository version:</strong> dd65764 </a>
</p>
</div>
<div id="strongRepositoryversionstrongdd65764" class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility. The version displayed above was the version of the Git repository at the time these results were generated. <br><br> Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .Rhistory
    Ignored:    analysis/.Rhistory

Untracked files:
    Untracked:  Faust_monocyte_boxplot.pdf
    Untracked:  analysis/DeRosaMonocyteProcessing.Rmd
    Untracked:  analysis/children/gsea_plot_delta_genes.Rmd
    Untracked:  analysis/children/stim_GSEA_interaction_case_effect.Rmd
    Untracked:  analysis/children/stim_GSEA_interaction_mono_effect.Rmd
    Untracked:  analysis/children/stim_GSEA_main_mono_effect.Rmd
    Untracked:  analysis/primary_interaction_mono_AMA.Rmd
    Untracked:  analysis/primary_interaction_mono_CSP.Rmd
    Untracked:  analysis/primary_interaction_mono_HBS.Rmd
    Untracked:  analysis/primary_main_M0_AMA.Rmd
    Untracked:  analysis/primary_main_M0_DMSO_bak.Rmd
    Untracked:  analysis/primary_main_mono_AMA.Rmd
    Untracked:  analysis/primary_main_mono_CSP.Rmd
    Untracked:  analysis/primary_main_mono_DMSO.Rmd
    Untracked:  analysis/primary_main_mono_HBS.Rmd
    Untracked:  analysis/process_deltas.Rmd
    Untracked:  analysis/secondary_covariates.Rmd
    Untracked:  code/ARDAP1_transcript.Rmd
    Untracked:  code/mal_utilities.R
    Untracked:  cor_table.RDS
    Untracked:  data/.~lock.170830-RTSS case control phenotyping.txt#
    Untracked:  data/170830-RTSS case control phenotyping.txt
    Untracked:  data/BTM_for_GSEA_20131008.gmt
    Untracked:  data/CD4_CSP_PFS.rds
    Untracked:  data/CD4_HBS_PFS.rds
    Untracked:  data/CD8_CSP_PFS.rds
    Untracked:  data/CD8_HBS_PFS.rds
    Untracked:  data/CiberSort_RTSS.csv
    Untracked:  data/DeRosaMonocyteFrequencies.csv
    Untracked:  data/ICS_Primary_MAL067.pptx
    Untracked:  data/PRIMARY_CD4_MAGNITUDE.rds
    Untracked:  data/Pulendran_GSEA_disease.csv
    Untracked:  data/RTSSPilotRNA-seqSAP_v4.pdf
    Untracked:  data/Vahey_GSEA_disease.csv
    Untracked:  data/btm_annotation_table.tsv
    Untracked:  data/btm_annotation_table_LNC.csv
    Untracked:  data/btm_annotation_table_LNC.ods
    Untracked:  data/faustOutputAllRtss_v02.csv
    Untracked:  data/ics_extra_vars_20180515.csv
    Untracked:  data/long_dis_DupCor.csv
    Untracked:  data/long_vac_DupCor.csv
    Untracked:  data/m067_seattle_data.RData
    Untracked:  data/phenoLineagePctLive.txt
    Untracked:  data/visit_DupCor.csv
    Untracked:  output/DMSO_M3_vaccine_disease_interaction.csv
    Untracked:  output/DMSO_M3_vaccine_disease_interaction_DEG.csv
    Untracked:  output/Fig2A_vaccine_both_dmso_M3.csv
    Untracked:  output/Fig2B_M3-M0_rtss.csv
    Untracked:  output/Fig3A_M3_case_control.csv
    Untracked:  output/Fig3B_M3-M0_rtss.csv
    Untracked:  output/GSEA_M0_ama1_vac_dis.csv
    Untracked:  output/GSEA_M0_csp_vac_dis.csv
    Untracked:  output/GSEA_M0_hbs_vac_dis.csv
    Untracked:  output/GSEA_M3_ama1_vac_dis.csv
    Untracked:  output/GSEA_M3_ama1_vac_dis_case.csv
    Untracked:  output/GSEA_M3_csp_vac_dis.csv
    Untracked:  output/GSEA_M3_hbs_vac_dis.csv
    Untracked:  output/M3-M0_both.csv
    Untracked:  output/M3-M0_by_vaccine.csv
    Untracked:  output/M3-M0_comp.csv
    Untracked:  output/M3-M0_disease_mal.csv
    Untracked:  output/M3-M0_disease_mal_comp.csv
    Untracked:  output/M3-M0_disease_mal_rtss.csv
    Untracked:  output/M3-M0_disease_nomal_comp.csv
    Untracked:  output/M3-M0_disease_nomal_rtss.csv
    Untracked:  output/M3-M0_rtss.csv
    Untracked:  output/M3_dmso_vaccine.csv
    Untracked:  output/cell_dist_mediods
    Untracked:  output/cibersort_cells.csv
    Untracked:  output/cibersort_cells.pdf
    Untracked:  output/disease_both_ama1_M0.csv
    Untracked:  output/disease_both_dmso_M0.csv
    Untracked:  output/disease_both_dmso_M3.csv
    Untracked:  output/disease_comp_dmso_M3.csv
    Untracked:  output/disease_ds_both_ama1_M0.csv
    Untracked:  output/disease_ds_both_dmso_M0.csv
    Untracked:  output/disease_ds_both_dmso_M3.csv
    Untracked:  output/disease_rtss_dmso_M3.csv
    Untracked:  output/dmso_M0_both_disease.csv
    Untracked:  output/dmso_M0_old_disease.csv
    Untracked:  output/dmso_M3_both_disease.csv
    Untracked:  output/dmso_M3_old_disease.csv
    Untracked:  output/dmso_M3_young_disease.csv
    Untracked:  output/figures/
    Untracked:  output/manuscript/
    Untracked:  output/sig_dis_genes_david.csv
    Untracked:  output/stim_deltas.rds
    Untracked:  output/stim_deltas_cpm_unlogged.rds
    Untracked:  output/vaccine_both_ama1_M0.csv
    Untracked:  output/vaccine_both_dmso_M0.csv
    Untracked:  output/vaccine_both_dmso_M3.csv
    Untracked:  secondary_faust.html
    Untracked:  test.pdf

Unstaged changes:
    Modified:   analysis/analysis_DMSO_M3_GSEA.Rmd
    Modified:   analysis/analysis_GSEA_M3_stimulations.Rmd
    Modified:   analysis/children/stim_GSEA_interaction_effect.Rmd
    Modified:   analysis/children/stim_GSEA_main_effect.Rmd
    Modified:   analysis/combine_GSEA.Rmd
    Modified:   analysis/experimental_design.Rmd
    Modified:   analysis/exploratory_ML_ratio.Rmd
    Modified:   analysis/exploratory_antibody.Rmd
    Modified:   analysis/exploratory_heno_gene_expression_disease_protection.Rmd
    Modified:   analysis/exploratory_monocyte_derosa.Rmd
    Modified:   analysis/index.Rmd
    Modified:   analysis/primary_Bagamoyo_GSEA.Rmd
    Modified:   analysis/primary_interaction_M3_AMA.Rmd
    Modified:   analysis/primary_main_M0_DMSO.Rmd
    Modified:   analysis/primary_main_M3_DMSO.Rmd
    Deleted:    analysis/secondary_disease_all_samples.Rmd
    Modified:   analysis/secondary_faust.Rmd

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the R Markdown and HTML files. If you’ve configured a remote Git repository (see <code>?wflow_git_remote</code>), click on the hyperlinks in the table below to view them.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
dd65764
</td>
<td>
carlmurie
</td>
<td>
2020-02-07
</td>
<td>
wflow_publish(“analysis/manuscript*.Rmd“)
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<div id="gsea-dmso-disease-main-effects" class="section level1 tabset tabset-fade tabset-pills">
<h1><span class="header-section-number">1</span> GSEA DMSO: disease main effects</h1>
<p><strong>disease linear model:</strong> ~plate + total_reads + age + case<br />
<strong>disease filtering :</strong> M3</p>
<table class="table table-striped table-hover table-condensed" style="width: auto !important; ">
<caption>
disease sample size
</caption>
<thead>
<tr>
<th style="border-bottom:hidden" colspan="1">
</th>
<th style="border-bottom:hidden" colspan="1">
</th>
<th style="border-bottom:hidden; padding-bottom:0; padding-left:3px;padding-right:3px;text-align: center; " colspan="2">
<div style="border-bottom: 1px solid #ddd; padding-bottom: 5px; ">
vaccine
</div>
</th>
<th style="border-bottom:hidden; padding-bottom:0; padding-left:3px;padding-right:3px;text-align: center; " colspan="2">
<div style="border-bottom: 1px solid #ddd; padding-bottom: 5px; ">
malaria
</div>
</th>
<th style="border-bottom:hidden; padding-bottom:0; padding-left:3px;padding-right:3px;text-align: center; " colspan="2">
<div style="border-bottom: 1px solid #ddd; padding-bottom: 5px; ">
age
</div>
</th>
<th style="border-bottom:hidden; padding-bottom:0; padding-left:3px;padding-right:3px;text-align: center; " colspan="2">
<div style="border-bottom: 1px solid #ddd; padding-bottom: 5px; ">
visit
</div>
</th>
<th style="border-bottom:hidden; padding-bottom:0; padding-left:3px;padding-right:3px;text-align: center; " colspan="2">
<div style="border-bottom: 1px solid #ddd; padding-bottom: 5px; ">
case
</div>
</th>
</tr>
<tr>
<th style="text-align:left;">
</th>
<th style="text-align:center;">
total
</th>
<th style="text-align:center;">
comparator
</th>
<th style="text-align:center;">
rtss
</th>
<th style="text-align:center;">
X0
</th>
<th style="text-align:center;">
X1
</th>
<th style="text-align:center;">
young
</th>
<th style="text-align:center;">
old
</th>
<th style="text-align:center;">
M0
</th>
<th style="text-align:center;">
M3
</th>
<th style="text-align:center;">
control
</th>
<th style="text-align:center;">
case
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
dmso
</td>
<td style="text-align:center;">
193
</td>
<td style="text-align:center;">
70
</td>
<td style="text-align:center;">
123
</td>
<td style="text-align:center;">
141
</td>
<td style="text-align:center;">
52
</td>
<td style="text-align:center;">
74
</td>
<td style="text-align:center;">
119
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
193
</td>
<td style="text-align:center;">
140
</td>
<td style="text-align:center;">
53
</td>
</tr>
</tbody>
</table>
<div id="disease-m3-malaria---no-malaria-rtss-only" class="section level2">
<h2><span class="header-section-number">1.1</span> disease M3 (malaria - no malaria): RTS,S only</h2>
<div style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:600px; ">
<table class="table table-striped table-hover table-condensed" style="width: auto !important; ">
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
geneset
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
NGenes
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Direction
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
PValue
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
FDR
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
respiratory electron transport chain (mitochondrion) (M238)
</td>
<td style="text-align:left;">
17
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0001453
</td>
<td style="text-align:left;">
0.0385
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="disease-m3-malaria---no-malaria-comparator-only" class="section level2">
<h2><span class="header-section-number">1.2</span> disease M3 (malaria - no malaria): comparator only</h2>
<div style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:600px; ">
<table class="table table-striped table-hover table-condensed" style="width: auto !important; ">
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
geneset
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
NGenes
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Direction
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
PValue
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
FDR
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
enriched in monocytes (II) (M11.0)
</td>
<td style="text-align:left;">
146
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0000000
</td>
<td style="text-align:left;">
0.0000005
</td>
</tr>
<tr>
<td style="text-align:left;">
Monocyte surface signature (S4)
</td>
<td style="text-align:left;">
69
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0000000
</td>
<td style="text-align:left;">
0.0000026
</td>
</tr>
<tr>
<td style="text-align:left;">
chemokines and inflammatory molecules in myeloid cells (M86.0)
</td>
<td style="text-align:left;">
17
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0000001
</td>
<td style="text-align:left;">
0.0000060
</td>
</tr>
<tr>
<td style="text-align:left;">
chemokine cluster (I) (M27.0)
</td>
<td style="text-align:left;">
20
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0000001
</td>
<td style="text-align:left;">
0.0000060
</td>
</tr>
<tr>
<td style="text-align:left;">
enriched in B cells (I) (M47.0)
</td>
<td style="text-align:left;">
39
</td>
<td style="text-align:left;">
Up
</td>
<td style="text-align:left;">
0.0000001
</td>
<td style="text-align:left;">
0.0000079
</td>
</tr>
<tr>
<td style="text-align:left;">
AP-1 transcription factor network (M20)
</td>
<td style="text-align:left;">
13
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0000002
</td>
<td style="text-align:left;">
0.0000106
</td>
</tr>
<tr>
<td style="text-align:left;">
enriched in activated dendritic cells/monocytes (M64)
</td>
<td style="text-align:left;">
17
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0000006
</td>
<td style="text-align:left;">
0.0000218
</td>
</tr>
<tr>
<td style="text-align:left;">
antigen processing and presentation (M200)
</td>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
Up
</td>
<td style="text-align:left;">
0.0000010
</td>
<td style="text-align:left;">
0.0000320
</td>
</tr>
<tr>
<td style="text-align:left;">
proinflammatory cytokines and chemokines (M29)
</td>
<td style="text-align:left;">
9
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0000035
</td>
<td style="text-align:left;">
0.0001028
</td>
</tr>
<tr>
<td style="text-align:left;">
enriched in neutrophils (I) (M37.1)
</td>
<td style="text-align:left;">
29
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0000055
</td>
<td style="text-align:left;">
0.0001460
</td>
</tr>
<tr>
<td style="text-align:left;">
suppression of MAPK signaling (M56)
</td>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0000067
</td>
<td style="text-align:left;">
0.0001490
</td>
</tr>
<tr>
<td style="text-align:left;">
TBA (M105)
</td>
<td style="text-align:left;">
11
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0000067
</td>
<td style="text-align:left;">
0.0001490
</td>
</tr>
<tr>
<td style="text-align:left;">
enriched in B cells (II) (M47.1)
</td>
<td style="text-align:left;">
32
</td>
<td style="text-align:left;">
Up
</td>
<td style="text-align:left;">
0.0000083
</td>
<td style="text-align:left;">
0.0001693
</td>
</tr>
<tr>
<td style="text-align:left;">
enriched in B cells (VI) (M69)
</td>
<td style="text-align:left;">
18
</td>
<td style="text-align:left;">
Up
</td>
<td style="text-align:left;">
0.0000113
</td>
<td style="text-align:left;">
0.0001850
</td>
</tr>
<tr>
<td style="text-align:left;">
targets of FOSL1/2 (M0)
</td>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0000114
</td>
<td style="text-align:left;">
0.0001850
</td>
</tr>
<tr>
<td style="text-align:left;">
enriched in naive and memory B cells (M83)
</td>
<td style="text-align:left;">
9
</td>
<td style="text-align:left;">
Up
</td>
<td style="text-align:left;">
0.0000114
</td>
<td style="text-align:left;">
0.0001850
</td>
</tr>
<tr>
<td style="text-align:left;">
NK cell surface signature (S1)
</td>
<td style="text-align:left;">
24
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0000119
</td>
<td style="text-align:left;">
0.0001850
</td>
</tr>
<tr>
<td style="text-align:left;">
TBA (M193)
</td>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0000151
</td>
<td style="text-align:left;">
0.0002223
</td>
</tr>
<tr>
<td style="text-align:left;">
immune activation - generic cluster (M37.0)
</td>
<td style="text-align:left;">
210
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0000393
</td>
<td style="text-align:left;">
0.0005482
</td>
</tr>
<tr>
<td style="text-align:left;">
cell cycle and growth arrest (M31)
</td>
<td style="text-align:left;">
11
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0000589
</td>
<td style="text-align:left;">
0.0007570
</td>
</tr>
<tr>
<td style="text-align:left;">
blood coagulation (M11.1)
</td>
<td style="text-align:left;">
22
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0000600
</td>
<td style="text-align:left;">
0.0007570
</td>
</tr>
<tr>
<td style="text-align:left;">
enriched in NK cells (I) (M7.2)
</td>
<td style="text-align:left;">
45
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0000697
</td>
<td style="text-align:left;">
0.0008207
</td>
</tr>
<tr>
<td style="text-align:left;">
extracellular matrix, complement (M140)
</td>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0000758
</td>
<td style="text-align:left;">
0.0008207
</td>
</tr>
<tr>
<td style="text-align:left;">
enriched in myeloid cells and monocytes (M81)
</td>
<td style="text-align:left;">
23
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0000764
</td>
<td style="text-align:left;">
0.0008207
</td>
</tr>
<tr>
<td style="text-align:left;">
TBA (M188)
</td>
<td style="text-align:left;">
9
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0000774
</td>
<td style="text-align:left;">
0.0008207
</td>
</tr>
<tr>
<td style="text-align:left;">
enriched in dendritic cells (M168)
</td>
<td style="text-align:left;">
12
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0000821
</td>
<td style="text-align:left;">
0.0008370
</td>
</tr>
<tr>
<td style="text-align:left;">
enriched in monocytes (III) (M73)
</td>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0001240
</td>
<td style="text-align:left;">
0.0012170
</td>
</tr>
<tr>
<td style="text-align:left;">
myeloid cell enriched receptors and transporters (M4.3)
</td>
<td style="text-align:left;">
26
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0001748
</td>
<td style="text-align:left;">
0.0016250
</td>
</tr>
<tr>
<td style="text-align:left;">
Resting dendritic cell surface signature (S10)
</td>
<td style="text-align:left;">
43
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0001778
</td>
<td style="text-align:left;">
0.0016250
</td>
</tr>
<tr>
<td style="text-align:left;">
putative targets of PAX3 (M89.0)
</td>
<td style="text-align:left;">
12
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0001898
</td>
<td style="text-align:left;">
0.0016770
</td>
</tr>
<tr>
<td style="text-align:left;">
enriched in monocytes (IV) (M118.0)
</td>
<td style="text-align:left;">
48
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0002396
</td>
<td style="text-align:left;">
0.0020480
</td>
</tr>
<tr>
<td style="text-align:left;">
DC surface signature (S5)
</td>
<td style="text-align:left;">
54
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0002807
</td>
<td style="text-align:left;">
0.0023250
</td>
</tr>
<tr>
<td style="text-align:left;">
TLR and inflammatory signaling (M16)
</td>
<td style="text-align:left;">
34
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0003277
</td>
<td style="text-align:left;">
0.0025900
</td>
</tr>
<tr>
<td style="text-align:left;">
cell cycle and transcription (M4.0)
</td>
<td style="text-align:left;">
226
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0003340
</td>
<td style="text-align:left;">
0.0025900
</td>
</tr>
<tr>
<td style="text-align:left;">
spliceosome (M250)
</td>
<td style="text-align:left;">
12
</td>
<td style="text-align:left;">
Up
</td>
<td style="text-align:left;">
0.0003421
</td>
<td style="text-align:left;">
0.0025900
</td>
</tr>
<tr>
<td style="text-align:left;">
chemokine cluster (II) (M27.1)
</td>
<td style="text-align:left;">
13
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0004078
</td>
<td style="text-align:left;">
0.0030020
</td>
</tr>
<tr>
<td style="text-align:left;">
cytoskeleton/actin (SRF transcription targets) (M145.0)
</td>
<td style="text-align:left;">
11
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0004752
</td>
<td style="text-align:left;">
0.0033190
</td>
</tr>
<tr>
<td style="text-align:left;">
G protein coupled receptors cluster (M155)
</td>
<td style="text-align:left;">
9
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0004759
</td>
<td style="text-align:left;">
0.0033190
</td>
</tr>
<tr>
<td style="text-align:left;">
mismatch repair (I) (M22.0)
</td>
<td style="text-align:left;">
25
</td>
<td style="text-align:left;">
Up
</td>
<td style="text-align:left;">
0.0005254
</td>
<td style="text-align:left;">
0.0035700
</td>
</tr>
<tr>
<td style="text-align:left;">
respiratory electron transport chain (mitochondrion) (M216)
</td>
<td style="text-align:left;">
12
</td>
<td style="text-align:left;">
Up
</td>
<td style="text-align:left;">
0.0005542
</td>
<td style="text-align:left;">
0.0036720
</td>
</tr>
<tr>
<td style="text-align:left;">
TBA (M131)
</td>
<td style="text-align:left;">
12
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0005960
</td>
<td style="text-align:left;">
0.0038520
</td>
</tr>
<tr>
<td style="text-align:left;">
translation initiation factor 3 complex (M245)
</td>
<td style="text-align:left;">
11
</td>
<td style="text-align:left;">
Up
</td>
<td style="text-align:left;">
0.0006247
</td>
<td style="text-align:left;">
0.0039420
</td>
</tr>
<tr>
<td style="text-align:left;">
myeloid cell cytokines, metallopeptidases and laminins (M78)
</td>
<td style="text-align:left;">
9
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0006786
</td>
<td style="text-align:left;">
0.0041730
</td>
</tr>
<tr>
<td style="text-align:left;">
enriched in monocytes (surface) (M118.1)
</td>
<td style="text-align:left;">
17
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0006929
</td>
<td style="text-align:left;">
0.0041730
</td>
</tr>
<tr>
<td style="text-align:left;">
enriched in NK cells (II) (M61.0)
</td>
<td style="text-align:left;">
13
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0007197
</td>
<td style="text-align:left;">
0.0042380
</td>
</tr>
<tr>
<td style="text-align:left;">
TBA (M66)
</td>
<td style="text-align:left;">
17
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0009316
</td>
<td style="text-align:left;">
0.0053670
</td>
</tr>
<tr>
<td style="text-align:left;">
cell adhesion (M51)
</td>
<td style="text-align:left;">
8
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0010040
</td>
<td style="text-align:left;">
0.0056590
</td>
</tr>
<tr>
<td style="text-align:left;">
cell movement, Adhesion &amp; Platelet activation (M30)
</td>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0011630
</td>
<td style="text-align:left;">
0.0064190
</td>
</tr>
<tr>
<td style="text-align:left;">
proteasome (M226)
</td>
<td style="text-align:left;">
12
</td>
<td style="text-align:left;">
Up
</td>
<td style="text-align:left;">
0.0013450
</td>
<td style="text-align:left;">
0.0071700
</td>
</tr>
<tr>
<td style="text-align:left;">
transcription regulation in cell development (M49)
</td>
<td style="text-align:left;">
37
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0013570
</td>
<td style="text-align:left;">
0.0071700
</td>
</tr>
<tr>
<td style="text-align:left;">
respiratory electron transport chain (mitochondrion) (M238)
</td>
<td style="text-align:left;">
17
</td>
<td style="text-align:left;">
Up
</td>
<td style="text-align:left;">
0.0013800
</td>
<td style="text-align:left;">
0.0071700
</td>
</tr>
<tr>
<td style="text-align:left;">
enriched in monocytes (I) (M4.15)
</td>
<td style="text-align:left;">
8
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0015340
</td>
<td style="text-align:left;">
0.0078190
</td>
</tr>
<tr>
<td style="text-align:left;">
type I interferon response (M127)
</td>
<td style="text-align:left;">
12
</td>
<td style="text-align:left;">
Up
</td>
<td style="text-align:left;">
0.0015960
</td>
<td style="text-align:left;">
0.0079800
</td>
</tr>
<tr>
<td style="text-align:left;">
TBA (M174)
</td>
<td style="text-align:left;">
22
</td>
<td style="text-align:left;">
Up
</td>
<td style="text-align:left;">
0.0017470
</td>
<td style="text-align:left;">
0.0085730
</td>
</tr>
<tr>
<td style="text-align:left;">
respiratory electron transport chain (mitochondrion) (M219)
</td>
<td style="text-align:left;">
18
</td>
<td style="text-align:left;">
Up
</td>
<td style="text-align:left;">
0.0018290
</td>
<td style="text-align:left;">
0.0087660
</td>
</tr>
<tr>
<td style="text-align:left;">
transmembrane transport (I) (M87)
</td>
<td style="text-align:left;">
19
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0018520
</td>
<td style="text-align:left;">
0.0087660
</td>
</tr>
<tr>
<td style="text-align:left;">
proinflammatory dendritic cell, myeloid cell response (M86.1)
</td>
<td style="text-align:left;">
11
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0020070
</td>
<td style="text-align:left;">
0.0091810
</td>
</tr>
<tr>
<td style="text-align:left;">
enriched in membrane proteins (M124)
</td>
<td style="text-align:left;">
13
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0020100
</td>
<td style="text-align:left;">
0.0091810
</td>
</tr>
<tr>
<td style="text-align:left;">
TBA (M79)
</td>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0020770
</td>
<td style="text-align:left;">
0.0093290
</td>
</tr>
<tr>
<td style="text-align:left;">
extracellular matrix (II) (M2.1)
</td>
<td style="text-align:left;">
18
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0023540
</td>
<td style="text-align:left;">
0.0104000
</td>
</tr>
<tr>
<td style="text-align:left;">
nuclear pore complex (M106.0)
</td>
<td style="text-align:left;">
17
</td>
<td style="text-align:left;">
Up
</td>
<td style="text-align:left;">
0.0025480
</td>
<td style="text-align:left;">
0.0110700
</td>
</tr>
<tr>
<td style="text-align:left;">
viral sensing &amp; immunity; IRF2 targets network (II) (M111.1)
</td>
<td style="text-align:left;">
11
</td>
<td style="text-align:left;">
Up
</td>
<td style="text-align:left;">
0.0028300
</td>
<td style="text-align:left;">
0.0121000
</td>
</tr>
<tr>
<td style="text-align:left;">
cytoskeleton/actin (SRF transcription targets) (M145.1)
</td>
<td style="text-align:left;">
9
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0031690
</td>
<td style="text-align:left;">
0.0133300
</td>
</tr>
<tr>
<td style="text-align:left;">
regulation of signal transduction (M3)
</td>
<td style="text-align:left;">
33
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0036210
</td>
<td style="text-align:left;">
0.0149900
</td>
</tr>
<tr>
<td style="text-align:left;">
cytokines - recepters cluster (M115)
</td>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0037500
</td>
<td style="text-align:left;">
0.0152900
</td>
</tr>
<tr>
<td style="text-align:left;">
enriched in antigen presentation (II) (M95.0)
</td>
<td style="text-align:left;">
21
</td>
<td style="text-align:left;">
Up
</td>
<td style="text-align:left;">
0.0039150
</td>
<td style="text-align:left;">
0.0155700
</td>
</tr>
<tr>
<td style="text-align:left;">
platelet activation - actin binding (M196)
</td>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0039350
</td>
<td style="text-align:left;">
0.0155700
</td>
</tr>
<tr>
<td style="text-align:left;">
chaperonin mediated protein folding (I) (M204.0)
</td>
<td style="text-align:left;">
14
</td>
<td style="text-align:left;">
Up
</td>
<td style="text-align:left;">
0.0044000
</td>
<td style="text-align:left;">
0.0171500
</td>
</tr>
<tr>
<td style="text-align:left;">
purine nucleotide biosynthesis (M212)
</td>
<td style="text-align:left;">
12
</td>
<td style="text-align:left;">
Up
</td>
<td style="text-align:left;">
0.0045570
</td>
<td style="text-align:left;">
0.0175000
</td>
</tr>
<tr>
<td style="text-align:left;">
TBA (M194)
</td>
<td style="text-align:left;">
13
</td>
<td style="text-align:left;">
Up
</td>
<td style="text-align:left;">
0.0047540
</td>
<td style="text-align:left;">
0.0180000
</td>
</tr>
<tr>
<td style="text-align:left;">
enriched in activated dendritic cells (II) (M165)
</td>
<td style="text-align:left;">
29
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0053170
</td>
<td style="text-align:left;">
0.0198500
</td>
</tr>
<tr>
<td style="text-align:left;">
recruitment of neutrophils (M132)
</td>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0054720
</td>
<td style="text-align:left;">
0.0201400
</td>
</tr>
<tr>
<td style="text-align:left;">
MHC-TLR7-TLR8 cluster (M146)
</td>
<td style="text-align:left;">
16
</td>
<td style="text-align:left;">
Up
</td>
<td style="text-align:left;">
0.0056540
</td>
<td style="text-align:left;">
0.0205200
</td>
</tr>
<tr>
<td style="text-align:left;">
formyl peptide receptor mediated neutrophil response (M11.2)
</td>
<td style="text-align:left;">
8
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0058170
</td>
<td style="text-align:left;">
0.0208300
</td>
</tr>
<tr>
<td style="text-align:left;">
RA, WNT, CSF receptors network (monocyte) (M23)
</td>
<td style="text-align:left;">
9
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0059290
</td>
<td style="text-align:left;">
0.0209500
</td>
</tr>
<tr>
<td style="text-align:left;">
TBA (M114.0)
</td>
<td style="text-align:left;">
21
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0068850
</td>
<td style="text-align:left;">
0.0238500
</td>
</tr>
<tr>
<td style="text-align:left;">
TBA (M201)
</td>
<td style="text-align:left;">
14
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0069290
</td>
<td style="text-align:left;">
0.0238500
</td>
</tr>
<tr>
<td style="text-align:left;">
inflammatory response (M33)
</td>
<td style="text-align:left;">
8
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0080840
</td>
<td style="text-align:left;">
0.0274600
</td>
</tr>
<tr>
<td style="text-align:left;">
enriched in antigen presentation (III) (M95.1)
</td>
<td style="text-align:left;">
13
</td>
<td style="text-align:left;">
Up
</td>
<td style="text-align:left;">
0.0083280
</td>
<td style="text-align:left;">
0.0275800
</td>
</tr>
<tr>
<td style="text-align:left;">
enriched in neutrophils (II) (M163)
</td>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0084130
</td>
<td style="text-align:left;">
0.0275800
</td>
</tr>
<tr>
<td style="text-align:left;">
mitosis (TF motif CCAATNNSNNNGCG) (M169)
</td>
<td style="text-align:left;">
14
</td>
<td style="text-align:left;">
Up
</td>
<td style="text-align:left;">
0.0084310
</td>
<td style="text-align:left;">
0.0275800
</td>
</tr>
<tr>
<td style="text-align:left;">
platelet activation and degranulation (M85)
</td>
<td style="text-align:left;">
8
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0087400
</td>
<td style="text-align:left;">
0.0279800
</td>
</tr>
<tr>
<td style="text-align:left;">
complement activation (I) (M112.0)
</td>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0087640
</td>
<td style="text-align:left;">
0.0279800
</td>
</tr>
<tr>
<td style="text-align:left;">
TBA (M72.0)
</td>
<td style="text-align:left;">
21
</td>
<td style="text-align:left;">
Up
</td>
<td style="text-align:left;">
0.0092900
</td>
<td style="text-align:left;">
0.0293100
</td>
</tr>
<tr>
<td style="text-align:left;">
adhesion and migration, chemotaxis (M91)
</td>
<td style="text-align:left;">
11
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0098430
</td>
<td style="text-align:left;">
0.0306900
</td>
</tr>
<tr>
<td style="text-align:left;">
chemokines and receptors (M38)
</td>
<td style="text-align:left;">
11
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0102000
</td>
<td style="text-align:left;">
0.0314200
</td>
</tr>
<tr>
<td style="text-align:left;">
TLR8-BAFF network (M25)
</td>
<td style="text-align:left;">
8
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0104400
</td>
<td style="text-align:left;">
0.0318000
</td>
</tr>
<tr>
<td style="text-align:left;">
lipid metabolism, endoplasmic reticulum (M92)
</td>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0108800
</td>
<td style="text-align:left;">
0.0327600
</td>
</tr>
<tr>
<td style="text-align:left;">
extracellular region cluster (GO) (M189)
</td>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0116300
</td>
<td style="text-align:left;">
0.0346400
</td>
</tr>
<tr>
<td style="text-align:left;">
CD1 and other DC receptors (M50)
</td>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0118800
</td>
<td style="text-align:left;">
0.0349900
</td>
</tr>
<tr>
<td style="text-align:left;">
enriched in activated dendritic cells (I) (M119)
</td>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0144600
</td>
<td style="text-align:left;">
0.0421000
</td>
</tr>
<tr>
<td style="text-align:left;">
nuclear pore, transport; mRNA splicing, processing (M143)
</td>
<td style="text-align:left;">
11
</td>
<td style="text-align:left;">
Up
</td>
<td style="text-align:left;">
0.0147900
</td>
<td style="text-align:left;">
0.0426000
</td>
</tr>
<tr>
<td style="text-align:left;">
cytoskeletal remodeling (enriched for SRF targets) (M34)
</td>
<td style="text-align:left;">
8
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0156800
</td>
<td style="text-align:left;">
0.0446800
</td>
</tr>
<tr>
<td style="text-align:left;">
TBA (M72.1)
</td>
<td style="text-align:left;">
16
</td>
<td style="text-align:left;">
Up
</td>
<td style="text-align:left;">
0.0171800
</td>
<td style="text-align:left;">
0.0484400
</td>
</tr>
<tr>
<td style="text-align:left;">
respiratory electron transport chain (mitochondrion) (M231)
</td>
<td style="text-align:left;">
11
</td>
<td style="text-align:left;">
Up
</td>
<td style="text-align:left;">
0.0176400
</td>
<td style="text-align:left;">
0.0492000
</td>
</tr>
<tr>
<td style="text-align:left;">
growth factor induced, enriched in nuclear receptor subfamily 4 (M94)
</td>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0181400
</td>
<td style="text-align:left;">
0.0500700
</td>
</tr>
<tr>
<td style="text-align:left;">
Activated (LPS) dendritic cell surface signature (S11)
</td>
<td style="text-align:left;">
28
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0202800
</td>
<td style="text-align:left;">
0.0553900
</td>
</tr>
<tr>
<td style="text-align:left;">
mismatch repair (II) (M22.1)
</td>
<td style="text-align:left;">
12
</td>
<td style="text-align:left;">
Up
</td>
<td style="text-align:left;">
0.0211200
</td>
<td style="text-align:left;">
0.0571200
</td>
</tr>
<tr>
<td style="text-align:left;">
TBA (M229)
</td>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
Up
</td>
<td style="text-align:left;">
0.0251700
</td>
<td style="text-align:left;">
0.0673600
</td>
</tr>
<tr>
<td style="text-align:left;">
regulation of transcription, transcription factors (M213)
</td>
<td style="text-align:left;">
17
</td>
<td style="text-align:left;">
Up
</td>
<td style="text-align:left;">
0.0257500
</td>
<td style="text-align:left;">
0.0682400
</td>
</tr>
<tr>
<td style="text-align:left;">
enriched for ubiquitination (M138)
</td>
<td style="text-align:left;">
11
</td>
<td style="text-align:left;">
Up
</td>
<td style="text-align:left;">
0.0260100
</td>
<td style="text-align:left;">
0.0682500
</td>
</tr>
<tr>
<td style="text-align:left;">
heme biosynthesis (I) (M171)
</td>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0295700
</td>
<td style="text-align:left;">
0.0768200
</td>
</tr>
<tr>
<td style="text-align:left;">
TBA (M243)
</td>
<td style="text-align:left;">
12
</td>
<td style="text-align:left;">
Up
</td>
<td style="text-align:left;">
0.0298600
</td>
<td style="text-align:left;">
0.0768200
</td>
</tr>
<tr>
<td style="text-align:left;">
TBA (M203)
</td>
<td style="text-align:left;">
13
</td>
<td style="text-align:left;">
Up
</td>
<td style="text-align:left;">
0.0305200
</td>
<td style="text-align:left;">
0.0777600
</td>
</tr>
<tr>
<td style="text-align:left;">
innate activation by cytosolic DNA sensing (M13)
</td>
<td style="text-align:left;">
11
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0313600
</td>
<td style="text-align:left;">
0.0791600
</td>
</tr>
<tr>
<td style="text-align:left;">
signaling in T cells (I) (M35.0)
</td>
<td style="text-align:left;">
13
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0375200
</td>
<td style="text-align:left;">
0.0936400
</td>
</tr>
<tr>
<td style="text-align:left;">
enriched in DNA interacting proteins (M182)
</td>
<td style="text-align:left;">
11
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0378100
</td>
<td style="text-align:left;">
0.0936400
</td>
</tr>
<tr>
<td style="text-align:left;">
enriched for TF motif TTCNRGNNNNTTC (M172)
</td>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0396700
</td>
<td style="text-align:left;">
0.0973300
</td>
</tr>
<tr>
<td style="text-align:left;">
TBA (M141)
</td>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0406800
</td>
<td style="text-align:left;">
0.0988900
</td>
</tr>
<tr>
<td style="text-align:left;">
chaperonin mediated protein folding (II) (M204.1)
</td>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
Up
</td>
<td style="text-align:left;">
0.0425400
</td>
<td style="text-align:left;">
0.1025000
</td>
</tr>
<tr>
<td style="text-align:left;">
B cell development (M9)
</td>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
Up
</td>
<td style="text-align:left;">
0.0430700
</td>
<td style="text-align:left;">
0.1028000
</td>
</tr>
<tr>
<td style="text-align:left;">
cell division - E2F transcription network (M4.8)
</td>
<td style="text-align:left;">
14
</td>
<td style="text-align:left;">
Up
</td>
<td style="text-align:left;">
0.0474100
</td>
<td style="text-align:left;">
0.1120000
</td>
</tr>
<tr>
<td style="text-align:left;">
enriched in G-protein coupled receptors (M130)
</td>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0477600
</td>
<td style="text-align:left;">
0.1120000
</td>
</tr>
<tr>
<td style="text-align:left;">
enriched in B cells (III) (M47.2)
</td>
<td style="text-align:left;">
22
</td>
<td style="text-align:left;">
Up
</td>
<td style="text-align:left;">
0.0506000
</td>
<td style="text-align:left;">
0.1176000
</td>
</tr>
<tr>
<td style="text-align:left;">
glycerophospholipid metabolism (M114.1)
</td>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0533100
</td>
<td style="text-align:left;">
0.1228000
</td>
</tr>
<tr>
<td style="text-align:left;">
nuclear pore complex (mitosis) (M106.1)
</td>
<td style="text-align:left;">
11
</td>
<td style="text-align:left;">
Up
</td>
<td style="text-align:left;">
0.0559000
</td>
<td style="text-align:left;">
0.1276000
</td>
</tr>
<tr>
<td style="text-align:left;">
BCR signaling (M54)
</td>
<td style="text-align:left;">
12
</td>
<td style="text-align:left;">
Up
</td>
<td style="text-align:left;">
0.0563400
</td>
<td style="text-align:left;">
0.1276000
</td>
</tr>
<tr>
<td style="text-align:left;">
DNA repair (M76)
</td>
<td style="text-align:left;">
19
</td>
<td style="text-align:left;">
Up
</td>
<td style="text-align:left;">
0.0580300
</td>
<td style="text-align:left;">
0.1303000
</td>
</tr>
<tr>
<td style="text-align:left;">
TBA (M93)
</td>
<td style="text-align:left;">
8
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0609300
</td>
<td style="text-align:left;">
0.1357000
</td>
</tr>
<tr>
<td style="text-align:left;">
lysosome (M209)
</td>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0618500
</td>
<td style="text-align:left;">
0.1365000
</td>
</tr>
<tr>
<td style="text-align:left;">
TBA (M128)
</td>
<td style="text-align:left;">
11
</td>
<td style="text-align:left;">
Up
</td>
<td style="text-align:left;">
0.0623200
</td>
<td style="text-align:left;">
0.1365000
</td>
</tr>
<tr>
<td style="text-align:left;">
mitochondrial cluster (M235)
</td>
<td style="text-align:left;">
17
</td>
<td style="text-align:left;">
Up
</td>
<td style="text-align:left;">
0.0631300
</td>
<td style="text-align:left;">
0.1371000
</td>
</tr>
<tr>
<td style="text-align:left;">
translation initiation (M227)
</td>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
Up
</td>
<td style="text-align:left;">
0.0641100
</td>
<td style="text-align:left;">
0.1381000
</td>
</tr>
<tr>
<td style="text-align:left;">
transcription elongation, RNA polymerase II (M234)
</td>
<td style="text-align:left;">
13
</td>
<td style="text-align:left;">
Up
</td>
<td style="text-align:left;">
0.0663200
</td>
<td style="text-align:left;">
0.1417000
</td>
</tr>
<tr>
<td style="text-align:left;">
cell cycle, ATP binding (M144)
</td>
<td style="text-align:left;">
16
</td>
<td style="text-align:left;">
Up
</td>
<td style="text-align:left;">
0.0706700
</td>
<td style="text-align:left;">
0.1498000
</td>
</tr>
<tr>
<td style="text-align:left;">
TBA (M153)
</td>
<td style="text-align:left;">
15
</td>
<td style="text-align:left;">
Up
</td>
<td style="text-align:left;">
0.0864500
</td>
<td style="text-align:left;">
0.1818000
</td>
</tr>
<tr>
<td style="text-align:left;">
TBA (M55)
</td>
<td style="text-align:left;">
11
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0874700
</td>
<td style="text-align:left;">
0.1825000
</td>
</tr>
<tr>
<td style="text-align:left;">
TBA (M242)
</td>
<td style="text-align:left;">
11
</td>
<td style="text-align:left;">
Up
</td>
<td style="text-align:left;">
0.0923200
</td>
<td style="text-align:left;">
0.1911000
</td>
</tr>
<tr>
<td style="text-align:left;">
platelet activation (I) (M32.0)
</td>
<td style="text-align:left;">
20
</td>
<td style="text-align:left;">
Up
</td>
<td style="text-align:left;">
0.0973600
</td>
<td style="text-align:left;">
0.1995000
</td>
</tr>
<tr>
<td style="text-align:left;">
CCR1, 7 and cell signaling (M59)
</td>
<td style="text-align:left;">
8
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0978700
</td>
<td style="text-align:left;">
0.1995000
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="plots-m3-dmso" class="section level1">
<h1><span class="header-section-number">2</span> Plots: M3 DMSO</h1>
<p><img src="figure/manuscript_Fig3_disease.Rmd/heatme_vac-1.png" width="672" style="display: block; margin: auto;" /></p>
<div id="disease-m3-mo" class="section level2 tabset tabset-fade tabset-pills">
<h2><span class="header-section-number">2.1</span> disease M3-MO</h2>
<p>Only Bagamoya samples with the dmso stimulation are used. A consequence of this is that only subjects from the old age group are included. Additionally only paired samples by subject are included.</p>
<table class="table table-striped table-hover table-condensed" style="width: auto !important; ">
<thead>
<tr>
<th style="border-bottom:hidden" colspan="1">
</th>
<th style="border-bottom:hidden" colspan="1">
</th>
<th style="border-bottom:hidden; padding-bottom:0; padding-left:3px;padding-right:3px;text-align: center; " colspan="2">
<div style="border-bottom: 1px solid #ddd; padding-bottom: 5px; ">
vaccine
</div>
</th>
<th style="border-bottom:hidden; padding-bottom:0; padding-left:3px;padding-right:3px;text-align: center; " colspan="2">
<div style="border-bottom: 1px solid #ddd; padding-bottom: 5px; ">
disease
</div>
</th>
<th style="border-bottom:hidden; padding-bottom:0; padding-left:3px;padding-right:3px;text-align: center; " colspan="2">
<div style="border-bottom: 1px solid #ddd; padding-bottom: 5px; ">
age
</div>
</th>
<th style="border-bottom:hidden; padding-bottom:0; padding-left:3px;padding-right:3px;text-align: center; " colspan="2">
<div style="border-bottom: 1px solid #ddd; padding-bottom: 5px; ">
visit
</div>
</th>
<th style="border-bottom:hidden; padding-bottom:0; padding-left:3px;padding-right:3px;text-align: center; " colspan="2">
<div style="border-bottom: 1px solid #ddd; padding-bottom: 5px; ">
case
</div>
</th>
</tr>
<tr>
<th style="text-align:left;">
</th>
<th style="text-align:center;">
total
</th>
<th style="text-align:center;">
comparator
</th>
<th style="text-align:center;">
rtss
</th>
<th style="text-align:center;">
X0
</th>
<th style="text-align:center;">
X1
</th>
<th style="text-align:center;">
young
</th>
<th style="text-align:center;">
old
</th>
<th style="text-align:center;">
M0
</th>
<th style="text-align:center;">
M3
</th>
<th style="text-align:center;">
control
</th>
<th style="text-align:center;">
case
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
dmso
</td>
<td style="text-align:center;">
106
</td>
<td style="text-align:center;">
34
</td>
<td style="text-align:center;">
72
</td>
<td style="text-align:center;">
80
</td>
<td style="text-align:center;">
26
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
106
</td>
<td style="text-align:center;">
53
</td>
<td style="text-align:center;">
53
</td>
<td style="text-align:center;">
80
</td>
<td style="text-align:center;">
26
</td>
</tr>
</tbody>
</table>
<p>formula: ~plate + total_reads + pid + visit<br />
coefficient: visitM3</p>
<div id="malaria-and-rtss" class="section level3">
<h3><span class="header-section-number">2.1.1</span> malaria and rtss</h3>
<div id="htmlwidget-cb19dd80c465e6b9c801" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-cb19dd80c465e6b9c801">{"x":{"filter":"none","data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23","24","25","26","27","28","29","30","31","32","33","34","35","36","37","38","39","40","41","42","43","44","45","46","47","48","49","50","51","52","53","54","55","56","57","58","59","60","61","62","63","64","65","66","67","68","69","70","71","72","73","74","75","76","77","78","79"],["chemokines and inflammatory molecules in myeloid cells (M86.0)","type I interferon response (M127)","enriched in monocytes (II) (M11.0)","cell cycle and growth arrest (M31)","cytokines - recepters cluster (M115)","lysosome (M209)","proinflammatory cytokines and chemokines (M29)","enriched in NK cells (I) (M7.2)","proinflammatory dendritic cell, myeloid cell response (M86.1)","regulation of transcription, transcription factors (M213)","Resting dendritic cell surface signature (S10)","chemokine cluster (I) (M27.0)","TBA (M177.1)","RIG-1 like receptor signaling (M68)","enriched in activated dendritic cells/monocytes (M64)","activated dendritic cells (M67)","cell cycle and transcription (M4.0)","extracellular matrix, complement (M140)","enriched in monocytes (surface) (M118.1)","putative targets of PAX3 (M89.0)","leukocyte migration (M88.0)","signaling in T cells (II) (M35.1)","Monocyte surface signature (S4)","enriched for TF motif PAX3 (M179)","enriched in monocytes (IV) (M118.0)","TBA (M174)","leukocyte differentiation (M160)","TBA (M72.2)","innate activation by cytosolic DNA sensing (M13)","immuregulation - monocytes, T and B cells (M57)","proteasome (M226)","lysosomal/endosomal proteins (M139)","TBA (M72.1)","myeloid cell enriched receptors and transporters (M4.3)","extracellular matrix (II) (M2.1)","TBA (M190)","TBA (M121)","TBA (source: B cells) (M152.0)","inflammatory response (M33)","antiviral IFN signature (M75)","BCR signaling (M54)","intracellular transport (M147)","TLR and inflammatory signaling (M16)","signaling in T cells (I) (M35.0)","platelet activation (I) (M32.0)","T cell activation (II) (M7.3)","translation initiation (M227)","growth factor induced, enriched in nuclear receptor subfamily 4 (M94)","chemokines and receptors (M38)","chaperonin mediated protein folding (II) (M204.1)","DC surface signature (S5)","leukocyte activation and migration (M45)","putative targets of PAX3 (M89.1)","TBA (M72.0)","cell cycle, ATP binding (M144)","Plasma cell surface signature (S3)","transcriptional targets of glucocorticoid receptor (M74)","enriched in T cells (I) (M7.0)","complement activation (I) (M112.0)","TBA (M185)","innate antiviral response (M150)","enriched in B cells (I) (M47.0)","C-MYC transcriptional network (M4.12)","TBA (M153)","TBA (M105)","TBA (M205)","glycerophospholipid metabolism (M114.1)","T cell activation (I) (M7.1)","enriched for ubiquitination (M138)","nuclear pore complex (M106.0)","TBA (M192)","enriched in NK cells (II) (M61.0)","mitosis (TF motif CCAATNNSNNNGCG) (M169)","TBA (M141)","E2F transcription factor network (M8)","TBA (M177.0)","enriched in B cells (II) (M47.1)","Memory B cell surface signature (S9)","viral sensing &amp; immunity; IRF2 targets network (II) (M111.1)"],[17,12,146,11,10,7,9,45,11,17,43,20,10,10,17,9,226,7,17,12,12,10,69,8,48,22,14,12,11,10,12,10,16,26,18,10,10,6,8,19,12,17,34,13,20,29,10,7,11,10,54,11,9,21,16,10,7,56,10,14,12,39,9,15,11,11,10,47,11,17,11,13,14,7,7,26,32,13,11],["Up","Up","Down","Up","Up","Down","Up","Up","Up","Up","Down","Up","Down","Up","Up","Up","Down","Down","Down","Up","Down","Up","Down","Up","Down","Up","Up","Up","Up","Up","Up","Down","Up","Down","Down","Up","Up","Down","Down","Up","Up","Up","Down","Up","Up","Up","Up","Up","Up","Up","Down","Up","Up","Up","Up","Down","Up","Up","Down","Up","Up","Up","Up","Up","Down","Up","Down","Up","Up","Up","Up","Up","Up","Down","Down","Down","Up","Down","Up"],[1.90453643227777e-09,2.63167073578011e-06,3.89677972697631e-06,5.05284615099142e-06,2.00591142370959e-05,2.64147660612391e-05,3.76981565817256e-05,5.07187953172544e-05,5.2434671750766e-05,0.000144214551658397,0.000166091840698774,0.000208331147106955,0.000215852840247947,0.000469446915600277,0.000592571137565419,0.000617123142701722,0.000905383022967583,0.0011537223145539,0.00117020502553468,0.00119659087365365,0.00125892823128851,0.00156017375871643,0.00173478999185587,0.00241158919578683,0.00247897726656561,0.00337300090490381,0.00346868441440782,0.00390503415473682,0.00411923050584745,0.00420600631705671,0.00549125012983506,0.00552696399136849,0.00574407944374738,0.00610874358831975,0.00626668513297943,0.00706526469105897,0.00833075812403209,0.00931846493643478,0.00996126337891035,0.0132268650418372,0.0146723747422088,0.0154536201806447,0.0156571208592184,0.0157961672511319,0.015802735341687,0.0160128986286287,0.0166090404170602,0.0181210882430552,0.0189307136398696,0.0192438033566582,0.0198239164306107,0.0213007359670724,0.0226745864950972,0.0231233777299294,0.0239824087190682,0.0242978686341297,0.0264219835132972,0.0298355500443178,0.0298626561187838,0.0312478216310807,0.0313094149899624,0.0328360048104965,0.0358899133225724,0.0359121302033559,0.0385072069708273,0.0390496660831632,0.0391700162016312,0.0413294629030862,0.0419732106587487,0.0423360316454158,0.0462794565256455,0.0468283965522597,0.0488074957910487,0.0513073285953758,0.0532620512521295,0.0549311709107337,0.0558507520476088,0.0578779448503246,0.059508998870743],[5.04702154553608e-07,0.000334751057503182,0.000334751057503182,0.000334751057503182,0.00106313305456608,0.00116665216770473,0.00142714449916533,0.00154390977932811,0.00154390977932811,0.00382168561894751,0.00400130343501592,0.00440007712813123,0.00440007712813123,0.00888595947386238,0.0102211020509973,0.0102211020509973,0.0141133235933182,0.0158548290759109,0.0158548290759109,0.0158548290759109,0.0158864752995931,0.0187930020936297,0.0199877977322524,0.0262771590255955,0.0262771590255955,0.0340444951784471,0.0340444951784471,0.0369583589644735,0.0371530558006676,0.0371530558006676,0.0457701705535203,0.0457701705535203,0.046126698563426,0.0474477588639871,0.0474477588639871,0.0520081984202952,0.0596662406180677,0.0649840317935583,0.0676855075746473,0.0876279809021713,0.0922482203605781,0.0922482203605781,0.0922482203605781,0.0922482203605781,0.0922482203605781,0.0922482203605781,0.0936467172451267,0.100043508008534,0.101992157790289,0.101992157790289,0.103006624590428,0.108551827524503,0.113372932475486,0.113475835156135,0.114980985500792,0.114980985500792,0.122839046158312,0.134128879177588,0.134128879177588,0.136016311021968,0.136016311021968,0.140347439915832,0.148698664123271,0.148698664123271,0.154926183484064,0.154926183484064,0.154926183484064,0.160272119800503,0.160272119800503,0.160272119800503,0.172354515088178,0.172354515088178,0.177177895679834,0.183735703753711,0.188192581090858,0.191536319622953,0.192213627176835,0.196636607504308,0.19961879368034]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>geneset<\/th>\n      <th>NGenes<\/th>\n      <th>Direction<\/th>\n      <th>PValue<\/th>\n      <th>FDR<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[2,4,5]},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
</div>
<div id="malaria-and-comparator" class="section level3">
<h3><span class="header-section-number">2.1.2</span> malaria and comparator</h3>
<div id="htmlwidget-467a997665fe6066d18f" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-467a997665fe6066d18f">{"x":{"filter":"none","data":[[],[],[],[],[],[]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>geneset<\/th>\n      <th>NGenes<\/th>\n      <th>Direction<\/th>\n      <th>PValue<\/th>\n      <th>FDR<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[2,4,5]},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
</div>
<div id="no-malaria-and-rtss" class="section level3">
<h3><span class="header-section-number">2.1.3</span> no malaria and rtss</h3>
<div id="htmlwidget-8dec71924fac8f69e718" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-8dec71924fac8f69e718">{"x":{"filter":"none","data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23","24","25","26","27","28","29","30","31","32","33","34","35","36","37","38","39","40","41","42"],["type I interferon response (M127)","enriched in NK cells (I) (M7.2)","signaling in T cells (II) (M35.1)","enriched in cell cycle (M167)","activated dendritic cells (M67)","signaling in T cells (I) (M35.0)","erythrocyte differentiation (M173)","antiviral IFN signature (M75)","NK cell surface signature (S1)","RIG-1 like receptor signaling (M68)","chemokines and inflammatory molecules in myeloid cells (M86.0)","chemokine cluster (I) (M27.0)","innate antiviral response (M150)","TBA (M32.6)","cytokines - recepters cluster (M115)","TBA (M121)","platelet activation (I) (M32.0)","TBA (M229)","CORO1A-DEF6 network (II) (M32.4)","TBA (M131)","heme biosynthesis (I) (M171)","platelet activation (II) (M32.1)","viral sensing &amp; immunity; IRF2 targets network (II) (M111.1)","KLF12 targets network (M32.3)","enriched in activated dendritic cells (II) (M165)","cell cycle and growth arrest (M31)","TBA (M203)","proinflammatory cytokines and chemokines (M29)","CD1 and other DC receptors (M50)","CORO1A-DEF6 network (I) (M32.2)","TBA (M177.1)","enriched in G-protein coupled receptors (M130)","chemokines and receptors (M38)","regulation of localization (GO) (M63)","TBA (M194)","TBA (M248)","Naive B cell surface signature (S8)","myeloid, dendritic cell activation via NFkB (I) (M43.0)","regulation of transcription, transcription factors (M213)","TBA (M218)","TBA (M136)","cell division (stimulated CD4+ T cells) (M46)"],[12,45,10,14,9,13,7,19,24,10,17,20,12,12,10,10,20,10,13,12,10,20,11,15,29,11,13,9,6,17,10,10,11,11,13,10,24,12,17,13,16,13],["Up","Up","Up","Up","Up","Up","Up","Up","Up","Up","Up","Up","Up","Up","Up","Up","Up","Up","Up","Up","Up","Up","Up","Up","Up","Up","Up","Up","Down","Up","Down","Up","Up","Up","Up","Up","Down","Up","Up","Down","Up","Down"],[3.44615928616607e-07,6.78806411974507e-05,0.000130188653858956,0.000255133439545161,0.000293803594387993,0.000381355171635222,0.000401493688650389,0.000462992671782105,0.000485779233192006,0.000522485541883971,0.000617599994521602,0.00119132794077416,0.00147808846187442,0.00164006331569626,0.00167404220474516,0.00173369336477406,0.00325926843839069,0.00470727925011392,0.00508151121351931,0.00520883080599396,0.00562507041399934,0.00573935358235048,0.0064932618283441,0.00662861991315535,0.00784350696498044,0.00788074681542239,0.0084661972632859,0.00980274184734975,0.0103811113837401,0.0148236388962913,0.0163580894653915,0.0169790382832913,0.0175628485494615,0.0185589042996995,0.0223469303647273,0.0236404604418915,0.0249318590023077,0.0278723272472593,0.0286747972955959,0.0292345397978472,0.0294624023571416,0.0296568242715698],[9.13232210834008e-05,0.00899418495866221,0.0114999977575411,0.0138458668599252,0.0138458668599252,0.0138458668599252,0.0138458668599252,0.0138458668599252,0.0138458668599252,0.0138458668599252,0.0148785453225659,0.0263084920254294,0.0287142963540704,0.0287142963540704,0.0287142963540704,0.0287142963540704,0.0508062433043254,0.06901700817942,0.06901700817942,0.06901700817942,0.0691331226964944,0.0691331226964944,0.0731910115410903,0.0731910115410903,0.0803229963879589,0.0803229963879589,0.0830941583248431,0.092775949626703,0.0948618798859008,0.130942143583906,0.139835280913831,0.140607660783506,0.141034995927494,0.144650283512364,0.169198187047221,0.17402005603059,0.17856601717869,0.187120438856333,0.187120438856333,0.187120438856333,0.187120438856333,0.187120438856333]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>geneset<\/th>\n      <th>NGenes<\/th>\n      <th>Direction<\/th>\n      <th>PValue<\/th>\n      <th>FDR<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[2,4,5]},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
</div>
<div id="no-malaria-and-comparator" class="section level3">
<h3><span class="header-section-number">2.1.4</span> no malaria and comparator</h3>
<div id="htmlwidget-d2a941107faeef0f93d3" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-d2a941107faeef0f93d3">{"x":{"filter":"none","data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23","24","25","26","27","28","29","30","31","32","33","34","35","36","37","38","39","40","41","42","43","44","45","46","47","48","49","50","51","52","53","54","55","56","57","58","59","60","61","62","63","64","65"],["enriched in monocytes (II) (M11.0)","formyl peptide receptor mediated neutrophil response (M11.2)","cell cycle and transcription (M4.0)","Monocyte surface signature (S4)","TBA (M66)","enriched in monocytes (I) (M4.15)","complement and other receptors in DCs (M40)","lysosomal/endosomal proteins (M139)","enriched in monocytes (III) (M73)","enriched in neutrophils (I) (M37.1)","respiratory electron transport chain (mitochondrion) (M219)","respiratory electron transport chain (mitochondrion) (M238)","extracellular matrix, complement (M140)","inflammatory response (M33)","myeloid cell enriched receptors and transporters (M4.3)","TLR and inflammatory signaling (M16)","enriched in antigen presentation (III) (M95.1)","transcription elongation, RNA polymerase II (M234)","transcription regulation in cell development (M49)","respiratory electron transport chain (mitochondrion) (M216)","enriched in antigen presentation (II) (M95.0)","extracellular matrix (II) (M2.1)","regulation of signal transduction (M3)","Resting dendritic cell surface signature (S10)","translation initiation factor 3 complex (M245)","cell junction (GO) (M4.13)","TBA (M243)","glycerophospholipid metabolism (M114.1)","regulation of antigen presentation and immune response (M5.0)","enriched in monocytes (surface) (M118.1)","enriched in membrane proteins (M124)","spliceosome (M250)","respiratory electron transport chain (mitochondrion) (M231)","DC surface signature (S5)","TLR8-BAFF network (M25)","enriched in neutrophils (II) (M163)","endoplasmic reticulum (M37.2)","transmembrane transport (I) (M87)","immune activation - generic cluster (M37.0)","heme biosynthesis (I) (M171)","heme biosynthesis (II) (M222)","TBA (M114.0)","complement activation (I) (M112.0)","CCR1, 7 and cell signaling (M59)","enriched in activated dendritic cells/monocytes (M64)","platelet activation and degranulation (M85)","enriched in monocytes (IV) (M118.0)","leukocyte activation and migration (M45)","cytoskeletal remodeling (enriched for SRF targets) (M34)","cytokines - recepters cluster (M115)","myeloid, dendritic cell activation via NFkB (I) (M43.0)","collagen, TGFB family et al (M77)","T cell signaling and costimulation (M44)","enriched in antigen presentation (I) (M71)","TBA (M79)","T cell surface, activation (M36)","innate antiviral response (M150)","lipid metabolism, endoplasmic reticulum (M92)","TBA (M32.6)","platelet activation (III) (M42)","extracellular region cluster (GO) (M189)","enriched in activated dendritic cells (II) (M165)","CD1 and other DC receptors (M50)","suppression of MAPK signaling (M56)","lysosome (M209)"],[146,8,226,69,17,8,11,10,10,29,18,17,7,8,26,34,13,13,37,12,21,18,33,43,11,11,12,10,76,17,13,12,11,54,8,7,11,19,210,10,10,21,10,8,17,8,48,11,8,10,12,14,10,16,7,11,12,6,12,8,6,29,6,10,7],["Down","Down","Down","Down","Down","Down","Down","Down","Down","Down","Down","Down","Down","Down","Down","Down","Down","Down","Down","Down","Down","Down","Down","Down","Down","Down","Down","Down","Down","Down","Down","Down","Down","Down","Down","Down","Down","Down","Down","Down","Down","Down","Down","Down","Down","Down","Down","Down","Down","Up","Up","Down","Down","Down","Down","Down","Up","Down","Down","Down","Down","Down","Down","Down","Down"],[1.4960195809498e-20,1.60713833462177e-09,1.55900346112604e-08,4.08264119738221e-07,4.81911134276879e-06,1.21281921265469e-05,1.22979985298106e-05,1.61755227353073e-05,1.6719138243374e-05,3.3051646186747e-05,4.00115162402026e-05,0.000167329615190511,0.000174188410146345,0.000183465512203384,0.000202032210698141,0.000203487975340271,0.000257559431652626,0.000273663934149102,0.000299116617680588,0.000418336634144487,0.000425106529518385,0.000435671078932672,0.000441666309224528,0.000582864324453929,0.000615033269348213,0.000814270096115619,0.00086172165614419,0.00223240948829614,0.00407290761736364,0.0053614546457925,0.00542820067017875,0.00651696379512017,0.00696365298002184,0.00701710002313147,0.0071134935721791,0.00771962266490986,0.00827784166846934,0.00851937345015527,0.0106721502486699,0.0117099216931766,0.0137191376397459,0.0141395169931446,0.0144234725152776,0.0150601411286461,0.0171681649476145,0.0172072340793812,0.0196205311177934,0.0200369338108021,0.0218826887234476,0.027411004062169,0.0282986471089623,0.02934479551211,0.0297737981376175,0.0303975840047775,0.030713330045874,0.0314406429424426,0.031634287700082,0.0320471927273314,0.0357943262356869,0.0385768794242853,0.0390008138496935,0.039311207814602,0.0414721574483831,0.0451167751351535,0.046312086508963],[3.96445188951698e-18,2.12945829337384e-07,1.37711972399467e-06,2.70474979326571e-05,0.000255412901166746,0.000465567087199974,0.000465567087199974,0.000492285737166011,0.000492285737166011,0.000875868623948794,0.000963913800332155,0.00337026959157324,0.00337026959157324,0.00337026959157324,0.00337026959157324,0.00337026959157324,0.00401489702282035,0.00402894125275067,0.00417188966765031,0.00508876399758695,0.00508876399758695,0.00508876399758695,0.00508876399758695,0.00643579358251213,0.00651935265509106,0.00829929136425534,0.00845763847697076,0.021128161228517,0.0372179489172884,0.0464023605676571,0.0464023605676571,0.0538593084750703,0.0538593084750703,0.0538593084750703,0.0538593084750703,0.0568250001722531,0.0592872443822804,0.0594114201129249,0.0725158927153208,0.077578231217295,0.0886724749886014,0.0888888422453153,0.0888888422453153,0.0907031227066183,0.0991286311094786,0.0991286311094786,0.11062057208047,0.11062057208047,0.118345153300278,0.145278321529496,0.146422518495566,0.146422518495566,0.146422518495566,0.146422518495566,0.146422518495566,0.146422518495566,0.146422518495566,0.146422518495566,0.160771126312831,0.168023710820476,0.168023710820476,0.168023710820476,0.174446376568595,0.186811647043995,0.188810814228849]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>geneset<\/th>\n      <th>NGenes<\/th>\n      <th>Direction<\/th>\n      <th>PValue<\/th>\n      <th>FDR<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[2,4,5]},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
</div>
</div>
</div>
<div id="plots-m3-m0" class="section level1">
<h1><span class="header-section-number">3</span> Plots: M3-M0</h1>
</div>
<div id="down-selected-disease-analysis" class="section level1">
<h1><span class="header-section-number">4</span> Down-selected disease analysis</h1>
<div id="m3-dmso-disease-comparison-on-down-selected-vaccine-gene-sets" class="section level2">
<h2><span class="header-section-number">4.1</span> M3 DMSO disease comparison on down-selected vaccine gene-sets</h2>
<div id="driver-genes-from-m3-disease-comparison-significant-gene-sets-section2" class="section level3">
<h3><span class="header-section-number">4.1.1</span> Driver genes from M3 disease comparison significant gene-sets (Section2)</h3>
<p>Extract driver genes from down-selected monocyte related significant gene-sets.</p>
</div>
<div id="enriched-in-monocytes-ii-m11.0" class="section level3 tabset tabset-fade tabset-pills">
<h3><span class="header-section-number">4.1.2</span> enriched in monocytes (II) (M11.0)</h3>
<div id="table-of-significant-genes" class="section level4">
<h4><span class="header-section-number">4.1.2.1</span> Table of significant genes</h4>
<p>Pvalue cutoff &lt;= 0.1</p>
<table class="table table-striped table-hover table-condensed" style="width: auto !important; ">
<thead>
<tr>
<th style="text-align:center;">
genenames
</th>
<th style="text-align:center;">
logFC
</th>
<th style="text-align:center;">
AveExpr
</th>
<th style="text-align:center;">
P.Value
</th>
<th style="text-align:center;">
adj.P.Val
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:center;">
RBM47
</td>
<td style="text-align:center;">
0.488
</td>
<td style="text-align:center;">
2.64
</td>
<td style="text-align:center;">
0.016
</td>
<td style="text-align:center;">
0.998
</td>
</tr>
<tr>
<td style="text-align:center;">
CLEC4E
</td>
<td style="text-align:center;">
0.539
</td>
<td style="text-align:center;">
3.39
</td>
<td style="text-align:center;">
0.027
</td>
<td style="text-align:center;">
0.998
</td>
</tr>
<tr>
<td style="text-align:center;">
SIGLEC9
</td>
<td style="text-align:center;">
-0.472
</td>
<td style="text-align:center;">
1.54
</td>
<td style="text-align:center;">
0.027
</td>
<td style="text-align:center;">
0.998
</td>
</tr>
<tr>
<td style="text-align:center;">
RIN2
</td>
<td style="text-align:center;">
0.487
</td>
<td style="text-align:center;">
2.05
</td>
<td style="text-align:center;">
0.031
</td>
<td style="text-align:center;">
0.998
</td>
</tr>
<tr>
<td style="text-align:center;">
CEBPD
</td>
<td style="text-align:center;">
0.450
</td>
<td style="text-align:center;">
1.79
</td>
<td style="text-align:center;">
0.039
</td>
<td style="text-align:center;">
0.998
</td>
</tr>
<tr>
<td style="text-align:center;">
CFP
</td>
<td style="text-align:center;">
0.442
</td>
<td style="text-align:center;">
2.34
</td>
<td style="text-align:center;">
0.047
</td>
<td style="text-align:center;">
0.998
</td>
</tr>
<tr>
<td style="text-align:center;">
DMXL2
</td>
<td style="text-align:center;">
0.460
</td>
<td style="text-align:center;">
3.02
</td>
<td style="text-align:center;">
0.050
</td>
<td style="text-align:center;">
0.998
</td>
</tr>
<tr>
<td style="text-align:center;">
TNS3
</td>
<td style="text-align:center;">
0.434
</td>
<td style="text-align:center;">
1.82
</td>
<td style="text-align:center;">
0.053
</td>
<td style="text-align:center;">
0.998
</td>
</tr>
<tr>
<td style="text-align:center;">
PRAM1
</td>
<td style="text-align:center;">
0.482
</td>
<td style="text-align:center;">
2.32
</td>
<td style="text-align:center;">
0.054
</td>
<td style="text-align:center;">
0.998
</td>
</tr>
<tr>
<td style="text-align:center;">
CPVL
</td>
<td style="text-align:center;">
0.320
</td>
<td style="text-align:center;">
0.94
</td>
<td style="text-align:center;">
0.054
</td>
<td style="text-align:center;">
0.998
</td>
</tr>
<tr>
<td style="text-align:center;">
FES
</td>
<td style="text-align:center;">
0.453
</td>
<td style="text-align:center;">
2.25
</td>
<td style="text-align:center;">
0.056
</td>
<td style="text-align:center;">
0.998
</td>
</tr>
<tr>
<td style="text-align:center;">
LAT2
</td>
<td style="text-align:center;">
-0.351
</td>
<td style="text-align:center;">
3.84
</td>
<td style="text-align:center;">
0.078
</td>
<td style="text-align:center;">
0.998
</td>
</tr>
<tr>
<td style="text-align:center;">
TLR4
</td>
<td style="text-align:center;">
0.371
</td>
<td style="text-align:center;">
1.63
</td>
<td style="text-align:center;">
0.083
</td>
<td style="text-align:center;">
0.998
</td>
</tr>
<tr>
<td style="text-align:center;">
CD302
</td>
<td style="text-align:center;">
0.371
</td>
<td style="text-align:center;">
2.78
</td>
<td style="text-align:center;">
0.089
</td>
<td style="text-align:center;">
0.998
</td>
</tr>
<tr>
<td style="text-align:center;">
FBP1
</td>
<td style="text-align:center;">
0.352
</td>
<td style="text-align:center;">
1.30
</td>
<td style="text-align:center;">
0.094
</td>
<td style="text-align:center;">
0.998
</td>
</tr>
</tbody>
</table>
</div>
<div id="plot-of-all-genes" class="section level4">
<h4><span class="header-section-number">4.1.2.2</span> Plot of all genes</h4>
<p><img src="figure/manuscript_Fig3_disease.Rmd/unnamed-chunk-6-1.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="plot-signifcant-genes" class="section level4">
<h4><span class="header-section-number">4.1.2.3</span> Plot signifcant genes</h4>
<p><img src="figure/manuscript_Fig3_disease.Rmd/unnamed-chunk-7-1.png" width="672" style="display: block; margin: auto;" /></p>
<p><img src="figure/manuscript_Fig3_disease.Rmd/unnamed-chunk-8-1.png" width="672" style="display: block; margin: auto;" /></p>
<p></br></p>
</div>
</div>
<div id="enriched-in-myeloid-cells-and-monocytes-m81" class="section level3 tabset tabset-fade tabset-pills">
<h3><span class="header-section-number">4.1.3</span> enriched in myeloid cells and monocytes (M81)</h3>
<div id="table-of-significant-genes-1" class="section level4">
<h4><span class="header-section-number">4.1.3.1</span> Table of significant genes</h4>
<p>Pvalue cutoff &lt;= 0.1</p>
<table class="table table-striped table-hover table-condensed" style="width: auto !important; ">
<thead>
<tr>
<th style="text-align:center;">
genenames
</th>
<th style="text-align:center;">
logFC
</th>
<th style="text-align:center;">
AveExpr
</th>
<th style="text-align:center;">
P.Value
</th>
<th style="text-align:center;">
adj.P.Val
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:center;">
PRAM1
</td>
<td style="text-align:center;">
0.482
</td>
<td style="text-align:center;">
2.320
</td>
<td style="text-align:center;">
0.054
</td>
<td style="text-align:center;">
0.998
</td>
</tr>
<tr>
<td style="text-align:center;">
SLC8A1
</td>
<td style="text-align:center;">
0.266
</td>
<td style="text-align:center;">
0.853
</td>
<td style="text-align:center;">
0.081
</td>
<td style="text-align:center;">
0.998
</td>
</tr>
<tr>
<td style="text-align:center;">
TLR4
</td>
<td style="text-align:center;">
0.371
</td>
<td style="text-align:center;">
1.630
</td>
<td style="text-align:center;">
0.083
</td>
<td style="text-align:center;">
0.998
</td>
</tr>
</tbody>
</table>
</div>
<div id="plot-of-all-genes-1" class="section level4">
<h4><span class="header-section-number">4.1.3.2</span> Plot of all genes</h4>
<p><img src="figure/manuscript_Fig3_disease.Rmd/unnamed-chunk-11-1.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="plot-signifcant-genes-1" class="section level4">
<h4><span class="header-section-number">4.1.3.3</span> Plot signifcant genes</h4>
<p><img src="figure/manuscript_Fig3_disease.Rmd/unnamed-chunk-12-1.png" width="672" style="display: block; margin: auto;" /></p>
<p><img src="figure/manuscript_Fig3_disease.Rmd/unnamed-chunk-13-1.png" width="672" style="display: block; margin: auto;" /></p>
<p></br></p>
</div>
</div>
</div>
<div id="m3-m0-dmso-disease-comparison-on-down-selected-vaccine-gene-sets" class="section level2">
<h2><span class="header-section-number">4.2</span> M3-M0 DMSO disease comparison on down-selected vaccine gene-sets</h2>
</div>
<div id="all-significant-genes" class="section level2">
<h2><span class="header-section-number">4.3</span> All significant genes</h2>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span> Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<p>R version 3.6.1 (2019-07-05) Platform: x86_64-pc-linux-gnu (64-bit) Running under: Ubuntu 14.04.5 LTS</p>
<p>Matrix products: default BLAS/LAPACK: /app/easybuild/software/OpenBLAS/0.2.18-GCC-5.4.0-2.26-LAPACK-3.6.1/lib/libopenblas_prescottp-r0.2.18.so</p>
<p>locale: [1] LC_CTYPE=en_US.UTF-8 LC_NUMERIC=C<br />
[3] LC_TIME=en_US.UTF-8 LC_COLLATE=en_US.UTF-8<br />
[5] LC_MONETARY=en_US.UTF-8 LC_MESSAGES=en_US.UTF-8<br />
[7] LC_PAPER=en_US.UTF-8 LC_NAME=C<br />
[9] LC_ADDRESS=C LC_TELEPHONE=C<br />
[11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C</p>
<p>attached base packages: [1] stats4 parallel stats graphics grDevices utils datasets [8] methods base</p>
<p>other attached packages: [1] mal067data_1.0 kableExtra_1.1.0 forcats_0.4.0<br />
[4] stringr_1.4.0 dplyr_0.8.3 purrr_0.3.3<br />
[7] tidyr_0.8.3 tibble_2.1.3 ggplot2_3.2.1<br />
[10] tidyverse_1.2.1 RColorBrewer_1.1-2 data.table_1.12.8<br />
[13] limma_3.40.2 GSEABase_1.46.0 graph_1.62.0<br />
[16] annotate_1.62.0 XML_3.98-1.19 AnnotationDbi_1.46.0 [19] IRanges_2.18.1 S4Vectors_0.22.0 DT_0.11<br />
[22] heatmap3_1.1.6 here_0.1 magrittr_1.5<br />
[25] Biobase_2.44.0 BiocGenerics_0.30.0 RNASeqUtilities_1.1 [28] readr_1.3.1 knitr_1.27 workflowr_1.6.0</p>
<p>loaded via a namespace (and not attached): [1] nlme_3.1-140 bitops_1.0-6 fs_1.3.1 lubridate_1.7.4<br />
[5] bit64_0.9-7 webshot_0.5.1 httr_1.4.1 rprojroot_1.3-2<br />
[9] tools_3.6.1 backports_1.1.5 R6_2.4.1 DBI_1.1.0<br />
[13] lazyeval_0.2.2 colorspace_1.4-1 withr_2.1.2 tidyselect_0.2.5<br />
[17] gridExtra_2.3 bit_1.1-15.1 compiler_3.6.1 git2r_0.26.1<br />
[21] cli_2.0.1 rvest_0.3.4 xml2_1.2.2 labeling_0.3<br />
[25] scales_1.1.0 digest_0.6.23 rmarkdown_2.0 pkgconfig_2.0.3<br />
[29] htmltools_0.4.0 highr_0.8 readxl_1.3.1 htmlwidgets_1.5.1 [33] rlang_0.4.2 rstudioapi_0.10 RSQLite_2.2.0 shiny_1.3.2<br />
[37] farver_2.0.3 generics_0.0.2 jsonlite_1.6 crosstalk_1.0.0<br />
[41] RCurl_1.95-4.12 fansi_0.4.1 Rcpp_1.0.3 munsell_0.5.0<br />
[45] lifecycle_0.1.0 stringi_1.4.5 whisker_0.3-2 yaml_2.2.0<br />
[49] plyr_1.8.5 grid_3.6.1 blob_1.2.0 promises_1.0.1<br />
[53] crayon_1.3.4 lattice_0.20-38 haven_2.1.1 hms_0.5.3<br />
[57] zeallot_0.1.0 pillar_1.4.3 fastcluster_1.1.25 reshape2_1.4.3<br />
[61] glue_1.3.1 evaluate_0.14 modelr_0.1.4 vctrs_0.2.1<br />
[65] httpuv_1.5.1 cellranger_1.1.0 gtable_0.3.0 assertthat_0.2.1<br />
[69] xfun_0.12 mime_0.8 xtable_1.8-4 broom_0.5.2<br />
[73] later_1.0.0 viridisLite_0.3.0 memoise_1.1.0</p>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>


</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open')
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_').toLowerCase();
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
