<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Carl Murie" />

<meta name="date" content="2020-02-07" />

<title>Manuscript: M3 DMSO</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/lumen.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<script src="site_libs/kePrint-0.0.1/kePrint.js"></script>
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>




<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #ffffff;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  background: white;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row-fluid">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">mal067project</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/jdblischak/workflowr">
    <span class="fa fa-github"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Manuscript: M3 DMSO</h1>
<h4 class="author">Carl Murie</h4>
<h4 class="date">February 07, 2020</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span> workflowr <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2020-02-07
</p>
<p>
<strong>Checks:</strong> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7 <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>mal067project/</code> <span class="glyphicon glyphicon-question-sign" aria-hidden="true" title="This is the local directory in which the code in this file was executed."> </span>
</p>
<p>
This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a> analysis was created with <a
  href="https://github.com/jdblischak/workflowr">workflowr</a> (version 1.6.0). The <em>Checks</em> tab describes the reproducibility checks that were applied when the results were created. The <em>Past versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date </a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate" class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git repository, you know the exact version of the code that produced these results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the global environment can affect the analysis in your R Markdown file in unknown ways. For reproduciblity it’s best to always run the code in an empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20181108code"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Seed:</strong> <code>set.seed(20181108)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20181108code" class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20181108)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Session information:</strong> recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be confident that you successfully produced the results during this run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongdd65764"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Repository version:</strong> dd65764 </a>
</p>
</div>
<div id="strongRepositoryversionstrongdd65764" class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility. The version displayed above was the version of the Git repository at the time these results were generated. <br><br> Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .Rhistory
    Ignored:    analysis/.Rhistory

Untracked files:
    Untracked:  Faust_monocyte_boxplot.pdf
    Untracked:  analysis/DeRosaMonocyteProcessing.Rmd
    Untracked:  analysis/children/gsea_plot_delta_genes.Rmd
    Untracked:  analysis/children/stim_GSEA_interaction_case_effect.Rmd
    Untracked:  analysis/children/stim_GSEA_interaction_mono_effect.Rmd
    Untracked:  analysis/children/stim_GSEA_main_mono_effect.Rmd
    Untracked:  analysis/primary_interaction_mono_AMA.Rmd
    Untracked:  analysis/primary_interaction_mono_CSP.Rmd
    Untracked:  analysis/primary_interaction_mono_HBS.Rmd
    Untracked:  analysis/primary_main_M0_AMA.Rmd
    Untracked:  analysis/primary_main_M0_DMSO_bak.Rmd
    Untracked:  analysis/primary_main_mono_AMA.Rmd
    Untracked:  analysis/primary_main_mono_CSP.Rmd
    Untracked:  analysis/primary_main_mono_DMSO.Rmd
    Untracked:  analysis/primary_main_mono_HBS.Rmd
    Untracked:  analysis/process_deltas.Rmd
    Untracked:  analysis/secondary_covariates.Rmd
    Untracked:  code/ARDAP1_transcript.Rmd
    Untracked:  code/mal_utilities.R
    Untracked:  cor_table.RDS
    Untracked:  data/.~lock.170830-RTSS case control phenotyping.txt#
    Untracked:  data/170830-RTSS case control phenotyping.txt
    Untracked:  data/BTM_for_GSEA_20131008.gmt
    Untracked:  data/CD4_CSP_PFS.rds
    Untracked:  data/CD4_HBS_PFS.rds
    Untracked:  data/CD8_CSP_PFS.rds
    Untracked:  data/CD8_HBS_PFS.rds
    Untracked:  data/CiberSort_RTSS.csv
    Untracked:  data/DeRosaMonocyteFrequencies.csv
    Untracked:  data/ICS_Primary_MAL067.pptx
    Untracked:  data/PRIMARY_CD4_MAGNITUDE.rds
    Untracked:  data/Pulendran_GSEA_disease.csv
    Untracked:  data/RTSSPilotRNA-seqSAP_v4.pdf
    Untracked:  data/Vahey_GSEA_disease.csv
    Untracked:  data/btm_annotation_table.tsv
    Untracked:  data/btm_annotation_table_LNC.csv
    Untracked:  data/btm_annotation_table_LNC.ods
    Untracked:  data/faustOutputAllRtss_v02.csv
    Untracked:  data/ics_extra_vars_20180515.csv
    Untracked:  data/long_dis_DupCor.csv
    Untracked:  data/long_vac_DupCor.csv
    Untracked:  data/m067_seattle_data.RData
    Untracked:  data/phenoLineagePctLive.txt
    Untracked:  data/visit_DupCor.csv
    Untracked:  output/DMSO_M3_vaccine_disease_interaction.csv
    Untracked:  output/DMSO_M3_vaccine_disease_interaction_DEG.csv
    Untracked:  output/Fig2A_vaccine_both_dmso_M3.csv
    Untracked:  output/Fig2B_M3-M0_rtss.csv
    Untracked:  output/Fig3A_M3_case_control.csv
    Untracked:  output/Fig3B_M3-M0_rtss.csv
    Untracked:  output/GSEA_M0_ama1_vac_dis.csv
    Untracked:  output/GSEA_M0_csp_vac_dis.csv
    Untracked:  output/GSEA_M0_hbs_vac_dis.csv
    Untracked:  output/GSEA_M3_ama1_vac_dis.csv
    Untracked:  output/GSEA_M3_ama1_vac_dis_case.csv
    Untracked:  output/GSEA_M3_csp_vac_dis.csv
    Untracked:  output/GSEA_M3_hbs_vac_dis.csv
    Untracked:  output/M3-M0_both.csv
    Untracked:  output/M3-M0_by_vaccine.csv
    Untracked:  output/M3-M0_comp.csv
    Untracked:  output/M3-M0_disease_mal.csv
    Untracked:  output/M3-M0_disease_mal_comp.csv
    Untracked:  output/M3-M0_disease_mal_rtss.csv
    Untracked:  output/M3-M0_disease_nomal_comp.csv
    Untracked:  output/M3-M0_disease_nomal_rtss.csv
    Untracked:  output/M3-M0_rtss.csv
    Untracked:  output/M3_dmso_vaccine.csv
    Untracked:  output/cell_dist_mediods
    Untracked:  output/cibersort_cells.csv
    Untracked:  output/cibersort_cells.pdf
    Untracked:  output/disease_both_ama1_M0.csv
    Untracked:  output/disease_both_dmso_M0.csv
    Untracked:  output/disease_both_dmso_M3.csv
    Untracked:  output/disease_comp_dmso_M3.csv
    Untracked:  output/disease_ds_both_ama1_M0.csv
    Untracked:  output/disease_ds_both_dmso_M0.csv
    Untracked:  output/disease_ds_both_dmso_M3.csv
    Untracked:  output/disease_rtss_dmso_M3.csv
    Untracked:  output/dmso_M0_both_disease.csv
    Untracked:  output/dmso_M0_old_disease.csv
    Untracked:  output/dmso_M3_both_disease.csv
    Untracked:  output/dmso_M3_old_disease.csv
    Untracked:  output/dmso_M3_young_disease.csv
    Untracked:  output/figures/
    Untracked:  output/manuscript/
    Untracked:  output/sig_dis_genes_david.csv
    Untracked:  output/stim_deltas.rds
    Untracked:  output/stim_deltas_cpm_unlogged.rds
    Untracked:  output/vaccine_both_ama1_M0.csv
    Untracked:  output/vaccine_both_dmso_M0.csv
    Untracked:  output/vaccine_both_dmso_M3.csv
    Untracked:  secondary_faust.html
    Untracked:  test.pdf

Unstaged changes:
    Modified:   analysis/analysis_DMSO_M3_GSEA.Rmd
    Modified:   analysis/analysis_GSEA_M3_stimulations.Rmd
    Modified:   analysis/children/stim_GSEA_interaction_effect.Rmd
    Modified:   analysis/children/stim_GSEA_main_effect.Rmd
    Modified:   analysis/combine_GSEA.Rmd
    Modified:   analysis/experimental_design.Rmd
    Modified:   analysis/exploratory_ML_ratio.Rmd
    Modified:   analysis/exploratory_antibody.Rmd
    Modified:   analysis/exploratory_heno_gene_expression_disease_protection.Rmd
    Modified:   analysis/exploratory_monocyte_derosa.Rmd
    Modified:   analysis/index.Rmd
    Modified:   analysis/primary_Bagamoyo_GSEA.Rmd
    Modified:   analysis/primary_interaction_M3_AMA.Rmd
    Modified:   analysis/primary_main_M0_DMSO.Rmd
    Modified:   analysis/primary_main_M3_DMSO.Rmd
    Deleted:    analysis/secondary_disease_all_samples.Rmd
    Modified:   analysis/secondary_faust.Rmd

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the R Markdown and HTML files. If you’ve configured a remote Git repository (see <code>?wflow_git_remote</code>), click on the hyperlinks in the table below to view them.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
dd65764
</td>
<td>
carlmurie
</td>
<td>
2020-02-07
</td>
<td>
wflow_publish(“analysis/manuscript*.Rmd“)
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<div id="gsea-dmso-vaccine-main-effects" class="section level1">
<h1><span class="header-section-number">1</span> GSEA DMSO: vaccine main effects</h1>
<p><strong>vaccine linear model:</strong> ~plate + total_reads + age + vaccine<br />
<strong>coefficient:</strong> vaccinertss<br />
<strong>vaccine filtering :</strong> M3</p>
<table class="table table-striped table-hover table-condensed" style="width: auto !important; ">
<caption>
vaccine sample size
</caption>
<thead>
<tr>
<th style="border-bottom:hidden" colspan="1">
</th>
<th style="border-bottom:hidden" colspan="1">
</th>
<th style="border-bottom:hidden; padding-bottom:0; padding-left:3px;padding-right:3px;text-align: center; " colspan="2">
<div style="border-bottom: 1px solid #ddd; padding-bottom: 5px; ">
vaccine
</div>
</th>
<th style="border-bottom:hidden; padding-bottom:0; padding-left:3px;padding-right:3px;text-align: center; " colspan="3">
<div style="border-bottom: 1px solid #ddd; padding-bottom: 5px; ">
case
</div>
</th>
<th style="border-bottom:hidden; padding-bottom:0; padding-left:3px;padding-right:3px;text-align: center; " colspan="2">
<div style="border-bottom: 1px solid #ddd; padding-bottom: 5px; ">
age
</div>
</th>
</tr>
<tr>
<th style="text-align:left;">
</th>
<th style="text-align:center;">
total
</th>
<th style="text-align:center;">
comparator
</th>
<th style="text-align:center;">
rtss
</th>
<th style="text-align:center;">
control
</th>
<th style="text-align:center;">
case
</th>
<th style="text-align:center;">
neither
</th>
<th style="text-align:center;">
young
</th>
<th style="text-align:center;">
old
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
dmso
</td>
<td style="text-align:center;">
349
</td>
<td style="text-align:center;">
131
</td>
<td style="text-align:center;">
218
</td>
<td style="text-align:center;">
140
</td>
<td style="text-align:center;">
53
</td>
<td style="text-align:center;">
156
</td>
<td style="text-align:center;">
129
</td>
<td style="text-align:center;">
220
</td>
</tr>
</tbody>
</table>
<div id="vaccine-m3-rtss---comparator" class="section level2">
<h2><span class="header-section-number">1.1</span> vaccine M3 (RTS,S - comparator)</h2>
<div style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:600px; ">
<table class="table table-striped table-hover table-condensed" style="width: auto !important; ">
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
geneset
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
NGenes
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Direction
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
PValue
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
FDR
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
enriched in T cells (I) (M7.0)
</td>
<td style="text-align:left;">
56
</td>
<td style="text-align:left;">
Up
</td>
<td style="text-align:left;">
0.0000000
</td>
<td style="text-align:left;">
0.0000039
</td>
</tr>
<tr>
<td style="text-align:left;">
enriched in NK cells (I) (M7.2)
</td>
<td style="text-align:left;">
45
</td>
<td style="text-align:left;">
Up
</td>
<td style="text-align:left;">
0.0000000
</td>
<td style="text-align:left;">
0.0000039
</td>
</tr>
<tr>
<td style="text-align:left;">
T cell activation (III) (M7.4)
</td>
<td style="text-align:left;">
13
</td>
<td style="text-align:left;">
Up
</td>
<td style="text-align:left;">
0.0001154
</td>
<td style="text-align:left;">
0.0094920
</td>
</tr>
<tr>
<td style="text-align:left;">
T cell activation (I) (M7.1)
</td>
<td style="text-align:left;">
47
</td>
<td style="text-align:left;">
Up
</td>
<td style="text-align:left;">
0.0001699
</td>
<td style="text-align:left;">
0.0094920
</td>
</tr>
<tr>
<td style="text-align:left;">
TBA (M72.0)
</td>
<td style="text-align:left;">
21
</td>
<td style="text-align:left;">
Up
</td>
<td style="text-align:left;">
0.0001791
</td>
<td style="text-align:left;">
0.0094920
</td>
</tr>
<tr>
<td style="text-align:left;">
enriched in B cells (II) (M47.1)
</td>
<td style="text-align:left;">
32
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0004283
</td>
<td style="text-align:left;">
0.0189200
</td>
</tr>
<tr>
<td style="text-align:left;">
TBA (M174)
</td>
<td style="text-align:left;">
22
</td>
<td style="text-align:left;">
Up
</td>
<td style="text-align:left;">
0.0008051
</td>
<td style="text-align:left;">
0.0278100
</td>
</tr>
<tr>
<td style="text-align:left;">
B cell surface signature (S2)
</td>
<td style="text-align:left;">
70
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0009061
</td>
<td style="text-align:left;">
0.0278100
</td>
</tr>
<tr>
<td style="text-align:left;">
enriched in NK cells (II) (M61.0)
</td>
<td style="text-align:left;">
13
</td>
<td style="text-align:left;">
Up
</td>
<td style="text-align:left;">
0.0009445
</td>
<td style="text-align:left;">
0.0278100
</td>
</tr>
<tr>
<td style="text-align:left;">
enriched for ubiquitination (M138)
</td>
<td style="text-align:left;">
11
</td>
<td style="text-align:left;">
Up
</td>
<td style="text-align:left;">
0.0014190
</td>
<td style="text-align:left;">
0.0360300
</td>
</tr>
<tr>
<td style="text-align:left;">
respiratory electron transport chain (mitochondrion) (M231)
</td>
<td style="text-align:left;">
11
</td>
<td style="text-align:left;">
Up
</td>
<td style="text-align:left;">
0.0014960
</td>
<td style="text-align:left;">
0.0360300
</td>
</tr>
<tr>
<td style="text-align:left;">
immune activation - generic cluster (M37.0)
</td>
<td style="text-align:left;">
210
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0021740
</td>
<td style="text-align:left;">
0.0480200
</td>
</tr>
<tr>
<td style="text-align:left;">
TBA (M211)
</td>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
Up
</td>
<td style="text-align:left;">
0.0023870
</td>
<td style="text-align:left;">
0.0486500
</td>
</tr>
<tr>
<td style="text-align:left;">
plasma cells &amp; B cells, immunoglobulins (M156.0)
</td>
<td style="text-align:left;">
17
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0033960
</td>
<td style="text-align:left;">
0.0602200
</td>
</tr>
<tr>
<td style="text-align:left;">
enriched in B cells (I) (M47.0)
</td>
<td style="text-align:left;">
39
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0034090
</td>
<td style="text-align:left;">
0.0602200
</td>
</tr>
<tr>
<td style="text-align:left;">
respiratory electron transport chain (mitochondrion) (M219)
</td>
<td style="text-align:left;">
18
</td>
<td style="text-align:left;">
Up
</td>
<td style="text-align:left;">
0.0036810
</td>
<td style="text-align:left;">
0.0609700
</td>
</tr>
<tr>
<td style="text-align:left;">
TBA (M203)
</td>
<td style="text-align:left;">
13
</td>
<td style="text-align:left;">
Up
</td>
<td style="text-align:left;">
0.0040670
</td>
<td style="text-align:left;">
0.0613600
</td>
</tr>
<tr>
<td style="text-align:left;">
complement activation (I) (M112.0)
</td>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0043570
</td>
<td style="text-align:left;">
0.0613600
</td>
</tr>
<tr>
<td style="text-align:left;">
respiratory electron transport chain (mitochondrion) (M238)
</td>
<td style="text-align:left;">
17
</td>
<td style="text-align:left;">
Up
</td>
<td style="text-align:left;">
0.0046000
</td>
<td style="text-align:left;">
0.0613600
</td>
</tr>
<tr>
<td style="text-align:left;">
enriched in myeloid cells and monocytes (M81)
</td>
<td style="text-align:left;">
23
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0046310
</td>
<td style="text-align:left;">
0.0613600
</td>
</tr>
<tr>
<td style="text-align:left;">
T cell differentiation (M14)
</td>
<td style="text-align:left;">
12
</td>
<td style="text-align:left;">
Up
</td>
<td style="text-align:left;">
0.0059470
</td>
<td style="text-align:left;">
0.0750500
</td>
</tr>
<tr>
<td style="text-align:left;">
enriched in B cells (III) (M47.2)
</td>
<td style="text-align:left;">
22
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0067310
</td>
<td style="text-align:left;">
0.0810700
</td>
</tr>
<tr>
<td style="text-align:left;">
chemokines and inflammatory molecules in myeloid cells (M86.0)
</td>
<td style="text-align:left;">
17
</td>
<td style="text-align:left;">
Up
</td>
<td style="text-align:left;">
0.0074430
</td>
<td style="text-align:left;">
0.0857500
</td>
</tr>
<tr>
<td style="text-align:left;">
myeloid cell enriched receptors and transporters (M4.3)
</td>
<td style="text-align:left;">
26
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0077820
</td>
<td style="text-align:left;">
0.0859300
</td>
</tr>
<tr>
<td style="text-align:left;">
T cell differentiation via ITK and PKC (M18)
</td>
<td style="text-align:left;">
11
</td>
<td style="text-align:left;">
Up
</td>
<td style="text-align:left;">
0.0102200
</td>
<td style="text-align:left;">
0.1075000
</td>
</tr>
<tr>
<td style="text-align:left;">
enriched in B cells (VI) (M69)
</td>
<td style="text-align:left;">
18
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0105500
</td>
<td style="text-align:left;">
0.1075000
</td>
</tr>
<tr>
<td style="text-align:left;">
T cell signaling and costimulation (M44)
</td>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
Up
</td>
<td style="text-align:left;">
0.0112200
</td>
<td style="text-align:left;">
0.1102000
</td>
</tr>
<tr>
<td style="text-align:left;">
TBA (M185)
</td>
<td style="text-align:left;">
14
</td>
<td style="text-align:left;">
Up
</td>
<td style="text-align:left;">
0.0119300
</td>
<td style="text-align:left;">
0.1107000
</td>
</tr>
<tr>
<td style="text-align:left;">
enriched in NK cells (III) (M157)
</td>
<td style="text-align:left;">
13
</td>
<td style="text-align:left;">
Up
</td>
<td style="text-align:left;">
0.0123100
</td>
<td style="text-align:left;">
0.1107000
</td>
</tr>
<tr>
<td style="text-align:left;">
mitotic cell cycle in stimulated CD4 T cells (M4.9)
</td>
<td style="text-align:left;">
13
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0125300
</td>
<td style="text-align:left;">
0.1107000
</td>
</tr>
<tr>
<td style="text-align:left;">
cell adhesion (GO) (M117)
</td>
<td style="text-align:left;">
20
</td>
<td style="text-align:left;">
Up
</td>
<td style="text-align:left;">
0.0134300
</td>
<td style="text-align:left;">
0.1148000
</td>
</tr>
<tr>
<td style="text-align:left;">
TBA (M201)
</td>
<td style="text-align:left;">
14
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0144600
</td>
<td style="text-align:left;">
0.1161000
</td>
</tr>
<tr>
<td style="text-align:left;">
signaling in T cells (II) (M35.1)
</td>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
Up
</td>
<td style="text-align:left;">
0.0148200
</td>
<td style="text-align:left;">
0.1161000
</td>
</tr>
<tr>
<td style="text-align:left;">
translation initiation factor 3 complex (M245)
</td>
<td style="text-align:left;">
11
</td>
<td style="text-align:left;">
Up
</td>
<td style="text-align:left;">
0.0148900
</td>
<td style="text-align:left;">
0.1161000
</td>
</tr>
<tr>
<td style="text-align:left;">
intracellular transport (M147)
</td>
<td style="text-align:left;">
17
</td>
<td style="text-align:left;">
Up
</td>
<td style="text-align:left;">
0.0154200
</td>
<td style="text-align:left;">
0.1168000
</td>
</tr>
<tr>
<td style="text-align:left;">
TBA (M72.1)
</td>
<td style="text-align:left;">
16
</td>
<td style="text-align:left;">
Up
</td>
<td style="text-align:left;">
0.0163500
</td>
<td style="text-align:left;">
0.1179000
</td>
</tr>
<tr>
<td style="text-align:left;">
respiratory electron transport chain (mitochondrion) (M216)
</td>
<td style="text-align:left;">
12
</td>
<td style="text-align:left;">
Up
</td>
<td style="text-align:left;">
0.0166700
</td>
<td style="text-align:left;">
0.1179000
</td>
</tr>
<tr>
<td style="text-align:left;">
T cell differentiation (Th2) (M19)
</td>
<td style="text-align:left;">
15
</td>
<td style="text-align:left;">
Up
</td>
<td style="text-align:left;">
0.0173600
</td>
<td style="text-align:left;">
0.1179000
</td>
</tr>
<tr>
<td style="text-align:left;">
signaling in T cells (I) (M35.0)
</td>
<td style="text-align:left;">
13
</td>
<td style="text-align:left;">
Up
</td>
<td style="text-align:left;">
0.0173600
</td>
<td style="text-align:left;">
0.1179000
</td>
</tr>
<tr>
<td style="text-align:left;">
chromosome Y linked (M240)
</td>
<td style="text-align:left;">
9
</td>
<td style="text-align:left;">
Up
</td>
<td style="text-align:left;">
0.0182300
</td>
<td style="text-align:left;">
0.1193000
</td>
</tr>
<tr>
<td style="text-align:left;">
leukocyte activation and migration (M45)
</td>
<td style="text-align:left;">
11
</td>
<td style="text-align:left;">
Up
</td>
<td style="text-align:left;">
0.0184500
</td>
<td style="text-align:left;">
0.1193000
</td>
</tr>
<tr>
<td style="text-align:left;">
heme biosynthesis (II) (M222)
</td>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0191200
</td>
<td style="text-align:left;">
0.1206000
</td>
</tr>
<tr>
<td style="text-align:left;">
leukocyte migration (M88.0)
</td>
<td style="text-align:left;">
12
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0207600
</td>
<td style="text-align:left;">
0.1229000
</td>
</tr>
<tr>
<td style="text-align:left;">
spliceosome (M250)
</td>
<td style="text-align:left;">
12
</td>
<td style="text-align:left;">
Up
</td>
<td style="text-align:left;">
0.0215600
</td>
<td style="text-align:left;">
0.1229000
</td>
</tr>
<tr>
<td style="text-align:left;">
Naive B cell surface signature (S8)
</td>
<td style="text-align:left;">
24
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0216200
</td>
<td style="text-align:left;">
0.1229000
</td>
</tr>
<tr>
<td style="text-align:left;">
transcription elongation, RNA polymerase II (M234)
</td>
<td style="text-align:left;">
13
</td>
<td style="text-align:left;">
Up
</td>
<td style="text-align:left;">
0.0217800
</td>
<td style="text-align:left;">
0.1229000
</td>
</tr>
<tr>
<td style="text-align:left;">
TBA (M194)
</td>
<td style="text-align:left;">
13
</td>
<td style="text-align:left;">
Up
</td>
<td style="text-align:left;">
0.0217900
</td>
<td style="text-align:left;">
0.1229000
</td>
</tr>
<tr>
<td style="text-align:left;">
T cell activation (II) (M7.3)
</td>
<td style="text-align:left;">
29
</td>
<td style="text-align:left;">
Up
</td>
<td style="text-align:left;">
0.0233100
</td>
<td style="text-align:left;">
0.1275000
</td>
</tr>
<tr>
<td style="text-align:left;">
enriched in G-protein coupled receptors (M130)
</td>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
Up
</td>
<td style="text-align:left;">
0.0235700
</td>
<td style="text-align:left;">
0.1275000
</td>
</tr>
<tr>
<td style="text-align:left;">
T cell activation and signaling (M5.1)
</td>
<td style="text-align:left;">
22
</td>
<td style="text-align:left;">
Up
</td>
<td style="text-align:left;">
0.0268800
</td>
<td style="text-align:left;">
0.1424000
</td>
</tr>
<tr>
<td style="text-align:left;">
antigen processing and presentation (M200)
</td>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0302800
</td>
<td style="text-align:left;">
0.1547000
</td>
</tr>
<tr>
<td style="text-align:left;">
DC surface signature (S5)
</td>
<td style="text-align:left;">
54
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0304700
</td>
<td style="text-align:left;">
0.1547000
</td>
</tr>
<tr>
<td style="text-align:left;">
leukocyte differentiation (M160)
</td>
<td style="text-align:left;">
14
</td>
<td style="text-align:left;">
Up
</td>
<td style="text-align:left;">
0.0320200
</td>
<td style="text-align:left;">
0.1547000
</td>
</tr>
<tr>
<td style="text-align:left;">
TBA (M229)
</td>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
Up
</td>
<td style="text-align:left;">
0.0320400
</td>
<td style="text-align:left;">
0.1547000
</td>
</tr>
<tr>
<td style="text-align:left;">
TBA (M205)
</td>
<td style="text-align:left;">
11
</td>
<td style="text-align:left;">
Up
</td>
<td style="text-align:left;">
0.0321200
</td>
<td style="text-align:left;">
0.1547000
</td>
</tr>
<tr>
<td style="text-align:left;">
transmembrane transport (I) (M87)
</td>
<td style="text-align:left;">
19
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0340400
</td>
<td style="text-align:left;">
0.1595000
</td>
</tr>
<tr>
<td style="text-align:left;">
proinflammatory cytokines and chemokines (M29)
</td>
<td style="text-align:left;">
9
</td>
<td style="text-align:left;">
Up
</td>
<td style="text-align:left;">
0.0343100
</td>
<td style="text-align:left;">
0.1595000
</td>
</tr>
<tr>
<td style="text-align:left;">
proteasome (M226)
</td>
<td style="text-align:left;">
12
</td>
<td style="text-align:left;">
Up
</td>
<td style="text-align:left;">
0.0360900
</td>
<td style="text-align:left;">
0.1649000
</td>
</tr>
<tr>
<td style="text-align:left;">
small GTPase mediated signal transduction (M215)
</td>
<td style="text-align:left;">
16
</td>
<td style="text-align:left;">
Up
</td>
<td style="text-align:left;">
0.0381500
</td>
<td style="text-align:left;">
0.1714000
</td>
</tr>
<tr>
<td style="text-align:left;">
chaperonin mediated protein folding (I) (M204.0)
</td>
<td style="text-align:left;">
14
</td>
<td style="text-align:left;">
Up
</td>
<td style="text-align:left;">
0.0402300
</td>
<td style="text-align:left;">
0.1777000
</td>
</tr>
<tr>
<td style="text-align:left;">
enriched in monocytes (II) (M11.0)
</td>
<td style="text-align:left;">
146
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0422200
</td>
<td style="text-align:left;">
0.1834000
</td>
</tr>
<tr>
<td style="text-align:left;">
cell cycle, ATP binding (M144)
</td>
<td style="text-align:left;">
16
</td>
<td style="text-align:left;">
Up
</td>
<td style="text-align:left;">
0.0435000
</td>
<td style="text-align:left;">
0.1859000
</td>
</tr>
<tr>
<td style="text-align:left;">
nucleotide metabolism (M181)
</td>
<td style="text-align:left;">
9
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0465700
</td>
<td style="text-align:left;">
0.1922000
</td>
</tr>
<tr>
<td style="text-align:left;">
TBA (M137)
</td>
<td style="text-align:left;">
15
</td>
<td style="text-align:left;">
Up
</td>
<td style="text-align:left;">
0.0472200
</td>
<td style="text-align:left;">
0.1922000
</td>
</tr>
<tr>
<td style="text-align:left;">
enriched in naive and memory B cells (M83)
</td>
<td style="text-align:left;">
9
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:left;">
0.0476900
</td>
<td style="text-align:left;">
0.1922000
</td>
</tr>
<tr>
<td style="text-align:left;">
chaperonin mediated protein folding (II) (M204.1)
</td>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
Up
</td>
<td style="text-align:left;">
0.0478700
</td>
<td style="text-align:left;">
0.1922000
</td>
</tr>
</tbody>
</table>
</div>
<p><img src="figure/manuscript_Fig2_vaccine.Rmd/heatme_vac-1.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="vaccine-m3-mo" class="section level2 tabset tabset-fade tabset-pills">
<h2><span class="header-section-number">1.2</span> vaccine M3-MO</h2>
<p>Only Bagamoya samples with the dmso stimulation are used. A consequence of this is that only subjects from the old age group are included. Additionally only paired samples by subject are included.</p>
<table class="table table-striped table-hover table-condensed" style="width: auto !important; ">
<thead>
<tr>
<th style="border-bottom:hidden" colspan="1">
</th>
<th style="border-bottom:hidden" colspan="1">
</th>
<th style="border-bottom:hidden; padding-bottom:0; padding-left:3px;padding-right:3px;text-align: center; " colspan="2">
<div style="border-bottom: 1px solid #ddd; padding-bottom: 5px; ">
vaccine
</div>
</th>
<th style="border-bottom:hidden; padding-bottom:0; padding-left:3px;padding-right:3px;text-align: center; " colspan="2">
<div style="border-bottom: 1px solid #ddd; padding-bottom: 5px; ">
disease
</div>
</th>
<th style="border-bottom:hidden; padding-bottom:0; padding-left:3px;padding-right:3px;text-align: center; " colspan="2">
<div style="border-bottom: 1px solid #ddd; padding-bottom: 5px; ">
age
</div>
</th>
<th style="border-bottom:hidden; padding-bottom:0; padding-left:3px;padding-right:3px;text-align: center; " colspan="2">
<div style="border-bottom: 1px solid #ddd; padding-bottom: 5px; ">
visit
</div>
</th>
<th style="border-bottom:hidden; padding-bottom:0; padding-left:3px;padding-right:3px;text-align: center; " colspan="3">
<div style="border-bottom: 1px solid #ddd; padding-bottom: 5px; ">
case
</div>
</th>
</tr>
<tr>
<th style="text-align:left;">
</th>
<th style="text-align:center;">
total
</th>
<th style="text-align:center;">
comparator
</th>
<th style="text-align:center;">
rtss
</th>
<th style="text-align:center;">
X0
</th>
<th style="text-align:center;">
X1
</th>
<th style="text-align:center;">
young
</th>
<th style="text-align:center;">
old
</th>
<th style="text-align:center;">
M0
</th>
<th style="text-align:center;">
M3
</th>
<th style="text-align:center;">
control
</th>
<th style="text-align:center;">
case
</th>
<th style="text-align:center;">
neither
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
dmso
</td>
<td style="text-align:center;">
222
</td>
<td style="text-align:center;">
70
</td>
<td style="text-align:center;">
152
</td>
<td style="text-align:center;">
194
</td>
<td style="text-align:center;">
28
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
222
</td>
<td style="text-align:center;">
111
</td>
<td style="text-align:center;">
111
</td>
<td style="text-align:center;">
81
</td>
<td style="text-align:center;">
26
</td>
<td style="text-align:center;">
115
</td>
</tr>
</tbody>
</table>
<p>formula: ~plate + total_reads + pid + visit<br />
coefficient: visitM3</p>
<div id="rtss-and-comparator-together" class="section level3">
<h3><span class="header-section-number">1.2.1</span> rts,s and comparator together</h3>
<table class="table table-striped table-hover table-condensed" style="width: auto !important; ">
<thead>
<tr>
<th style="text-align:center;">
geneset
</th>
<th style="text-align:center;">
NGenes
</th>
<th style="text-align:center;">
Direction
</th>
<th style="text-align:center;">
PValue
</th>
<th style="text-align:center;">
FDR
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:center;">
type I interferon response (M127)
</td>
<td style="text-align:center;">
12
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.000
</td>
<td style="text-align:center;">
0.000
</td>
</tr>
<tr>
<td style="text-align:center;">
chemokines and inflammatory molecules in myeloid cells (M86.0)
</td>
<td style="text-align:center;">
17
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.000
</td>
<td style="text-align:center;">
0.000
</td>
</tr>
<tr>
<td style="text-align:center;">
enriched in monocytes (II) (M11.0)
</td>
<td style="text-align:center;">
146
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.000
</td>
<td style="text-align:center;">
0.000
</td>
</tr>
<tr>
<td style="text-align:center;">
proinflammatory dendritic cell, myeloid cell response (M86.1)
</td>
<td style="text-align:center;">
11
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.000
</td>
<td style="text-align:center;">
0.000
</td>
</tr>
<tr>
<td style="text-align:center;">
cytokines - recepters cluster (M115)
</td>
<td style="text-align:center;">
10
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.000
</td>
<td style="text-align:center;">
0.000
</td>
</tr>
<tr>
<td style="text-align:center;">
RIG-1 like receptor signaling (M68)
</td>
<td style="text-align:center;">
10
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.000
</td>
<td style="text-align:center;">
0.000
</td>
</tr>
<tr>
<td style="text-align:center;">
cell cycle and growth arrest (M31)
</td>
<td style="text-align:center;">
11
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.000
</td>
<td style="text-align:center;">
0.000
</td>
</tr>
<tr>
<td style="text-align:center;">
regulation of transcription, transcription factors (M213)
</td>
<td style="text-align:center;">
17
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.000
</td>
<td style="text-align:center;">
0.000
</td>
</tr>
<tr>
<td style="text-align:center;">
Monocyte surface signature (S4)
</td>
<td style="text-align:center;">
69
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.000
</td>
<td style="text-align:center;">
0.000
</td>
</tr>
<tr>
<td style="text-align:center;">
chemokine cluster (I) (M27.0)
</td>
<td style="text-align:center;">
20
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.000
</td>
<td style="text-align:center;">
0.000
</td>
</tr>
<tr>
<td style="text-align:center;">
innate antiviral response (M150)
</td>
<td style="text-align:center;">
12
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.000
</td>
<td style="text-align:center;">
0.000
</td>
</tr>
<tr>
<td style="text-align:center;">
cell cycle and transcription (M4.0)
</td>
<td style="text-align:center;">
226
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.000
</td>
<td style="text-align:center;">
0.000
</td>
</tr>
<tr>
<td style="text-align:center;">
proinflammatory cytokines and chemokines (M29)
</td>
<td style="text-align:center;">
9
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.000
</td>
<td style="text-align:center;">
0.000
</td>
</tr>
<tr>
<td style="text-align:center;">
antiviral IFN signature (M75)
</td>
<td style="text-align:center;">
19
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.000
</td>
<td style="text-align:center;">
0.000
</td>
</tr>
<tr>
<td style="text-align:center;">
activated dendritic cells (M67)
</td>
<td style="text-align:center;">
9
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.000
</td>
<td style="text-align:center;">
0.000
</td>
</tr>
<tr>
<td style="text-align:center;">
extracellular matrix, complement (M140)
</td>
<td style="text-align:center;">
7
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.000
</td>
<td style="text-align:center;">
0.000
</td>
</tr>
<tr>
<td style="text-align:center;">
signaling in T cells (II) (M35.1)
</td>
<td style="text-align:center;">
10
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.000
</td>
<td style="text-align:center;">
0.001
</td>
</tr>
<tr>
<td style="text-align:center;">
Resting dendritic cell surface signature (S10)
</td>
<td style="text-align:center;">
43
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.000
</td>
<td style="text-align:center;">
0.001
</td>
</tr>
<tr>
<td style="text-align:center;">
viral sensing &amp; immunity; IRF2 targets network (II) (M111.1)
</td>
<td style="text-align:center;">
11
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.000
</td>
<td style="text-align:center;">
0.001
</td>
</tr>
<tr>
<td style="text-align:center;">
myeloid, dendritic cell activation via NFkB (II) (M43.1)
</td>
<td style="text-align:center;">
13
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.000
</td>
<td style="text-align:center;">
0.001
</td>
</tr>
<tr>
<td style="text-align:center;">
leukocyte differentiation (M160)
</td>
<td style="text-align:center;">
14
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.000
</td>
<td style="text-align:center;">
0.001
</td>
</tr>
<tr>
<td style="text-align:center;">
myeloid, dendritic cell activation via NFkB (I) (M43.0)
</td>
<td style="text-align:center;">
12
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.000
</td>
<td style="text-align:center;">
0.002
</td>
</tr>
<tr>
<td style="text-align:center;">
myeloid cell enriched receptors and transporters (M4.3)
</td>
<td style="text-align:center;">
26
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.000
</td>
<td style="text-align:center;">
0.002
</td>
</tr>
<tr>
<td style="text-align:center;">
putative targets of PAX3 (M89.1)
</td>
<td style="text-align:center;">
9
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.000
</td>
<td style="text-align:center;">
0.003
</td>
</tr>
<tr>
<td style="text-align:center;">
TBA (M72.0)
</td>
<td style="text-align:center;">
21
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.000
</td>
<td style="text-align:center;">
0.004
</td>
</tr>
<tr>
<td style="text-align:center;">
immuregulation - monocytes, T and B cells (M57)
</td>
<td style="text-align:center;">
10
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.000
</td>
<td style="text-align:center;">
0.005
</td>
</tr>
<tr>
<td style="text-align:center;">
TBA (M72.1)
</td>
<td style="text-align:center;">
16
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.001
</td>
<td style="text-align:center;">
0.005
</td>
</tr>
<tr>
<td style="text-align:center;">
enriched in activated dendritic cells/monocytes (M64)
</td>
<td style="text-align:center;">
17
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.001
</td>
<td style="text-align:center;">
0.005
</td>
</tr>
<tr>
<td style="text-align:center;">
TBA (M121)
</td>
<td style="text-align:center;">
10
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.001
</td>
<td style="text-align:center;">
0.006
</td>
</tr>
<tr>
<td style="text-align:center;">
innate activation by cytosolic DNA sensing (M13)
</td>
<td style="text-align:center;">
11
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.001
</td>
<td style="text-align:center;">
0.007
</td>
</tr>
<tr>
<td style="text-align:center;">
TBA (M190)
</td>
<td style="text-align:center;">
10
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.001
</td>
<td style="text-align:center;">
0.008
</td>
</tr>
<tr>
<td style="text-align:center;">
putative targets of PAX3 (M89.0)
</td>
<td style="text-align:center;">
12
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.001
</td>
<td style="text-align:center;">
0.011
</td>
</tr>
<tr>
<td style="text-align:center;">
signaling in T cells (I) (M35.0)
</td>
<td style="text-align:center;">
13
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.001
</td>
<td style="text-align:center;">
0.011
</td>
</tr>
<tr>
<td style="text-align:center;">
TBA (source: B cells) (M152.0)
</td>
<td style="text-align:center;">
6
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.002
</td>
<td style="text-align:center;">
0.014
</td>
</tr>
<tr>
<td style="text-align:center;">
growth factor induced, enriched in nuclear receptor subfamily 4 (M94)
</td>
<td style="text-align:center;">
7
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.002
</td>
<td style="text-align:center;">
0.018
</td>
</tr>
<tr>
<td style="text-align:center;">
enriched in monocytes (surface) (M118.1)
</td>
<td style="text-align:center;">
17
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.003
</td>
<td style="text-align:center;">
0.019
</td>
</tr>
<tr>
<td style="text-align:center;">
TBA (M174)
</td>
<td style="text-align:center;">
22
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.003
</td>
<td style="text-align:center;">
0.021
</td>
</tr>
<tr>
<td style="text-align:center;">
CORO1A-DEF6 network (II) (M32.4)
</td>
<td style="text-align:center;">
13
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.004
</td>
<td style="text-align:center;">
0.025
</td>
</tr>
<tr>
<td style="text-align:center;">
TBA (M72.2)
</td>
<td style="text-align:center;">
12
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.004
</td>
<td style="text-align:center;">
0.025
</td>
</tr>
<tr>
<td style="text-align:center;">
chemokines and receptors (M38)
</td>
<td style="text-align:center;">
11
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.004
</td>
<td style="text-align:center;">
0.025
</td>
</tr>
<tr>
<td style="text-align:center;">
TBA (M177.1)
</td>
<td style="text-align:center;">
10
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.004
</td>
<td style="text-align:center;">
0.025
</td>
</tr>
<tr>
<td style="text-align:center;">
Plasma cell surface signature (S3)
</td>
<td style="text-align:center;">
10
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.005
</td>
<td style="text-align:center;">
0.028
</td>
</tr>
<tr>
<td style="text-align:center;">
TBA (M203)
</td>
<td style="text-align:center;">
13
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.005
</td>
<td style="text-align:center;">
0.028
</td>
</tr>
<tr>
<td style="text-align:center;">
enriched in activated dendritic cells (II) (M165)
</td>
<td style="text-align:center;">
29
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.005
</td>
<td style="text-align:center;">
0.028
</td>
</tr>
<tr>
<td style="text-align:center;">
lysosome (M209)
</td>
<td style="text-align:center;">
7
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.005
</td>
<td style="text-align:center;">
0.028
</td>
</tr>
<tr>
<td style="text-align:center;">
TLR and inflammatory signaling (M16)
</td>
<td style="text-align:center;">
34
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.005
</td>
<td style="text-align:center;">
0.028
</td>
</tr>
<tr>
<td style="text-align:center;">
TBA (M194)
</td>
<td style="text-align:center;">
13
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.005
</td>
<td style="text-align:center;">
0.028
</td>
</tr>
<tr>
<td style="text-align:center;">
CD1 and other DC receptors (M50)
</td>
<td style="text-align:center;">
6
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.005
</td>
<td style="text-align:center;">
0.028
</td>
</tr>
<tr>
<td style="text-align:center;">
cell activation (IL15, IL23, TNF) (M24)
</td>
<td style="text-align:center;">
12
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.005
</td>
<td style="text-align:center;">
0.028
</td>
</tr>
<tr>
<td style="text-align:center;">
immune activation - generic cluster (M37.0)
</td>
<td style="text-align:center;">
210
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.007
</td>
<td style="text-align:center;">
0.037
</td>
</tr>
<tr>
<td style="text-align:center;">
proteasome (M226)
</td>
<td style="text-align:center;">
12
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.007
</td>
<td style="text-align:center;">
0.038
</td>
</tr>
<tr>
<td style="text-align:center;">
CORO1A-DEF6 network (I) (M32.2)
</td>
<td style="text-align:center;">
17
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.008
</td>
<td style="text-align:center;">
0.039
</td>
</tr>
<tr>
<td style="text-align:center;">
B cell surface signature (S2)
</td>
<td style="text-align:center;">
70
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.008
</td>
<td style="text-align:center;">
0.040
</td>
</tr>
<tr>
<td style="text-align:center;">
TBA (M104)
</td>
<td style="text-align:center;">
6
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.010
</td>
<td style="text-align:center;">
0.047
</td>
</tr>
<tr>
<td style="text-align:center;">
TBA (M141)
</td>
<td style="text-align:center;">
7
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.010
</td>
<td style="text-align:center;">
0.048
</td>
</tr>
<tr>
<td style="text-align:center;">
glycerophospholipid metabolism (M114.1)
</td>
<td style="text-align:center;">
10
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.011
</td>
<td style="text-align:center;">
0.050
</td>
</tr>
<tr>
<td style="text-align:center;">
mitosis (TF motif CCAATNNSNNNGCG) (M169)
</td>
<td style="text-align:center;">
14
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.011
</td>
<td style="text-align:center;">
0.051
</td>
</tr>
<tr>
<td style="text-align:center;">
TBA (source: memory B cells) (M152.2)
</td>
<td style="text-align:center;">
6
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.012
</td>
<td style="text-align:center;">
0.054
</td>
</tr>
<tr>
<td style="text-align:center;">
leukocyte migration (M88.0)
</td>
<td style="text-align:center;">
12
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.013
</td>
<td style="text-align:center;">
0.058
</td>
</tr>
<tr>
<td style="text-align:center;">
platelet activation (I) (M32.0)
</td>
<td style="text-align:center;">
20
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.015
</td>
<td style="text-align:center;">
0.066
</td>
</tr>
<tr>
<td style="text-align:center;">
enriched in NK cells (I) (M7.2)
</td>
<td style="text-align:center;">
45
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.016
</td>
<td style="text-align:center;">
0.066
</td>
</tr>
<tr>
<td style="text-align:center;">
enriched for TF motif PAX3 (M179)
</td>
<td style="text-align:center;">
8
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.016
</td>
<td style="text-align:center;">
0.066
</td>
</tr>
<tr>
<td style="text-align:center;">
enriched in monocytes (IV) (M118.0)
</td>
<td style="text-align:center;">
48
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.016
</td>
<td style="text-align:center;">
0.066
</td>
</tr>
<tr>
<td style="text-align:center;">
TBA (M229)
</td>
<td style="text-align:center;">
10
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.017
</td>
<td style="text-align:center;">
0.071
</td>
</tr>
<tr>
<td style="text-align:center;">
extracellular matrix (II) (M2.1)
</td>
<td style="text-align:center;">
18
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.018
</td>
<td style="text-align:center;">
0.072
</td>
</tr>
<tr>
<td style="text-align:center;">
TBA (M32.6)
</td>
<td style="text-align:center;">
12
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.018
</td>
<td style="text-align:center;">
0.072
</td>
</tr>
<tr>
<td style="text-align:center;">
Memory B cell surface signature (S9)
</td>
<td style="text-align:center;">
13
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.018
</td>
<td style="text-align:center;">
0.073
</td>
</tr>
<tr>
<td style="text-align:center;">
E2F transcription factor network (M8)
</td>
<td style="text-align:center;">
7
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.022
</td>
<td style="text-align:center;">
0.087
</td>
</tr>
<tr>
<td style="text-align:center;">
complement activation (I) (M112.0)
</td>
<td style="text-align:center;">
10
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.023
</td>
<td style="text-align:center;">
0.090
</td>
</tr>
<tr>
<td style="text-align:center;">
TBA (M32.7)
</td>
<td style="text-align:center;">
10
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.025
</td>
<td style="text-align:center;">
0.093
</td>
</tr>
<tr>
<td style="text-align:center;">
TBA (M120)
</td>
<td style="text-align:center;">
11
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.025
</td>
<td style="text-align:center;">
0.093
</td>
</tr>
<tr>
<td style="text-align:center;">
intracellular transport (M147)
</td>
<td style="text-align:center;">
17
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.025
</td>
<td style="text-align:center;">
0.093
</td>
</tr>
<tr>
<td style="text-align:center;">
Naive B cell surface signature (S8)
</td>
<td style="text-align:center;">
24
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.028
</td>
<td style="text-align:center;">
0.101
</td>
</tr>
<tr>
<td style="text-align:center;">
TBA (M80)
</td>
<td style="text-align:center;">
12
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.028
</td>
<td style="text-align:center;">
0.101
</td>
</tr>
<tr>
<td style="text-align:center;">
cell division (stimulated CD4+ T cells) (M46)
</td>
<td style="text-align:center;">
13
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.029
</td>
<td style="text-align:center;">
0.101
</td>
</tr>
<tr>
<td style="text-align:center;">
TBA (M137)
</td>
<td style="text-align:center;">
15
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.030
</td>
<td style="text-align:center;">
0.104
</td>
</tr>
<tr>
<td style="text-align:center;">
platelet activation (II) (M32.1)
</td>
<td style="text-align:center;">
20
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.033
</td>
<td style="text-align:center;">
0.114
</td>
</tr>
<tr>
<td style="text-align:center;">
TBA (M242)
</td>
<td style="text-align:center;">
11
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.035
</td>
<td style="text-align:center;">
0.120
</td>
</tr>
<tr>
<td style="text-align:center;">
cell cycle (I) (M4.1)
</td>
<td style="text-align:center;">
89
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.037
</td>
<td style="text-align:center;">
0.124
</td>
</tr>
<tr>
<td style="text-align:center;">
transmembrane transport (I) (M87)
</td>
<td style="text-align:center;">
19
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.038
</td>
<td style="text-align:center;">
0.125
</td>
</tr>
<tr>
<td style="text-align:center;">
chaperonin mediated protein folding (I) (M204.0)
</td>
<td style="text-align:center;">
14
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.039
</td>
<td style="text-align:center;">
0.126
</td>
</tr>
<tr>
<td style="text-align:center;">
chemokine cluster (II) (M27.1)
</td>
<td style="text-align:center;">
13
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.040
</td>
<td style="text-align:center;">
0.128
</td>
</tr>
<tr>
<td style="text-align:center;">
cell cycle, ATP binding (M144)
</td>
<td style="text-align:center;">
16
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.041
</td>
<td style="text-align:center;">
0.130
</td>
</tr>
<tr>
<td style="text-align:center;">
enriched in neutrophils (I) (M37.1)
</td>
<td style="text-align:center;">
29
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.042
</td>
<td style="text-align:center;">
0.133
</td>
</tr>
<tr>
<td style="text-align:center;">
KLF12 targets network (M32.3)
</td>
<td style="text-align:center;">
15
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.049
</td>
<td style="text-align:center;">
0.152
</td>
</tr>
<tr>
<td style="text-align:center;">
enriched in B cells (VI) (M69)
</td>
<td style="text-align:center;">
18
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.050
</td>
<td style="text-align:center;">
0.153
</td>
</tr>
<tr>
<td style="text-align:center;">
AP-1 transcription factor network (M20)
</td>
<td style="text-align:center;">
13
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.050
</td>
<td style="text-align:center;">
0.154
</td>
</tr>
<tr>
<td style="text-align:center;">
TBA (M153)
</td>
<td style="text-align:center;">
15
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.056
</td>
<td style="text-align:center;">
0.169
</td>
</tr>
<tr>
<td style="text-align:center;">
lysosomal/endosomal proteins (M139)
</td>
<td style="text-align:center;">
10
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.058
</td>
<td style="text-align:center;">
0.172
</td>
</tr>
<tr>
<td style="text-align:center;">
DNA repair (M76)
</td>
<td style="text-align:center;">
19
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.059
</td>
<td style="text-align:center;">
0.173
</td>
</tr>
<tr>
<td style="text-align:center;">
enriched in B cells (I) (M47.0)
</td>
<td style="text-align:center;">
39
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.061
</td>
<td style="text-align:center;">
0.177
</td>
</tr>
<tr>
<td style="text-align:center;">
TBA (M32.5)
</td>
<td style="text-align:center;">
14
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.062
</td>
<td style="text-align:center;">
0.177
</td>
</tr>
<tr>
<td style="text-align:center;">
TBA (M180)
</td>
<td style="text-align:center;">
11
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.062
</td>
<td style="text-align:center;">
0.177
</td>
</tr>
<tr>
<td style="text-align:center;">
enriched in cell cycle (M167)
</td>
<td style="text-align:center;">
14
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.064
</td>
<td style="text-align:center;">
0.179
</td>
</tr>
<tr>
<td style="text-align:center;">
TBA (M131)
</td>
<td style="text-align:center;">
12
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.064
</td>
<td style="text-align:center;">
0.179
</td>
</tr>
<tr>
<td style="text-align:center;">
regulation of localization (GO) (M63)
</td>
<td style="text-align:center;">
11
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.066
</td>
<td style="text-align:center;">
0.184
</td>
</tr>
<tr>
<td style="text-align:center;">
enriched in NK cells (III) (M157)
</td>
<td style="text-align:center;">
13
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.070
</td>
<td style="text-align:center;">
0.190
</td>
</tr>
<tr>
<td style="text-align:center;">
nuclear pore, transport; mRNA splicing, processing (M143)
</td>
<td style="text-align:center;">
11
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.073
</td>
<td style="text-align:center;">
0.198
</td>
</tr>
</tbody>
</table>
</div>
<div id="rtss" class="section level3">
<h3><span class="header-section-number">1.2.2</span> rts,s</h3>
<table class="table table-striped table-hover table-condensed" style="width: auto !important; ">
<thead>
<tr>
<th style="text-align:center;">
geneset
</th>
<th style="text-align:center;">
NGenes
</th>
<th style="text-align:center;">
Direction
</th>
<th style="text-align:center;">
PValue
</th>
<th style="text-align:center;">
FDR
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:center;">
chemokines and inflammatory molecules in myeloid cells (M86.0)
</td>
<td style="text-align:center;">
17
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.000
</td>
<td style="text-align:center;">
0.000
</td>
</tr>
<tr>
<td style="text-align:center;">
type I interferon response (M127)
</td>
<td style="text-align:center;">
12
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.000
</td>
<td style="text-align:center;">
0.000
</td>
</tr>
<tr>
<td style="text-align:center;">
cell cycle and growth arrest (M31)
</td>
<td style="text-align:center;">
11
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.000
</td>
<td style="text-align:center;">
0.000
</td>
</tr>
<tr>
<td style="text-align:center;">
proinflammatory dendritic cell, myeloid cell response (M86.1)
</td>
<td style="text-align:center;">
11
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.000
</td>
<td style="text-align:center;">
0.000
</td>
</tr>
<tr>
<td style="text-align:center;">
chemokine cluster (I) (M27.0)
</td>
<td style="text-align:center;">
20
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.000
</td>
<td style="text-align:center;">
0.000
</td>
</tr>
<tr>
<td style="text-align:center;">
RIG-1 like receptor signaling (M68)
</td>
<td style="text-align:center;">
10
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.000
</td>
<td style="text-align:center;">
0.000
</td>
</tr>
<tr>
<td style="text-align:center;">
cytokines - recepters cluster (M115)
</td>
<td style="text-align:center;">
10
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.000
</td>
<td style="text-align:center;">
0.000
</td>
</tr>
<tr>
<td style="text-align:center;">
proinflammatory cytokines and chemokines (M29)
</td>
<td style="text-align:center;">
9
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.000
</td>
<td style="text-align:center;">
0.000
</td>
</tr>
<tr>
<td style="text-align:center;">
regulation of transcription, transcription factors (M213)
</td>
<td style="text-align:center;">
17
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.000
</td>
<td style="text-align:center;">
0.001
</td>
</tr>
<tr>
<td style="text-align:center;">
signaling in T cells (II) (M35.1)
</td>
<td style="text-align:center;">
10
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.000
</td>
<td style="text-align:center;">
0.002
</td>
</tr>
<tr>
<td style="text-align:center;">
activated dendritic cells (M67)
</td>
<td style="text-align:center;">
9
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.000
</td>
<td style="text-align:center;">
0.003
</td>
</tr>
<tr>
<td style="text-align:center;">
enriched in activated dendritic cells/monocytes (M64)
</td>
<td style="text-align:center;">
17
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.000
</td>
<td style="text-align:center;">
0.004
</td>
</tr>
<tr>
<td style="text-align:center;">
antiviral IFN signature (M75)
</td>
<td style="text-align:center;">
19
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.000
</td>
<td style="text-align:center;">
0.005
</td>
</tr>
<tr>
<td style="text-align:center;">
TBA (M203)
</td>
<td style="text-align:center;">
13
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.000
</td>
<td style="text-align:center;">
0.006
</td>
</tr>
<tr>
<td style="text-align:center;">
leukocyte differentiation (M160)
</td>
<td style="text-align:center;">
14
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.000
</td>
<td style="text-align:center;">
0.006
</td>
</tr>
<tr>
<td style="text-align:center;">
TBA (M121)
</td>
<td style="text-align:center;">
10
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.001
</td>
<td style="text-align:center;">
0.008
</td>
</tr>
<tr>
<td style="text-align:center;">
TBA (M72.0)
</td>
<td style="text-align:center;">
21
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.001
</td>
<td style="text-align:center;">
0.008
</td>
</tr>
<tr>
<td style="text-align:center;">
enriched in monocytes (II) (M11.0)
</td>
<td style="text-align:center;">
146
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.001
</td>
<td style="text-align:center;">
0.008
</td>
</tr>
<tr>
<td style="text-align:center;">
TBA (M72.1)
</td>
<td style="text-align:center;">
16
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.001
</td>
<td style="text-align:center;">
0.008
</td>
</tr>
<tr>
<td style="text-align:center;">
TBA (source: B cells) (M152.0)
</td>
<td style="text-align:center;">
6
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.001
</td>
<td style="text-align:center;">
0.008
</td>
</tr>
<tr>
<td style="text-align:center;">
putative targets of PAX3 (M89.0)
</td>
<td style="text-align:center;">
12
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.001
</td>
<td style="text-align:center;">
0.009
</td>
</tr>
<tr>
<td style="text-align:center;">
putative targets of PAX3 (M89.1)
</td>
<td style="text-align:center;">
9
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.001
</td>
<td style="text-align:center;">
0.009
</td>
</tr>
<tr>
<td style="text-align:center;">
signaling in T cells (I) (M35.0)
</td>
<td style="text-align:center;">
13
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.001
</td>
<td style="text-align:center;">
0.012
</td>
</tr>
<tr>
<td style="text-align:center;">
viral sensing &amp; immunity; IRF2 targets network (II) (M111.1)
</td>
<td style="text-align:center;">
11
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.001
</td>
<td style="text-align:center;">
0.012
</td>
</tr>
<tr>
<td style="text-align:center;">
innate antiviral response (M150)
</td>
<td style="text-align:center;">
12
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.001
</td>
<td style="text-align:center;">
0.012
</td>
</tr>
<tr>
<td style="text-align:center;">
Monocyte surface signature (S4)
</td>
<td style="text-align:center;">
69
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.001
</td>
<td style="text-align:center;">
0.012
</td>
</tr>
<tr>
<td style="text-align:center;">
TBA (M190)
</td>
<td style="text-align:center;">
10
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.001
</td>
<td style="text-align:center;">
0.012
</td>
</tr>
<tr>
<td style="text-align:center;">
TBA (M174)
</td>
<td style="text-align:center;">
22
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.001
</td>
<td style="text-align:center;">
0.012
</td>
</tr>
<tr>
<td style="text-align:center;">
immuregulation - monocytes, T and B cells (M57)
</td>
<td style="text-align:center;">
10
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.002
</td>
<td style="text-align:center;">
0.017
</td>
</tr>
<tr>
<td style="text-align:center;">
myeloid, dendritic cell activation via NFkB (II) (M43.1)
</td>
<td style="text-align:center;">
13
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.002
</td>
<td style="text-align:center;">
0.020
</td>
</tr>
<tr>
<td style="text-align:center;">
cell cycle and transcription (M4.0)
</td>
<td style="text-align:center;">
226
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.003
</td>
<td style="text-align:center;">
0.022
</td>
</tr>
<tr>
<td style="text-align:center;">
growth factor induced, enriched in nuclear receptor subfamily 4 (M94)
</td>
<td style="text-align:center;">
7
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.003
</td>
<td style="text-align:center;">
0.022
</td>
</tr>
<tr>
<td style="text-align:center;">
innate activation by cytosolic DNA sensing (M13)
</td>
<td style="text-align:center;">
11
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.003
</td>
<td style="text-align:center;">
0.022
</td>
</tr>
<tr>
<td style="text-align:center;">
extracellular matrix, complement (M140)
</td>
<td style="text-align:center;">
7
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.003
</td>
<td style="text-align:center;">
0.022
</td>
</tr>
<tr>
<td style="text-align:center;">
enriched in activated dendritic cells (II) (M165)
</td>
<td style="text-align:center;">
29
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.003
</td>
<td style="text-align:center;">
0.022
</td>
</tr>
<tr>
<td style="text-align:center;">
chemokines and receptors (M38)
</td>
<td style="text-align:center;">
11
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.003
</td>
<td style="text-align:center;">
0.023
</td>
</tr>
<tr>
<td style="text-align:center;">
TBA (M177.1)
</td>
<td style="text-align:center;">
10
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.003
</td>
<td style="text-align:center;">
0.023
</td>
</tr>
<tr>
<td style="text-align:center;">
myeloid, dendritic cell activation via NFkB (I) (M43.0)
</td>
<td style="text-align:center;">
12
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.004
</td>
<td style="text-align:center;">
0.026
</td>
</tr>
<tr>
<td style="text-align:center;">
TBA (M131)
</td>
<td style="text-align:center;">
12
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.004
</td>
<td style="text-align:center;">
0.030
</td>
</tr>
<tr>
<td style="text-align:center;">
Resting dendritic cell surface signature (S10)
</td>
<td style="text-align:center;">
43
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.005
</td>
<td style="text-align:center;">
0.035
</td>
</tr>
<tr>
<td style="text-align:center;">
myeloid cell enriched receptors and transporters (M4.3)
</td>
<td style="text-align:center;">
26
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.006
</td>
<td style="text-align:center;">
0.035
</td>
</tr>
<tr>
<td style="text-align:center;">
TBA (M72.2)
</td>
<td style="text-align:center;">
12
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.006
</td>
<td style="text-align:center;">
0.035
</td>
</tr>
<tr>
<td style="text-align:center;">
TBA (M141)
</td>
<td style="text-align:center;">
7
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.007
</td>
<td style="text-align:center;">
0.042
</td>
</tr>
<tr>
<td style="text-align:center;">
lysosome (M209)
</td>
<td style="text-align:center;">
7
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.008
</td>
<td style="text-align:center;">
0.046
</td>
</tr>
<tr>
<td style="text-align:center;">
enriched in NK cells (I) (M7.2)
</td>
<td style="text-align:center;">
45
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.008
</td>
<td style="text-align:center;">
0.046
</td>
</tr>
<tr>
<td style="text-align:center;">
TBA (M194)
</td>
<td style="text-align:center;">
13
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.008
</td>
<td style="text-align:center;">
0.048
</td>
</tr>
<tr>
<td style="text-align:center;">
Plasma cell surface signature (S3)
</td>
<td style="text-align:center;">
10
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.009
</td>
<td style="text-align:center;">
0.048
</td>
</tr>
<tr>
<td style="text-align:center;">
TBA (M229)
</td>
<td style="text-align:center;">
10
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.009
</td>
<td style="text-align:center;">
0.050
</td>
</tr>
<tr>
<td style="text-align:center;">
CD1 and other DC receptors (M50)
</td>
<td style="text-align:center;">
6
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.010
</td>
<td style="text-align:center;">
0.053
</td>
</tr>
<tr>
<td style="text-align:center;">
TBA (source: memory B cells) (M152.2)
</td>
<td style="text-align:center;">
6
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.013
</td>
<td style="text-align:center;">
0.067
</td>
</tr>
<tr>
<td style="text-align:center;">
TBA (M180)
</td>
<td style="text-align:center;">
11
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.013
</td>
<td style="text-align:center;">
0.067
</td>
</tr>
<tr>
<td style="text-align:center;">
proteasome (M226)
</td>
<td style="text-align:center;">
12
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.014
</td>
<td style="text-align:center;">
0.069
</td>
</tr>
<tr>
<td style="text-align:center;">
Memory B cell surface signature (S9)
</td>
<td style="text-align:center;">
13
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.014
</td>
<td style="text-align:center;">
0.069
</td>
</tr>
<tr>
<td style="text-align:center;">
cell activation (IL15, IL23, TNF) (M24)
</td>
<td style="text-align:center;">
12
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.015
</td>
<td style="text-align:center;">
0.072
</td>
</tr>
<tr>
<td style="text-align:center;">
CORO1A-DEF6 network (I) (M32.2)
</td>
<td style="text-align:center;">
17
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.015
</td>
<td style="text-align:center;">
0.074
</td>
</tr>
<tr>
<td style="text-align:center;">
B cell surface signature (S2)
</td>
<td style="text-align:center;">
70
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.016
</td>
<td style="text-align:center;">
0.076
</td>
</tr>
<tr>
<td style="text-align:center;">
enriched in cell cycle (M167)
</td>
<td style="text-align:center;">
14
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.017
</td>
<td style="text-align:center;">
0.077
</td>
</tr>
<tr>
<td style="text-align:center;">
AP-1 transcription factor network (M20)
</td>
<td style="text-align:center;">
13
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.017
</td>
<td style="text-align:center;">
0.079
</td>
</tr>
<tr>
<td style="text-align:center;">
TBA (M104)
</td>
<td style="text-align:center;">
6
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.017
</td>
<td style="text-align:center;">
0.079
</td>
</tr>
<tr>
<td style="text-align:center;">
TBA (M32.6)
</td>
<td style="text-align:center;">
12
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.019
</td>
<td style="text-align:center;">
0.083
</td>
</tr>
<tr>
<td style="text-align:center;">
CORO1A-DEF6 network (II) (M32.4)
</td>
<td style="text-align:center;">
13
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.019
</td>
<td style="text-align:center;">
0.083
</td>
</tr>
<tr>
<td style="text-align:center;">
leukocyte migration (M88.0)
</td>
<td style="text-align:center;">
12
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.020
</td>
<td style="text-align:center;">
0.084
</td>
</tr>
<tr>
<td style="text-align:center;">
chemokine cluster (II) (M27.1)
</td>
<td style="text-align:center;">
13
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.021
</td>
<td style="text-align:center;">
0.088
</td>
</tr>
<tr>
<td style="text-align:center;">
enriched in monocytes (surface) (M118.1)
</td>
<td style="text-align:center;">
17
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.022
</td>
<td style="text-align:center;">
0.091
</td>
</tr>
<tr>
<td style="text-align:center;">
E2F transcription factor network (M8)
</td>
<td style="text-align:center;">
7
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.023
</td>
<td style="text-align:center;">
0.092
</td>
</tr>
<tr>
<td style="text-align:center;">
Naive B cell surface signature (S8)
</td>
<td style="text-align:center;">
24
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.026
</td>
<td style="text-align:center;">
0.106
</td>
</tr>
<tr>
<td style="text-align:center;">
platelet activation (I) (M32.0)
</td>
<td style="text-align:center;">
20
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.027
</td>
<td style="text-align:center;">
0.107
</td>
</tr>
<tr>
<td style="text-align:center;">
platelet activation (II) (M32.1)
</td>
<td style="text-align:center;">
20
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.032
</td>
<td style="text-align:center;">
0.121
</td>
</tr>
<tr>
<td style="text-align:center;">
TBA (M242)
</td>
<td style="text-align:center;">
11
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.032
</td>
<td style="text-align:center;">
0.121
</td>
</tr>
<tr>
<td style="text-align:center;">
TBA (M137)
</td>
<td style="text-align:center;">
15
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.034
</td>
<td style="text-align:center;">
0.128
</td>
</tr>
<tr>
<td style="text-align:center;">
spliceosome (M250)
</td>
<td style="text-align:center;">
12
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.035
</td>
<td style="text-align:center;">
0.130
</td>
</tr>
<tr>
<td style="text-align:center;">
enriched for TF motif PAX3 (M179)
</td>
<td style="text-align:center;">
8
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.036
</td>
<td style="text-align:center;">
0.132
</td>
</tr>
<tr>
<td style="text-align:center;">
cytoskeletal remodeling (enriched for SRF targets) (M34)
</td>
<td style="text-align:center;">
8
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.037
</td>
<td style="text-align:center;">
0.135
</td>
</tr>
<tr>
<td style="text-align:center;">
targets of FOSL1/2 (M0)
</td>
<td style="text-align:center;">
7
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.038
</td>
<td style="text-align:center;">
0.136
</td>
</tr>
<tr>
<td style="text-align:center;">
chaperonin mediated protein folding (I) (M204.0)
</td>
<td style="text-align:center;">
14
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.040
</td>
<td style="text-align:center;">
0.142
</td>
</tr>
<tr>
<td style="text-align:center;">
intracellular transport (M147)
</td>
<td style="text-align:center;">
17
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.041
</td>
<td style="text-align:center;">
0.142
</td>
</tr>
<tr>
<td style="text-align:center;">
cell division (stimulated CD4+ T cells) (M46)
</td>
<td style="text-align:center;">
13
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.041
</td>
<td style="text-align:center;">
0.142
</td>
</tr>
<tr>
<td style="text-align:center;">
cell adhesion (M51)
</td>
<td style="text-align:center;">
8
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.044
</td>
<td style="text-align:center;">
0.151
</td>
</tr>
<tr>
<td style="text-align:center;">
TLR and inflammatory signaling (M16)
</td>
<td style="text-align:center;">
34
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.047
</td>
<td style="text-align:center;">
0.157
</td>
</tr>
<tr>
<td style="text-align:center;">
enriched in B cells (VI) (M69)
</td>
<td style="text-align:center;">
18
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.055
</td>
<td style="text-align:center;">
0.181
</td>
</tr>
<tr>
<td style="text-align:center;">
mitosis (TF motif CCAATNNSNNNGCG) (M169)
</td>
<td style="text-align:center;">
14
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.060
</td>
<td style="text-align:center;">
0.192
</td>
</tr>
<tr>
<td style="text-align:center;">
cell cycle, ATP binding (M144)
</td>
<td style="text-align:center;">
16
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.061
</td>
<td style="text-align:center;">
0.192
</td>
</tr>
<tr>
<td style="text-align:center;">
TBA (M80)
</td>
<td style="text-align:center;">
12
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.062
</td>
<td style="text-align:center;">
0.192
</td>
</tr>
<tr>
<td style="text-align:center;">
glycerophospholipid metabolism (M114.1)
</td>
<td style="text-align:center;">
10
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.062
</td>
<td style="text-align:center;">
0.192
</td>
</tr>
<tr>
<td style="text-align:center;">
regulation of localization (GO) (M63)
</td>
<td style="text-align:center;">
11
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.062
</td>
<td style="text-align:center;">
0.192
</td>
</tr>
<tr>
<td style="text-align:center;">
inflammasome receptors and signaling (M53)
</td>
<td style="text-align:center;">
9
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.063
</td>
<td style="text-align:center;">
0.192
</td>
</tr>
<tr>
<td style="text-align:center;">
TBA (M32.7)
</td>
<td style="text-align:center;">
10
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.063
</td>
<td style="text-align:center;">
0.192
</td>
</tr>
<tr>
<td style="text-align:center;">
chaperonin mediated protein folding (II) (M204.1)
</td>
<td style="text-align:center;">
10
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.064
</td>
<td style="text-align:center;">
0.194
</td>
</tr>
</tbody>
</table>
</div>
<div id="comparator" class="section level3">
<h3><span class="header-section-number">1.2.3</span> comparator</h3>
<table class="table table-striped table-hover table-condensed" style="width: auto !important; ">
<thead>
<tr>
<th style="text-align:center;">
geneset
</th>
<th style="text-align:center;">
NGenes
</th>
<th style="text-align:center;">
Direction
</th>
<th style="text-align:center;">
PValue
</th>
<th style="text-align:center;">
FDR
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:center;">
enriched in monocytes (II) (M11.0)
</td>
<td style="text-align:center;">
146
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.000
</td>
<td style="text-align:center;">
0.000
</td>
</tr>
<tr>
<td style="text-align:center;">
type I interferon response (M127)
</td>
<td style="text-align:center;">
12
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.000
</td>
<td style="text-align:center;">
0.000
</td>
</tr>
<tr>
<td style="text-align:center;">
Monocyte surface signature (S4)
</td>
<td style="text-align:center;">
69
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.000
</td>
<td style="text-align:center;">
0.000
</td>
</tr>
<tr>
<td style="text-align:center;">
cell cycle and transcription (M4.0)
</td>
<td style="text-align:center;">
226
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.000
</td>
<td style="text-align:center;">
0.000
</td>
</tr>
<tr>
<td style="text-align:center;">
extracellular matrix, complement (M140)
</td>
<td style="text-align:center;">
7
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.000
</td>
<td style="text-align:center;">
0.000
</td>
</tr>
<tr>
<td style="text-align:center;">
Resting dendritic cell surface signature (S10)
</td>
<td style="text-align:center;">
43
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.000
</td>
<td style="text-align:center;">
0.000
</td>
</tr>
<tr>
<td style="text-align:center;">
myeloid cell enriched receptors and transporters (M4.3)
</td>
<td style="text-align:center;">
26
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.000
</td>
<td style="text-align:center;">
0.000
</td>
</tr>
<tr>
<td style="text-align:center;">
formyl peptide receptor mediated neutrophil response (M11.2)
</td>
<td style="text-align:center;">
8
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.000
</td>
<td style="text-align:center;">
0.000
</td>
</tr>
<tr>
<td style="text-align:center;">
innate antiviral response (M150)
</td>
<td style="text-align:center;">
12
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.000
</td>
<td style="text-align:center;">
0.000
</td>
</tr>
<tr>
<td style="text-align:center;">
myeloid, dendritic cell activation via NFkB (I) (M43.0)
</td>
<td style="text-align:center;">
12
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.000
</td>
<td style="text-align:center;">
0.000
</td>
</tr>
<tr>
<td style="text-align:center;">
transcription regulation in cell development (M49)
</td>
<td style="text-align:center;">
37
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.000
</td>
<td style="text-align:center;">
0.000
</td>
</tr>
<tr>
<td style="text-align:center;">
myeloid, dendritic cell activation via NFkB (II) (M43.1)
</td>
<td style="text-align:center;">
13
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.000
</td>
<td style="text-align:center;">
0.000
</td>
</tr>
<tr>
<td style="text-align:center;">
enriched in neutrophils (I) (M37.1)
</td>
<td style="text-align:center;">
29
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.000
</td>
<td style="text-align:center;">
0.001
</td>
</tr>
<tr>
<td style="text-align:center;">
TLR and inflammatory signaling (M16)
</td>
<td style="text-align:center;">
34
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.000
</td>
<td style="text-align:center;">
0.001
</td>
</tr>
<tr>
<td style="text-align:center;">
cytokines - recepters cluster (M115)
</td>
<td style="text-align:center;">
10
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.000
</td>
<td style="text-align:center;">
0.002
</td>
</tr>
<tr>
<td style="text-align:center;">
inflammatory response (M33)
</td>
<td style="text-align:center;">
8
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.000
</td>
<td style="text-align:center;">
0.003
</td>
</tr>
<tr>
<td style="text-align:center;">
enriched in monocytes (surface) (M118.1)
</td>
<td style="text-align:center;">
17
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.000
</td>
<td style="text-align:center;">
0.003
</td>
</tr>
<tr>
<td style="text-align:center;">
complement and other receptors in DCs (M40)
</td>
<td style="text-align:center;">
11
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.000
</td>
<td style="text-align:center;">
0.004
</td>
</tr>
<tr>
<td style="text-align:center;">
immune activation - generic cluster (M37.0)
</td>
<td style="text-align:center;">
210
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.000
</td>
<td style="text-align:center;">
0.004
</td>
</tr>
<tr>
<td style="text-align:center;">
antiviral IFN signature (M75)
</td>
<td style="text-align:center;">
19
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.000
</td>
<td style="text-align:center;">
0.005
</td>
</tr>
<tr>
<td style="text-align:center;">
viral sensing &amp; immunity; IRF2 targets network (II) (M111.1)
</td>
<td style="text-align:center;">
11
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.000
</td>
<td style="text-align:center;">
0.005
</td>
</tr>
<tr>
<td style="text-align:center;">
extracellular matrix (II) (M2.1)
</td>
<td style="text-align:center;">
18
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.001
</td>
<td style="text-align:center;">
0.006
</td>
</tr>
<tr>
<td style="text-align:center;">
transmembrane transport (I) (M87)
</td>
<td style="text-align:center;">
19
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.001
</td>
<td style="text-align:center;">
0.006
</td>
</tr>
<tr>
<td style="text-align:center;">
regulation of transcription, transcription factors (M213)
</td>
<td style="text-align:center;">
17
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.001
</td>
<td style="text-align:center;">
0.007
</td>
</tr>
<tr>
<td style="text-align:center;">
TBA (M66)
</td>
<td style="text-align:center;">
17
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.001
</td>
<td style="text-align:center;">
0.009
</td>
</tr>
<tr>
<td style="text-align:center;">
platelet activation and degranulation (M85)
</td>
<td style="text-align:center;">
8
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.001
</td>
<td style="text-align:center;">
0.011
</td>
</tr>
<tr>
<td style="text-align:center;">
proinflammatory dendritic cell, myeloid cell response (M86.1)
</td>
<td style="text-align:center;">
11
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.001
</td>
<td style="text-align:center;">
0.012
</td>
</tr>
<tr>
<td style="text-align:center;">
RIG-1 like receptor signaling (M68)
</td>
<td style="text-align:center;">
10
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.002
</td>
<td style="text-align:center;">
0.015
</td>
</tr>
<tr>
<td style="text-align:center;">
enriched in monocytes (IV) (M118.0)
</td>
<td style="text-align:center;">
48
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.002
</td>
<td style="text-align:center;">
0.016
</td>
</tr>
<tr>
<td style="text-align:center;">
activated dendritic cells (M67)
</td>
<td style="text-align:center;">
9
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.002
</td>
<td style="text-align:center;">
0.017
</td>
</tr>
<tr>
<td style="text-align:center;">
enriched in monocytes (I) (M4.15)
</td>
<td style="text-align:center;">
8
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.002
</td>
<td style="text-align:center;">
0.017
</td>
</tr>
<tr>
<td style="text-align:center;">
leukocyte differentiation (M160)
</td>
<td style="text-align:center;">
14
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.002
</td>
<td style="text-align:center;">
0.017
</td>
</tr>
<tr>
<td style="text-align:center;">
enriched in membrane proteins (M124)
</td>
<td style="text-align:center;">
13
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.002
</td>
<td style="text-align:center;">
0.018
</td>
</tr>
<tr>
<td style="text-align:center;">
heme biosynthesis (II) (M222)
</td>
<td style="text-align:center;">
10
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.002
</td>
<td style="text-align:center;">
0.018
</td>
</tr>
<tr>
<td style="text-align:center;">
CORO1A-DEF6 network (II) (M32.4)
</td>
<td style="text-align:center;">
13
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.002
</td>
<td style="text-align:center;">
0.018
</td>
</tr>
<tr>
<td style="text-align:center;">
regulation of signal transduction (M3)
</td>
<td style="text-align:center;">
33
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.003
</td>
<td style="text-align:center;">
0.022
</td>
</tr>
<tr>
<td style="text-align:center;">
mitosis (TF motif CCAATNNSNNNGCG) (M169)
</td>
<td style="text-align:center;">
14
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.003
</td>
<td style="text-align:center;">
0.022
</td>
</tr>
<tr>
<td style="text-align:center;">
immuregulation - monocytes, T and B cells (M57)
</td>
<td style="text-align:center;">
10
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.003
</td>
<td style="text-align:center;">
0.024
</td>
</tr>
<tr>
<td style="text-align:center;">
DC surface signature (S5)
</td>
<td style="text-align:center;">
54
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.004
</td>
<td style="text-align:center;">
0.027
</td>
</tr>
<tr>
<td style="text-align:center;">
IL2, IL7, TCR network (M65)
</td>
<td style="text-align:center;">
9
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.004
</td>
<td style="text-align:center;">
0.030
</td>
</tr>
<tr>
<td style="text-align:center;">
chemokines and inflammatory molecules in myeloid cells (M86.0)
</td>
<td style="text-align:center;">
17
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.005
</td>
<td style="text-align:center;">
0.032
</td>
</tr>
<tr>
<td style="text-align:center;">
glycerophospholipid metabolism (M114.1)
</td>
<td style="text-align:center;">
10
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.005
</td>
<td style="text-align:center;">
0.033
</td>
</tr>
<tr>
<td style="text-align:center;">
putative targets of PAX3 (M89.1)
</td>
<td style="text-align:center;">
9
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.006
</td>
<td style="text-align:center;">
0.040
</td>
</tr>
<tr>
<td style="text-align:center;">
enriched in monocytes (III) (M73)
</td>
<td style="text-align:center;">
10
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.007
</td>
<td style="text-align:center;">
0.040
</td>
</tr>
<tr>
<td style="text-align:center;">
complement activation (I) (M112.0)
</td>
<td style="text-align:center;">
10
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.009
</td>
<td style="text-align:center;">
0.051
</td>
</tr>
<tr>
<td style="text-align:center;">
heme biosynthesis (I) (M171)
</td>
<td style="text-align:center;">
10
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.010
</td>
<td style="text-align:center;">
0.055
</td>
</tr>
<tr>
<td style="text-align:center;">
TBA (M72.0)
</td>
<td style="text-align:center;">
21
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.010
</td>
<td style="text-align:center;">
0.056
</td>
</tr>
<tr>
<td style="text-align:center;">
innate activation by cytosolic DNA sensing (M13)
</td>
<td style="text-align:center;">
11
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.010
</td>
<td style="text-align:center;">
0.057
</td>
</tr>
<tr>
<td style="text-align:center;">
enriched in dendritic cells (M168)
</td>
<td style="text-align:center;">
12
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.011
</td>
<td style="text-align:center;">
0.058
</td>
</tr>
<tr>
<td style="text-align:center;">
TBA (M120)
</td>
<td style="text-align:center;">
11
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.013
</td>
<td style="text-align:center;">
0.069
</td>
</tr>
<tr>
<td style="text-align:center;">
TBA (M48)
</td>
<td style="text-align:center;">
10
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.015
</td>
<td style="text-align:center;">
0.076
</td>
</tr>
<tr>
<td style="text-align:center;">
DNA repair (M76)
</td>
<td style="text-align:center;">
19
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.015
</td>
<td style="text-align:center;">
0.079
</td>
</tr>
<tr>
<td style="text-align:center;">
lysosomal/endosomal proteins (M139)
</td>
<td style="text-align:center;">
10
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.018
</td>
<td style="text-align:center;">
0.091
</td>
</tr>
<tr>
<td style="text-align:center;">
enriched for TF motif PAX3 (M179)
</td>
<td style="text-align:center;">
8
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.020
</td>
<td style="text-align:center;">
0.096
</td>
</tr>
<tr>
<td style="text-align:center;">
TLR8-BAFF network (M25)
</td>
<td style="text-align:center;">
8
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.020
</td>
<td style="text-align:center;">
0.096
</td>
</tr>
<tr>
<td style="text-align:center;">
cell activation (IL15, IL23, TNF) (M24)
</td>
<td style="text-align:center;">
12
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.022
</td>
<td style="text-align:center;">
0.105
</td>
</tr>
<tr>
<td style="text-align:center;">
lysosome (M209)
</td>
<td style="text-align:center;">
7
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.024
</td>
<td style="text-align:center;">
0.110
</td>
</tr>
<tr>
<td style="text-align:center;">
cell division - E2F transcription network (M4.8)
</td>
<td style="text-align:center;">
14
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.025
</td>
<td style="text-align:center;">
0.112
</td>
</tr>
<tr>
<td style="text-align:center;">
TBA (M32.7)
</td>
<td style="text-align:center;">
10
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.025
</td>
<td style="text-align:center;">
0.112
</td>
</tr>
<tr>
<td style="text-align:center;">
NK cell surface signature (S1)
</td>
<td style="text-align:center;">
24
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.025
</td>
<td style="text-align:center;">
0.112
</td>
</tr>
<tr>
<td style="text-align:center;">
signaling in T cells (II) (M35.1)
</td>
<td style="text-align:center;">
10
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.028
</td>
<td style="text-align:center;">
0.123
</td>
</tr>
<tr>
<td style="text-align:center;">
enriched in neutrophils (II) (M163)
</td>
<td style="text-align:center;">
7
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.029
</td>
<td style="text-align:center;">
0.123
</td>
</tr>
<tr>
<td style="text-align:center;">
G protein coupled receptors cluster (M155)
</td>
<td style="text-align:center;">
9
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.030
</td>
<td style="text-align:center;">
0.124
</td>
</tr>
<tr>
<td style="text-align:center;">
proteasome (M226)
</td>
<td style="text-align:center;">
12
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.030
</td>
<td style="text-align:center;">
0.124
</td>
</tr>
<tr>
<td style="text-align:center;">
CORO1A-DEF6 network (I) (M32.2)
</td>
<td style="text-align:center;">
17
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.031
</td>
<td style="text-align:center;">
0.126
</td>
</tr>
<tr>
<td style="text-align:center;">
endoplasmic reticulum (M37.2)
</td>
<td style="text-align:center;">
11
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.031
</td>
<td style="text-align:center;">
0.126
</td>
</tr>
<tr>
<td style="text-align:center;">
recruitment of neutrophils (M132)
</td>
<td style="text-align:center;">
6
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.034
</td>
<td style="text-align:center;">
0.135
</td>
</tr>
<tr>
<td style="text-align:center;">
TBA (M72.2)
</td>
<td style="text-align:center;">
12
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.035
</td>
<td style="text-align:center;">
0.137
</td>
</tr>
<tr>
<td style="text-align:center;">
Plasma cell surface signature (S3)
</td>
<td style="text-align:center;">
10
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.036
</td>
<td style="text-align:center;">
0.137
</td>
</tr>
<tr>
<td style="text-align:center;">
TBA (M72.1)
</td>
<td style="text-align:center;">
16
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.037
</td>
<td style="text-align:center;">
0.142
</td>
</tr>
<tr>
<td style="text-align:center;">
B cell surface signature (S2)
</td>
<td style="text-align:center;">
70
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.039
</td>
<td style="text-align:center;">
0.145
</td>
</tr>
<tr>
<td style="text-align:center;">
KLF12 targets network (M32.3)
</td>
<td style="text-align:center;">
15
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.039
</td>
<td style="text-align:center;">
0.145
</td>
</tr>
<tr>
<td style="text-align:center;">
cell cycle (I) (M4.1)
</td>
<td style="text-align:center;">
89
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.040
</td>
<td style="text-align:center;">
0.146
</td>
</tr>
<tr>
<td style="text-align:center;">
suppression of MAPK signaling (M56)
</td>
<td style="text-align:center;">
10
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.042
</td>
<td style="text-align:center;">
0.152
</td>
</tr>
<tr>
<td style="text-align:center;">
TBA (M194)
</td>
<td style="text-align:center;">
13
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.044
</td>
<td style="text-align:center;">
0.156
</td>
</tr>
<tr>
<td style="text-align:center;">
enriched in myeloid cells and monocytes (M81)
</td>
<td style="text-align:center;">
23
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.048
</td>
<td style="text-align:center;">
0.166
</td>
</tr>
<tr>
<td style="text-align:center;">
platelet activation (I) (M32.0)
</td>
<td style="text-align:center;">
20
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.048
</td>
<td style="text-align:center;">
0.166
</td>
</tr>
<tr>
<td style="text-align:center;">
TBA (M190)
</td>
<td style="text-align:center;">
10
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.052
</td>
<td style="text-align:center;">
0.176
</td>
</tr>
<tr>
<td style="text-align:center;">
Membrane, ER proteins (M134)
</td>
<td style="text-align:center;">
8
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.054
</td>
<td style="text-align:center;">
0.176
</td>
</tr>
<tr>
<td style="text-align:center;">
enriched for TF motif TNCATNTCCYR (M232)
</td>
<td style="text-align:center;">
13
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.054
</td>
<td style="text-align:center;">
0.176
</td>
</tr>
<tr>
<td style="text-align:center;">
RA, WNT, CSF receptors network (monocyte) (M23)
</td>
<td style="text-align:center;">
9
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.054
</td>
<td style="text-align:center;">
0.176
</td>
</tr>
<tr>
<td style="text-align:center;">
B cell development/activation (M58)
</td>
<td style="text-align:center;">
10
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.055
</td>
<td style="text-align:center;">
0.176
</td>
</tr>
<tr>
<td style="text-align:center;">
cell division (M37.3)
</td>
<td style="text-align:center;">
8
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.057
</td>
<td style="text-align:center;">
0.182
</td>
</tr>
<tr>
<td style="text-align:center;">
cell cycle (II) (M4.10)
</td>
<td style="text-align:center;">
7
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.059
</td>
<td style="text-align:center;">
0.185
</td>
</tr>
<tr>
<td style="text-align:center;">
platelet activation - actin binding (M196)
</td>
<td style="text-align:center;">
6
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.060
</td>
<td style="text-align:center;">
0.186
</td>
</tr>
<tr>
<td style="text-align:center;">
nuclear pore, transport; mRNA splicing, processing (M143)
</td>
<td style="text-align:center;">
11
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.062
</td>
<td style="text-align:center;">
0.189
</td>
</tr>
<tr>
<td style="text-align:center;">
T cell surface, activation (M36)
</td>
<td style="text-align:center;">
11
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.062
</td>
<td style="text-align:center;">
0.189
</td>
</tr>
<tr>
<td style="text-align:center;">
TBA (M193)
</td>
<td style="text-align:center;">
10
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.064
</td>
<td style="text-align:center;">
0.189
</td>
</tr>
<tr>
<td style="text-align:center;">
CD1 and other DC receptors (M50)
</td>
<td style="text-align:center;">
6
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.064
</td>
<td style="text-align:center;">
0.189
</td>
</tr>
<tr>
<td style="text-align:center;">
collagen, TGFB family et al (M77)
</td>
<td style="text-align:center;">
14
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.065
</td>
<td style="text-align:center;">
0.189
</td>
</tr>
<tr>
<td style="text-align:center;">
cell cycle and growth arrest (M31)
</td>
<td style="text-align:center;">
11
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.065
</td>
<td style="text-align:center;">
0.189
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="heatmap" class="section level2">
<h2><span class="header-section-number">1.3</span> Heatmap</h2>
<p><img src="figure/manuscript_Fig2_vaccine.Rmd/heatme_M3-M0-1.png" width="672" style="display: block; margin: auto;" /><img src="figure/manuscript_Fig2_vaccine.Rmd/heatme_M3-M0-2.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="bubble-map" class="section level2">
<h2><span class="header-section-number">1.4</span> Bubble map</h2>
<p><img src="figure/manuscript_Fig2_vaccine.Rmd/bubble-1.png" width="672" style="display: block; margin: auto;" /></p>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span> Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<p>R version 3.6.1 (2019-07-05) Platform: x86_64-pc-linux-gnu (64-bit) Running under: Ubuntu 14.04.5 LTS</p>
<p>Matrix products: default BLAS/LAPACK: /app/easybuild/software/OpenBLAS/0.2.18-GCC-5.4.0-2.26-LAPACK-3.6.1/lib/libopenblas_prescottp-r0.2.18.so</p>
<p>locale: [1] LC_CTYPE=en_US.UTF-8 LC_NUMERIC=C<br />
[3] LC_TIME=en_US.UTF-8 LC_COLLATE=en_US.UTF-8<br />
[5] LC_MONETARY=en_US.UTF-8 LC_MESSAGES=en_US.UTF-8<br />
[7] LC_PAPER=en_US.UTF-8 LC_NAME=C<br />
[9] LC_ADDRESS=C LC_TELEPHONE=C<br />
[11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C</p>
<p>attached base packages: [1] stats4 parallel stats graphics grDevices utils datasets [8] methods base</p>
<p>other attached packages: [1] mal067data_1.0 kableExtra_1.1.0 RColorBrewer_1.1-2<br />
[4] data.table_1.12.8 forcats_0.4.0 stringr_1.4.0<br />
[7] dplyr_0.8.3 purrr_0.3.3 tidyr_0.8.3<br />
[10] tibble_2.1.3 ggplot2_3.2.1 tidyverse_1.2.1<br />
[13] limma_3.40.2 GSEABase_1.46.0 graph_1.62.0<br />
[16] annotate_1.62.0 XML_3.98-1.19 AnnotationDbi_1.46.0 [19] IRanges_2.18.1 S4Vectors_0.22.0 DT_0.11<br />
[22] heatmap3_1.1.6 here_0.1 magrittr_1.5<br />
[25] Biobase_2.44.0 BiocGenerics_0.30.0 RNASeqUtilities_1.1 [28] readr_1.3.1 knitr_1.27 workflowr_1.6.0</p>
<p>loaded via a namespace (and not attached): [1] nlme_3.1-140 bitops_1.0-6 fs_1.3.1 lubridate_1.7.4<br />
[5] bit64_0.9-7 webshot_0.5.1 httr_1.4.1 rprojroot_1.3-2<br />
[9] tools_3.6.1 backports_1.1.5 R6_2.4.1 DBI_1.1.0<br />
[13] lazyeval_0.2.2 colorspace_1.4-1 withr_2.1.2 tidyselect_0.2.5<br />
[17] gridExtra_2.3 bit_1.1-15.1 compiler_3.6.1 git2r_0.26.1<br />
[21] cli_2.0.1 rvest_0.3.4 xml2_1.2.2 labeling_0.3<br />
[25] scales_1.1.0 digest_0.6.23 rmarkdown_2.0 pkgconfig_2.0.3<br />
[29] htmltools_0.4.0 highr_0.8 readxl_1.3.1 htmlwidgets_1.5.1 [33] rlang_0.4.2 rstudioapi_0.10 RSQLite_2.2.0 farver_2.0.3<br />
[37] generics_0.0.2 jsonlite_1.6 RCurl_1.95-4.12 fansi_0.4.1<br />
[41] Rcpp_1.0.3 munsell_0.5.0 lifecycle_0.1.0 stringi_1.4.5<br />
[45] whisker_0.3-2 yaml_2.2.0 plyr_1.8.5 grid_3.6.1<br />
[49] blob_1.2.0 promises_1.0.1 crayon_1.3.4 lattice_0.20-38<br />
[53] haven_2.1.1 hms_0.5.3 zeallot_0.1.0 pillar_1.4.3<br />
[57] fastcluster_1.1.25 glue_1.3.1 evaluate_0.14 modelr_0.1.4<br />
[61] vctrs_0.2.1 httpuv_1.5.1 cellranger_1.1.0 gtable_0.3.0<br />
[65] assertthat_0.2.1 xfun_0.12 xtable_1.8-4 broom_0.5.2<br />
[69] later_1.0.0 viridisLite_0.3.0 memoise_1.1.0</p>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>


</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open')
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_').toLowerCase();
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
