---
title: "Integrate ICS CD4 Polyfunctional Scores (PFS) and RNASeq data"
subtitle: "HBS"
author: "Carl Murie and Raphael Gottardo"
date: '`r format(Sys.Date(), "%B %d, %Y")`'
output: 
   BiocStyle::html_document:
    theme: lumen
    toc: true
    toc_float: true
    number_sections: true
---

```{r overall-knitr-options, echo=FALSE, results='hide'}
library(knitr)
opts_chunk$set(cache=FALSE, eval=TRUE, echo=FALSE, message=FALSE, warning=FALSE, results='asis')
```

```{r packages}
library(RNASeqUtilities)
library(Biobase)
library(limma)
library(magrittr)
library(here)
library(data.table)
library(survival)   ## clogit
library(GSEABase)
library(tidyverse)
library(gridExtra)
library(DT)

source(here("code", "mal_utilities.R"))
```

```{r load-data}
library(mal067data)
data(mal067_eset)
```


```{r gsea}
min_gene_set <- 5

## set up GSEA analysis
btm_gtm <- getGmt(here("data/BTM_for_GSEA_20131008.gmt"))
btm_ind <- ids2indices(geneIds(btm_gtm), rownames(mal067_eset))
btm_ind <- btm_ind[sapply(btm_ind, length) > min_gene_set]
```


```{r getmorevars}
pfs <- readRDS(here("data/CD4_HBS_PFS.rds")) 
stim <- "hbs"
```

```{r format}

## prune rnaseq eset to contain only same stimulations and sids
var_sids <- unique(pfs$PTID)

pData(mal067_eset) %>%
  filter(stimulation=="dmso" | stimulation==stim) %>%
  filter(sid %in% var_sids) %$%
  col_id ->
  overlap

prune_meta <- mal067_voom[,overlap]
prune_meta$targets <- prune_meta$targets[,c("col_id", "plate", "age_weeks", "sid",
                                            "pid", "visit", "stimulation",
                                            "total_reads", "age", "case")]
prune_meta$targets$stimulation <- fct_drop(prune_meta$targets$stimulation)

## filter ics data to reduce size
pfs %>%
  dplyr::rename(sid=PTID) %>%
  select(c(sid, PFS)) ->
  data
```

# Polyfunctionality Score (PFS) analysis with RNASeq gene expression data

## GSEA analysis {.tabset .tabset-fade .tabset-pills}  

```{r read_child1}
## read in child file for rendering
child1 <- here::here("analysis/children", "pfs_GSEA.Rmd")
childy <- read_file(child1)
```

`r paste(knit_child(text = childy), collapse = '\n')`
</br>


## DEG analysis {.tabset .tabset-fade .tabset-pills}  

```{r read_child1a}
## read in child file for rendering
child1 <- here::here("analysis/children", "exp_PFS_DEG.Rmd")
childy <- read_file(child1)
```

`r paste(knit_child(text = childy), collapse = '\n')`
</br>
