---
title: "Integrate cytokine and RNASeq data"
author: "Carl Murie and Raphael Gottardo"
date: '`r format(Sys.Date(), "%B %d, %Y")`'
output: 
   BiocStyle::html_document:
    theme: lumen
    toc: true
    toc_float: true
    number_sections: true
---

```{r overall-knitr-options, echo=FALSE, results='hide'}
library(knitr)
opts_chunk$set(cache=FALSE, eval=TRUE, echo=FALSE, message=FALSE, warning=FALSE, results='asis')
```

```{r packages}
library(RNASeqUtilities)
library(Biobase)
library(limma)
library(magrittr)
library(here)
library(data.table)
library(survival)   ## clogit
library(GSEABase)
library(tidyverse)
library(gridExtra)
library(htmltools)

source(here("code", "mal_utilities.R"))
```

```{r load-data}
library(mal067data)
data(mal067_eset)
meta_dt <- as_tibble(pData(mal067_eset))
```


```{r gsea}
min_gene_set <- 5

## set up GSEA analysis
btm_gtm <- getGmt(here("data/BTM_for_GSEA_20131008.gmt"))
btm_ind <- ids2indices(geneIds(btm_gtm), rownames(mal067_eset))
btm_ind <- btm_ind[sapply(btm_ind, length) > min_gene_set]
```


```{r getmorevars}

## l_df_flow and l_df_rna
##
## 1. cytokines: stimulation (HBS, CSP),antigens (28), and log10 stim/DMSO ratios
## 2. IAVI: IgG (CSP - NANP), IgG (CSP - C-term)
## 3. CEVAC: IgG CSP-NANP titers, IgG HBsAg titers (both in log10 of the titers)
## 4. antibody Luminex: isotope (IgG, IgM), analyte (42 antigens), log10 of the
##                      median fluorecence intensity
load("data/m067_seattle_data.RData")

```

# Cytokines and RNASeq analysis  
  
Database of  cytokine Luminex contains 3 new variables: stimulation (CSP and HBS), analyte (28 antigens) and the Log10_ratio_conc_imp_trunc (log10 of the CSP/DMSO and HBS/DMSO of the ratio).  
  
```{r cytokines} 

## format rnaseq meta data
meta_dt %>%
  filter(visit=="M3") %>%
  filter(vaccine=="rtss") %>%
  mutate(stimulation=as.character(stimulation)) %>%
  dplyr::select(sid, stimulation, col_id, case, malaria_status, match) ->
  meta_cyto

## format cytokine data
l_df_rna[[1]] %>%
  mutate(sid=as.character(sid)) %>%
  mutate(stimulation=tolower(stimulation)) %>%
  mutate(analyte=str_replace_all(analyte, "-", "_")) ->
  df_rna

##countSampleSizes(df_rna, "analyte", c("stimulation", "case")) %>%
##  headerKable()

## join cytokine data with rnaseq meta data
dplyr::inner_join(df_rna, meta_cyto, by=c("sid", "stimulation")) %>%
dplyr::rename(log10_ratio = Log10_ratio_conc_imp_trunc) ->
  cyto

## check that total sample size is correct
total <- length(intersect(paste(meta_cyto$stimulation,meta_cyto$sid),
                          paste(df_rna$stimulation, df_rna$sid))) *
  length(unique(df_rna$analyte))
if(total != nrow(cyto)) {stop("Sample size is not correct")}

```

## Sample size

###cytokine database  
  
There are only samples from visit M3.  
  
```{r samplesize1}
countSampleSizes(df_rna, "analyte", c("stimulation", "vaccine")) %>%
  headerKable()
```
 
###Plot of cytokine database  
  
```{r plotty}

tmp <- data.table(df_rna)[stimulation=="csp"]
ggplot(tmp, aes(x=analyte, y=Log10_ratio_conc_imp_trunc)) +
  geom_boxplot() + ggtitle("CSP: log10_ratio") + 
  theme(axis.text.x=element_text(angle=45, hjust=1))
```
###after joining rnaSeq meta-data and cytokine database  
  
All comparator samples were removed.  
  
```{r samplesize2}
countSampleSizes(cyto, "analyte", c("stimulation", "case", "vaccine")) %>%
  headerKable()
```

## Disease prediction  

A clogit test of case-control vs cytokine ratio with gender and match as a stratified variable was applied to the cytokine frequency.  

```{r cyto_logit}

## gather just case/control samples which the match stratified variable
cyto %>%
  filter(case != "neither") %>%
  mutate(case = fct_drop(case)) %>%                    ## remove "neither" level
  ## clogit needs case as a binary value (0,1)
  mutate(case_logit=as.numeric(plyr::mapvalues(as.character(case), 
                                               from=c("control", "case"),
                                               to=c(0,1)))) %>%
  as.data.table() ->
  cyto_logit
```

###Sample size  
  
```{r sample3}
countSampleSizes(cyto_logit, "analyte", c("stimulation", "case")) %>%
  headerKable()
```

```{r output}
 pvals <- cyto_logit[,summary(clogit(case_logit~sex+log10_ratio+strata(match)))$coefficient["log10_ratio", 5], by=.(stimulation, analyte)]
 
  ## plot boxplots of log10 ratio to case/control
 gg_csp <- ggplot(cyto_logit[stimulation=="csp"], aes(x=case, y=log10_ratio)) + 
   geom_boxplot() + xlab("case") + facet_wrap(~analyte) + ggtitle("CSP")
 
 gg_hbs <- ggplot(cyto_logit[stimulation=="hbs"], aes(x=case, y=log10_ratio)) + 
   geom_boxplot() + xlab("case") + facet_wrap(~analyte) + ggtitle("HBS")

 ## output p-values and plots
 pvals %>% 
   mutate(V1=signif(V1, 4)) %>%
   dplyr::rename("p-value"=V1) %>%
   spread(key="stimulation", value="p-value") %>%
   mutate(csp_fdr=p.adjust(csp, method="fdr")) %>%
   mutate(csp_fdr=signif(csp_fdr, 4)) %>%
   mutate(hbs_fdr=p.adjust(hbs, method="fdr")) %>%
   mutate(hbs_fdr=signif(hbs_fdr, 4)) %>%
   mutate(csp=cell_spec(csp, color = ifelse(csp <= 0.2, "red", "black"))) %>%
   mutate(hbs=cell_spec(hbs, color = ifelse(hbs <= 0.2, "red", "black"))) %>%
   mutate(csp_fdr=cell_spec(csp_fdr, color = ifelse(csp_fdr <= 0.2, 
                                                    "red", "black"))) %>%
   mutate(hbs_fdr=cell_spec(hbs_fdr, color = ifelse(hbs_fdr <= 0.2, 
                                                    "red", "black"))) %>%
 headerKable(title="clogit p-values")
 
```
 
###Log10 ratio plots of case and control
 
```{r plots} 
 print(gg_csp)
 print(gg_hbs)
```
  
##GSEA analysis 
  
  
```{r function} 
applyGSEA <- function(analyte, dat, btm=btm_ind) {
  
  ## remove any potential NAs
  na_ind <- is.na(dat$targets[,analyte])
  voomy <- dat[,!na_ind]
  
  form <- paste0("~plate + total_reads + age_weeks + ", analyte, "*stimulation")
  des <- model.matrix(formula(form), voomy$targets)
  colnames(des)[ncol(des)] <- "analyte"
 
  ## run camera
  cam_vac <- as_tibble(camera(voomy, btm, design=des,
                              contrast="analyte"),
                       rownames="geneset")
  cam_vac %<>%
    mutate(PValue=signif(PValue, 3)) %<>%
    mutate(FDR=signif(FDR, 3))
 return(cam_vac)
}

## get list of analytes
unique(df_rna$analyte)  ->
  analytes
```
  
```{r setup}
fdr_cut <- 0.2

df_wide <- dcast(df_rna, pid+sid+sex+vaccine+stimulation~analyte,
                 value.var="Log10_ratio_conc_imp_trunc")
var_sids <- unique(df_wide$sid)

pData(mal067_eset) %>%
  filter(stimulation !="ama1") %>%
  filter(sid %in% var_sids) %$%
  col_id ->
  overlap

prune_meta <- mal067_voom[,overlap]
prune_meta$targets <- prune_meta$targets[,c("col_id", "plate", "age_weeks", "sid",
                                            "visit", "pid", "stimulation",
                                            "total_reads", "case")]
```
    
Investigate whether the associations between the cytokines and the RNASeq gene expression differe with regard to stimulations.

filtering: M3, rtss, comparator

linear model: ~plate + total_reads + age_weeks + analyte*stimulation"  
  
FDR cutoff <= `r fdr_cut`
  
###CSP {.tabset .tabset-fade .tabset-pills}  
  
```{r csp_filter}

## filter for dmso and csp
df_wide %>% 
  filter(stimulation == "csp") %>%
  select(-c("stimulation")) -> 
  df_csp

csp_sids <- unique(df_csp$sid)

prune_meta$targets %>%
  filter(stimulation != "hbs") %>%
  filter(sid %in% csp_sids) %$%
  col_id ->
  csp_ind

voom_csp <- prune_meta[,csp_ind]
voom_csp$targets$stimulation <- fct_drop(voom_csp$targets$stimulation)

dplyr::inner_join(voom_csp$targets, df_csp, by="sid") ->
  df_join 

if(sum(voom_csp$col_id != df_join$col_id) > 0) {
  stop("Joining analyte and rnaseq data failed")
}

voom_csp$targets <- df_join
```
  
Sample Sizes  
  
```{r csp_samples} 
countSampleSizes(voom_csp$targets, "stimulation", 
                 c("visit", "vaccine")) %>%
  headerKable(labels=c("visit", "vaccine"), levels=c(2,2))
```

```{r run_it_csp}
all_cam <- lapply(analytes, applyGSEA, dat=voom_csp)
names(all_cam) <- analytes

## get number of signficant gene sets
num_sigs <- sapply(all_cam, function(x) return(sum(x$FDR<=fdr_cut)),
                   simplify=TRUE)
out_sigs <- data.frame('significant sets'=sort(num_sigs,
                                                decreasing=TRUE))

## get significant gene sets
sigs <- lapply(all_cam, function(x) return(x[x$FDR <= fdr_cut,]))

```

```{r out_csp}
headerKable(out_sigs)

for(name in rownames(out_sigs)) {
  tmp <- sigs[[name]]
  if(nrow(tmp) > 0) {
     knit_print.html(cat(paste("####", name, "{-} \n  \n")))
     knit_print.html(cat(headerKable(tmp), "\n"))
  }
}

```

###HBS {.tabset .tabset-fade .tabset-pills}  
  
```{r hbs_filter}

## filter for dmso and hbs
df_wide %>% 
  filter(stimulation == "hbs") %>%
  select(-c("stimulation")) -> 
  df_csp

csp_sids <- unique(df_csp$sid)

prune_meta$targets %>%
  filter(stimulation != "csp") %>%
  filter(sid %in% csp_sids) %$%
  col_id ->
  csp_ind

voom_csp <- prune_meta[,csp_ind]
voom_csp$targets$stimulation <- fct_drop(voom_csp$targets$stimulation)

dplyr::inner_join(voom_csp$targets, df_csp, by="sid") ->
  df_join 

if(sum(voom_csp$col_id != df_join$col_id) > 0) {
  stop("Joining analyte and rnaseq data failed")
}

voom_csp$targets <- df_join
```
  
Sample Sizes  
  
```{r hbs_samples} 
countSampleSizes(voom_csp$targets, "stimulation", 
                 c("visit", "vaccine")) %>%
  headerKable(labels=c("visit", "vaccine"), levels=c(2,2))
```

```{r run_it_hbs}
all_cam <- lapply(analytes, applyGSEA, dat=voom_csp)
names(all_cam) <- analytes

## get number of signficant gene sets
num_sigs <- sapply(all_cam, function(x) return(sum(x$FDR<=fdr_cut)),
                   simplify=TRUE)
out_sigs <- data.frame('significant sets'=sort(num_sigs,
                                                decreasing=TRUE))

## get significant gene sets
sigs <- lapply(all_cam, function(x) return(x[x$FDR <= fdr_cut,]))

```

```{r out_hbs}
headerKable(out_sigs)

for(name in rownames(out_sigs)) {
  tmp <- sigs[[name]]
  if(nrow(tmp) > 0) {
     knit_print.html(cat(paste("####", name, "{-} \n  \n")))
     knit_print.html(cat(headerKable(tmp), "\n"))
  }
}

```
