---
title: "Integrate CEVAC and RNASeq data"
author: "Carl Murie and Raphael Gottardo"
date: '`r format(Sys.Date(), "%B %d, %Y")`'
output: 
   BiocStyle::html_document:
    theme: lumen
    toc: true
    toc_float: true
    number_sections: true
---

```{r overall-knitr-options, echo=FALSE, results='hide'}
library(knitr)
opts_chunk$set(cache=FALSE, eval=TRUE, echo=FALSE, message=FALSE, warning=FALSE, results='asis')
```

```{r packages}
library(RNASeqUtilities)
library(Biobase)
library(limma)
library(magrittr)
library(here)
library(data.table)
library(survival)   ## clogit
library(GSEABase)
library(tidyverse)

source(here("code", "mal_utilities.R"))
```

```{r load-data}
library(mal067data)
data(mal067_eset)
meta_dt <- as_tibble(pData(mal067_eset))
```


```{r gsea}
min_gene_set <- 5

## set up GSEA analysis
btm_gtm <- getGmt(here("data/BTM_for_GSEA_20131008.gmt"))
btm_ind <- ids2indices(geneIds(btm_gtm), rownames(mal067_eset))
btm_ind <- btm_ind[sapply(btm_ind, length) > min_gene_set]
```


```{r getmorevars}

## l_df_flow and l_df_rna
##
## 1. cytokines: stimulation (HBS, CSP),antigens (28), and log10 stim/DMSO ratios
## 2. IAVI: IgG (CSP - NANP), IgG (CSP - C-term)
## 3. CEVAC: IgG CSP-NANP titers, IgG HBsAg titers (both in log10 of the titers)
## 4. antibody Luminex: isotope (IgG, IgM), analyte (42 antigens), log10 of the
##                      median fluorecence intensity
load("data/m067_seattle_data.RData")

```

# CEVAC and RNASeq analysis  
  
Database of CEVAC contains 2 new variables: log_val.Anti-CS (IgG CSP-NANP titers) and  log_val.HBV.S AB (IgG HBsAg titers), as log10 of the titers.
  
```{r iavi} 

## format rnaseq meta data
meta_dt %>%
  filter(stimulation=="csp" | stimulation=="hbs") %>%
  mutate(stimulation = as.character(fct_drop(stimulation))) %>%
  filter(vaccine=="rtss") %>%
  mutate(visit=as.character(visit)) %>%
  dplyr::select(pid, visit, col_id, case, malaria_status, match, 
                vaccine, stimulation, sex) ->
  meta_cyto

## format CEVAC data
l_df_rna[[3]] %>%
  rename(csp="log_val.Anti-CS") %>%
  rename(hbs= "log_val.HBV.S AB") %>%
  melt(measure=c("csp", "hbs")) %>%
  rename(stimulation=variable) %>%
  rename(log10_score=value) %>%
  filter(!is.na(log10_score)) %>%
  mutate(stimulation=as.character(stimulation))->
  df_rna
```

##Sample size  
  
```{r}
countSampleSizes(df_rna, "visit", "stimulation") %>%
  headerKable()
```

##Plots of new variables

```{r plot}
ggplot(df_rna, aes(x=1:nrow(df_rna), y=log10_score)) + geom_point() +
  facet_wrap(~visit+stimulation) + xlab("order")
```

```{r}
## join cevac data with rnaseq meta data
dups <- !duplicated(paste(meta_cyto$visit,meta_cyto$pid))
dplyr::inner_join(df_rna, meta_cyto, by=c("pid", "visit", "stimulation")) ->
  cyto

## check that total sample size is correct
total <- length(intersect(paste(meta_cyto$visit,meta_cyto$pid,
                                meta_cyto$stimulation),
                          paste(df_rna$visit, df_rna$pid, df_rna$stimulation))) 
if(total != nrow(cyto)) {stop("Sample size is not correct")}

```

## Sample size  
  
There are only 3 samples at time point M0 after removing the comparator samples so only the M3 time point will be analysed.
  
```{r samplesize1}
 countSampleSizes(cyto, "visit", c("stimulation", "vaccine", "case")) %>%
  headerKable()
```

## Disease prediction  

A clogit test of case-control vs isotype titers with gender and match as a stratified variable was applied to the CEVAC measurements.  

```{r cyto_logit}

## gather just case/control samples which the match stratified variable
cyto %>%
  filter(case != "neither") %>%
  mutate(case = fct_drop(case)) %>%                    ## remove "neither" level
  ## clogit needs case as a binary value (0,1)
  mutate(case_logit=as.numeric(plyr::mapvalues(as.character(case), 
                                               from=c("control", "case"),
                                               to=c(0,1)))) %>%
  filter(visit=="M3") %>%
  mutate(visit = fct_drop(visit)) %>%
  as.data.table() ->
  cyto_logit
```

###Sample size  
  
```{r sample3} 
countSampleSizes(cyto_logit, "visit", c("case", "vaccine")) %>%
  headerKable()
```

```{r output}

 ##x <- clogit(case_logit~sex+cterm+strata(match), data=cyto_logit[visit=="M0"])

 pvals <- cyto_logit[,summary(clogit(case_logit~sex+log10_score+strata(match)))$coefficient["log10_score", 5], by=.(stimulation)]
colnames(pvals)[2] <- "pValue"
 
  ## plot boxplots of log10 ratio to case/control
 gg_plott <- ggplot(cyto_logit, aes(x=case, y=log10_score)) + geom_boxplot() +
   xlab("case") + facet_wrap(~stimulation)
 
```
  
  
```{r table}
 ## output p-values and plots
 pvals %>%
   mutate(pValue=signif(pValue, 4)) %>%
   mutate(FDR=signif(p.adjust(pValue, method="fdr"), 4)) %>%
   mutate(FDR=cell_spec(FDR, color = ifelse(FDR<=0.2, "red", "black"))) %>%
   mutate(pValue=cell_spec(pValue, color = ifelse(pValue<=0.2, "red", "black"))) %>%
 headerKable(title="clogit p-values")
```
 
##plots of case and control
 
```{r plots} 
 print(gg_plott)
```
  
##Correlations between CEVAC and RNASeq gene expression for significant GSEA gene-sets  {.tabset .tabset-fade .tabset-pills}  
  
  
```{r sig1}
sigSetName <- "enriched in myeloid cells and monocytes (M81)"
sigSet <- btm_ind[[sigSetName]]
```
  
###`r sigSetName` | IAVA vs RNASeq gene expression 
  
Correlations greater than 0.2 or less than -0.2 are in red.  
  
```{r sig1_analysis}

rna_DT <- melt(exprs(mal067_eset)[sigSet,], variable.names=1:ncol(mal067_eset))
dplyr::inner_join(cyto, rna_DT, by=c("col_id"="Var2")) %>%
  filter(visit=="M3") %>%
  rename(gene=Var1) %>%
  data.table() ->
  all_DT

corAll <- all_DT[,.(pearson=signif(cor(value, log10_score,
                                            use="pairwise.complete.obs"), 3),
              spearman=signif(cor(value, log10_score, method="spearman",
                                  use="pairwise.complete.obs"), 3)),
              by=stimulation]

corGene <- all_DT[,.(pearson=signif(cor(value, log10_score,
                                            use="pairwise.complete.obs"), 3),
              spearman=signif(cor(value, log10_score, method="spearman",
                                  use="pairwise.complete.obs"), 3)),
              by=.(gene, stimulation)]

corAll %>%
mutate(pearson=cell_spec(pearson, 
                         color = ifelse(pearson >= 0.2, "red", "black"))) %>%
mutate(spearman=cell_spec(spearman, 
                         color = ifelse(spearman >= 0.2, "red", "black"))) %>%
headerKable(title="all genes together")

```

####Correlation plots  
  
```{r corplots}

melt(corGene, measure=c("pearson", "spearman")) %>%
  ggplot(aes(x=stimulation, y=value)) + geom_boxplot() + geom_point()  +
  ylim(-1,1) + ggtitle("CEVAC") + facet_wrap(~variable) + ylab("correlation") +
  geom_hline(yintercept=c(-0.2, 0.2), linetype="dashed", color = "red")

```

####Correlation tables  
  
Correlations greater than 0.2 or less than -0.2 are in red.   
  
```{r cortable}

corGene %>%
  select(c(gene, stimulation, spearman)) %>%
  spread(key="stimulation", value=c("spearman")) %>%
  mutate(csp=cell_spec(csp, color = ifelse(csp >= 0.2 | csp <= -0.2, 
                                        "red", "black")))%>%
  mutate(hbs=cell_spec(hbs, color = ifelse(hbs >= 0.2 | hbs <= -0.2, 
                                        "red", "black"))) -> 
  spear

corGene %>%
  select(c(gene, stimulation, pearson)) %>%
  spread(key="stimulation", value=c("pearson")) %>%
  mutate(csp=cell_spec(csp, color = ifelse(csp >= 0.2 | csp <= -0.2, 
                                        "red", "black")))%>%
  mutate(hbs=cell_spec(hbs, color = ifelse(hbs >= 0.2 | hbs <= -0.2, 
                                        "red", "black"))) -> 
  pear

  cbind(pear, spear[,-1]) %>%
    headerKable() %>%
    add_header_above(c(" "=1, "pearson"=2, "spearman"=2))
  
```




```{r sig2}
sigSetName <- "NK cell surface signature (S1)"
sigSet <- btm_ind[[sigSetName]]
```
 
###`r sigSetName` | IAVA vs RNASeq gene expression 
  
Correlations greater than 0.2 or less than -0.2 are in red.  
  
```{r sig1_analysis2}

rna_DT <- melt(exprs(mal067_eset)[sigSet,], variable.names=1:ncol(mal067_eset))
dplyr::inner_join(cyto, rna_DT, by=c("col_id"="Var2")) %>%
  filter(visit=="M3") %>%
  rename(gene=Var1) %>%
  data.table() ->
  all_DT

corAll <- all_DT[,.(pearson=signif(cor(value, log10_score,
                                            use="pairwise.complete.obs"), 3),
              spearman=signif(cor(value, log10_score, method="spearman",
                                  use="pairwise.complete.obs"), 3)),
              by=stimulation]

corGene <- all_DT[,.(pearson=signif(cor(value, log10_score,
                                            use="pairwise.complete.obs"), 3),
              spearman=signif(cor(value, log10_score, method="spearman",
                                  use="pairwise.complete.obs"), 3)),
              by=.(gene, stimulation)]

corAll %>%
mutate(pearson=cell_spec(pearson, 
                         color = ifelse(pearson >= 0.2, "red", "black"))) %>%
mutate(spearman=cell_spec(spearman, 
                         color = ifelse(spearman >= 0.2, "red", "black"))) %>%
headerKable(title="all genes together")

```

####Correlation plots  
  
```{r corplots2}

melt(corGene, measure=c("pearson", "spearman")) %>%
  ggplot(aes(x=stimulation, y=value)) + geom_boxplot() + geom_point()  +
  ylim(-1,1) + ggtitle("CEVAC") + facet_wrap(~variable) + ylab("correlation") +
  geom_hline(yintercept=c(-0.2, 0.2), linetype="dashed", color = "red")

```

####Correlation tables  
  
Correlations greater than 0.2 or less than -0.2 are in red.   
  
```{r cortable2}

corGene %>%
  select(c(gene, stimulation, spearman)) %>%
  spread(key="stimulation", value=c("spearman")) %>%
  mutate(csp=cell_spec(csp, color = ifelse(csp >= 0.2 | csp <= -0.2, 
                                        "red", "black")))%>%
  mutate(hbs=cell_spec(hbs, color = ifelse(hbs >= 0.2 | hbs <= -0.2, 
                                        "red", "black"))) -> 
  spear

corGene %>%
  select(c(gene, stimulation, pearson)) %>%
  spread(key="stimulation", value=c("pearson")) %>%
  mutate(csp=cell_spec(csp, color = ifelse(csp >= 0.2 | csp <= -0.2, 
                                        "red", "black")))%>%
  mutate(hbs=cell_spec(hbs, color = ifelse(hbs >= 0.2 | hbs <= -0.2, 
                                        "red", "black"))) -> 
  pear

  cbind(pear, spear[,-1]) %>%
    headerKable() %>%
    add_header_above(c(" "=1, "pearson"=2, "spearman"=2))
  
```



```{r sig3}
sigSetName <- "enriched in monocytes (I) (M4.15)"
sigSet <- btm_ind[[sigSetName]]
```

###`r sigSetName` | IAVA vs RNASeq gene expression 
  
Correlations greater than 0.2 or less than -0.2 are in red.  
  
```{r sig1_analysis3}

rna_DT <- melt(exprs(mal067_eset)[sigSet,], variable.names=1:ncol(mal067_eset))
dplyr::inner_join(cyto, rna_DT, by=c("col_id"="Var2")) %>%
  filter(visit=="M3") %>%
  rename(gene=Var1) %>%
  data.table() ->
  all_DT

corAll <- all_DT[,.(pearson=signif(cor(value, log10_score,
                                            use="pairwise.complete.obs"), 3),
              spearman=signif(cor(value, log10_score, method="spearman",
                                  use="pairwise.complete.obs"), 3)),
              by=stimulation]

corGene <- all_DT[,.(pearson=signif(cor(value, log10_score,
                                            use="pairwise.complete.obs"), 3),
              spearman=signif(cor(value, log10_score, method="spearman",
                                  use="pairwise.complete.obs"), 3)),
              by=.(gene, stimulation)]

corAll %>%
mutate(pearson=cell_spec(pearson, 
                         color = ifelse(pearson >= 0.2, "red", "black"))) %>%
mutate(spearman=cell_spec(spearman, 
                         color = ifelse(spearman >= 0.2, "red", "black"))) %>%
headerKable(title="all genes together")

```

####Correlation plots  
  
```{r corplots3}

melt(corGene, measure=c("pearson", "spearman")) %>%
  ggplot(aes(x=stimulation, y=value)) + geom_boxplot() + geom_point()  +
  ylim(-1,1) + ggtitle("CEVAC") + facet_wrap(~variable) + ylab("correlation") +
  geom_hline(yintercept=c(-0.2, 0.2), linetype="dashed", color = "red")

```

####Correlation tables  
  
Correlations greater than 0.2 or less than -0.2 are in red.   
  
```{r cortable3}

corGene %>%
  select(c(gene, stimulation, spearman)) %>%
  spread(key="stimulation", value=c("spearman")) %>%
  mutate(csp=cell_spec(csp, color = ifelse(csp >= 0.2 | csp <= -0.2, 
                                        "red", "black")))%>%
  mutate(hbs=cell_spec(hbs, color = ifelse(hbs >= 0.2 | hbs <= -0.2, 
                                        "red", "black"))) -> 
  spear

corGene %>%
  select(c(gene, stimulation, pearson)) %>%
  spread(key="stimulation", value=c("pearson")) %>%
  mutate(csp=cell_spec(csp, color = ifelse(csp >= 0.2 | csp <= -0.2, 
                                        "red", "black")))%>%
  mutate(hbs=cell_spec(hbs, color = ifelse(hbs >= 0.2 | hbs <= -0.2, 
                                        "red", "black"))) -> 
  pear

  cbind(pear, spear[,-1]) %>%
    headerKable() %>%
    add_header_above(c(" "=1, "pearson"=2, "spearman"=2))
  
```


